<!DOCTYPE html>
<html lang="en" dir="ltr" prefix="og: https://ogp.me/ns#">
  <head>
    <meta charset="utf-8" />
<meta name="description" content="The ME/CFS Research Roadmap webinar on metabolism in ME/CFS took place on October 26, 2023." />
<meta name="keywords" content="Metabolism – October 26, 2023" />
<meta name="robots" content="index, follow" />
<link rel="canonical" href="https://www.nih.gov/research-training/medical-research-initiatives/advancing-mecfs-research/events/metabolism-october-26-2023" />
<link rel="shortlink" href="https://www.nih.gov/node/2331" />
<meta property="og:site_name" content="National Institutes of Health (NIH)" />
<meta property="og:type" content="Article" />
<meta property="og:url" content="https://www.nih.gov/research-training/medical-research-initiatives/advancing-mecfs-research/events/metabolism-october-26-2023" />
<meta property="og:title" content="Metabolism – October 26, 2023" />
<meta property="og:description" content="[Enter summary here]" />
<meta property="og:updated_time" content="2025-01-22T14:35:43-05:00" />
<meta property="article:modified_time" content="2025-01-22T14:35:43-05:00" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:description" content="[Enter summary here]" />
<meta name="twitter:title" content="Metabolism – October 26, 2023" />
<meta name="Generator" content="Drupal 10 (https://www.drupal.org)" />
<meta name="MobileOptimized" content="width" />
<meta name="HandheldFriendly" content="true" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<link rel="icon" href="/themes/custom/nih2_uswds/assets/img/favicons/custom/favicon-192x192.png" type="image/png" />

    <title>Metabolism – October 26, 2023 | National Institutes of Health (NIH)</title>
    <link rel="stylesheet" media="all" href="/sites/default/files/css/css_b7RB5ckPbDIfuDDeXs0bte1FGx8XxXGa1oNT-xX4DCY.css?delta=0&amp;language=en&amp;theme=nih2_uswds&amp;include=eJxFjVsOwyAMBC_kgtQLRQ64Ca3BCEMSevq2eag_K-1qNOsYVbuNpIoTKTgpZH1pGdngE7djSFIicngT0FY5pNeFnBUYu7Q6-KBOFirdSiInDJF8QCsUR_Lm8bPUSgVSmO9D09WrdU2rxJvW_hVNkLHgVDDPel38F9NSbiMHncmDdq0U7YhKcJgO_ix7whJoVbunieIb0wcyx14f" />
<link rel="stylesheet" media="all" href="/sites/default/files/css/css_zvnmitdxF7Azlr40tTVTUAnwywPlL_yNojzXeI2geVM.css?delta=1&amp;language=en&amp;theme=nih2_uswds&amp;include=eJxFjVsOwyAMBC_kgtQLRQ64Ca3BCEMSevq2eag_K-1qNOsYVbuNpIoTKTgpZH1pGdngE7djSFIicngT0FY5pNeFnBUYu7Q6-KBOFirdSiInDJF8QCsUR_Lm8bPUSgVSmO9D09WrdU2rxJvW_hVNkLHgVDDPel38F9NSbiMHncmDdq0U7YhKcJgO_ix7whJoVbunieIb0wcyx14f" />

    <script type="application/json" data-drupal-selector="drupal-settings-json">{"path":{"baseUrl":"\/","pathPrefix":"","currentPath":"node\/2331","currentPathIsAdmin":false,"isFront":false,"currentLanguage":"en"},"pluralDelimiter":"\u0003","suppressDeprecationErrors":true,"gtag":{"tagId":"","consentMode":false,"otherIds":[],"events":[],"additionalConfigInfo":[]},"ajaxPageState":{"libraries":"eJxdUYtuxCAM-yFapP1QlUJKs_ESCb3jvn7s2m7tJGRiGxEHjAfmpmdgVGavAzKDQ1YmFdQxlQCeXt0u8GronD4LhU_xFL-0LTWDHw-qFpKNLOtj_-UUSZRLyXmcBJx2Hf7zET7heReD8tBSlckSm7RhaTpFNMmrgJZAJwwz2nH5CSqCRUVaP6bKj97SVJYUBpbWk7mrs3jKg4Fir2JBRhlyn_6uSi1xkNRXvhpMFiNsXXY97t0RHMBjkdPMUMAVyCufz_WnjDXmOnviFa3ixoJh_5H9sv38Qd6oNsIH6zeOIdnq8Rto7bEd","theme":"nih2_uswds","theme_token":null},"ajaxTrustedUrl":[],"gtm":{"tagId":null,"settings":{"data_layer":"dataLayer","include_classes":false,"allowlist_classes":"","blocklist_classes":"","include_environment":false,"environment_id":"","environment_token":""},"tagIds":["GTM-5CBCNW2"]},"fitvids":{"selectors":".node\r,.field--name-field-media-oembed-video","custom_vendors":"iframe[src^=\u0022https:\/\/youtu.be\u0022],iframe[src^=\u0022https:\/\/youtu.be\u0022],iframe[src^=\u0022https:\/\/www.youtube.com\u0022],iframe[src^=\u0022\/media\/oembed?url=https:\/\/youtu.be\u0022],iframe[src^=\u0022\/media\/oembed?url=https%3A\/\/youtu.be\u0022],iframe[src^=\u0022\/media\/oembed?url=https:\/\/youtube.com\u0022],iframe[src^=\u0022\/media\/oembed?url=https%3A\/\/youtube.com\u0022]","ignore_selectors":""},"data":{"extlink":{"extTarget":true,"extTargetAppendNewWindowLabel":"(opens in a new window)","extTargetNoOverride":false,"extNofollow":true,"extTitleNoOverride":false,"extNoreferrer":true,"extFollowNoOverride":false,"extClass":"ext","extLabel":"(link is external)","extImgClass":false,"extSubdomains":false,"extExclude":"","extInclude":"","extCssExclude":".usa-nav, .usa-sidenav, .usa-button, .icon-link, .share-buttons, .usa-footer__social-links, .teaser-list, .usa-card-group, .link--no-tag","extCssInclude":"","extCssExplicit":"","extAlert":false,"extAlertText":"You are leaving the NIH website.\r\n\r\nThis external link provides additional information that is consistent with the intended purpose of this site. NIH cannot attest to the accuracy of a non-federal site.\r\n\r\nLinking to a non-federal site does not constitute an endorsement by NIH or any of its employees of the sponsors or the information and products presented on the site. You will be subject to the destination site\u2019s privacy policy when you follow the link.","extHideIcons":false,"mailtoClass":"mailto","telClass":"0","mailtoLabel":"(link sends email)","telLabel":"","extUseFontAwesome":false,"extIconPlacement":"after","extPreventOrphan":false,"extFaLinkClasses":"fa fa-external-link","extFaMailtoClasses":"fa fa-envelope-o","extAdditionalLinkClasses":"","extAdditionalMailtoClasses":"","extAdditionalTelClasses":"","extFaTelClasses":"fa fa-phone","whitelistedDomains":["www.nih.gov","salud.nih.gov","ned.nih.gov","calendar.nih.gov","search.usa.gov","search.nih.gov","search.gov","cloud.acquia.com"],"extExcludeNoreferrer":""}},"user":{"uid":0,"permissionsHash":"52c204d27c77368827b33df801757282ad522f57fc486a23070b7ac529ab2eeb"}}</script>
<script src="/sites/default/files/js/js_I8yN2jcadLwQ2UnJhfoYhgH0rMoIYFCge7ov_eWFMGQ.js?scope=header&amp;delta=0&amp;language=en&amp;theme=nih2_uswds&amp;include=eJxdjYEOwiAMRH8IRuIPLQ3UWkUgpczp14uKZjNprr13ydULPO5I5Pw4DK4aOV1ckFYgTsOaI-vCobqxf54Tq6GcKeKsQI66_PsJzrDu4dUkPh3mVm-vzsjFepCwhYIV1RYg3FNtkqzmPmUbVA6YYOmY-pd9omghoug3_PC3PgGlVmKC"></script>
<script src="https://script.crazyegg.com/pages/scripts/0011/2616.js" async></script>
<script src="/modules/contrib/google_tag/js/gtag.js?t0h3vu"></script>
<script src="/modules/contrib/google_tag/js/gtm.js?t0h3vu"></script>


    <script type="text/javascript">
      // ForeSee Production Embed Script v2.01
      // DO NOT MODIFY BELOW THIS LINE *****************************************
      ;(function (g) {
        var d = document, am = d.createElement('script'), h = d.head || d.getElementsByTagName("head")[0], fsr = 'fsReady',
        aex = {
          "src": "//gateway.foresee.com/sites/nih-gov/production/gateway.min.js",
          "type": "text/javascript",
          "async": "true",
          "data-vendor": "fs",
          "data-role": "gateway"
        };
        for (var attr in aex) { am.setAttribute(attr, aex[attr]); } h.appendChild(am); g[fsr] || (g[fsr] = function () { var aT = '__' + fsr + '_stk__'; g[aT] = g[aT] || []; g[aT].push(arguments); });
      })(window);
      // DO NOT MODIFY ABOVE THIS LINE *****************************************
  
      // Un-comment out the function below when you are ready to input your variable
      /*fsReady(function() {
        FSR.CPPS.set('name','value'); // use single quotes when passing a static-value
        FSR.CPPS.set('name2',somevariable); // don't use quotes for a dynamic value
        FSR.CPPS.set('name3',othervariable); // add as many CPPs as you like in this way
      });*/
    </script>
  </head>
  <body class="exclude-node-title path-node page-node-type-page">
        <a href="#main-content" class="visually-hidden focusable skip-link">
      Skip to main content
    </a>
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5CBCNW2"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>

      <div class="dialog-off-canvas-main-canvas" data-off-canvas-main-canvas>
    
  <section class="usa-banner" aria-label="Official government website">
  <div class="usa-accordion">
    <header class="usa-banner__header">
      <div class="usa-banner__inner">
        <div class="grid-col-auto">
          <img class="usa-banner__header-flag" src="/themes/custom/nih2_uswds/assets/img/us_flag.svg" alt="U.S. flag">
        </div>
        <div class="grid-col-fill tablet:grid-col-auto">
          <p class="usa-banner__header-text">An official website of the United States government</p>
          <p class="usa-banner__header-action" aria-hidden="true">Here’s how you know</p>
        </div>
        <button class="usa-accordion__button usa-banner__button"
          aria-expanded="false" aria-controls="gov-banner">
          <span class="usa-banner__button-text">Here’s how you know</span>
        </button>
      </div>
    </header>
    <div class="usa-banner__content usa-accordion__content" id="gov-banner" hidden>
      <div class="grid-row grid-gap-lg">
        <div class="usa-banner__guidance tablet:grid-col-6">
          <img class="usa-banner__icon usa-media-block__img" src="/themes/custom/nih2_uswds/assets/img/icon-dot-gov.svg" role="img" alt="Dot gov">
          <div class="usa-media-block__body">
            <p>
              <strong>
                Official websites use .gov              </strong>
              <br/>
                              A <strong>.gov</strong> website belongs to an official government organization in the United States.
                          </p>
          </div>
        </div>
        <div class="usa-banner__guidance tablet:grid-col-6">
          <img class="usa-banner__icon usa-media-block__img" src="/themes/custom/nih2_uswds/assets/img/icon-https.svg" role="img" alt="Https">
          <div class="usa-media-block__body">
            <p>
              <strong>
                Secure .gov websites use HTTPS              </strong>
              <br/>
              A <strong>lock</strong>
                (<span class="icon-lock"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="img" aria-labelledby="banner-lock-title banner-lock-description"><title id="banner-lock-title">Lock</title><desc id="banner-lock-description">A locked padlock</desc><path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"/></svg></span>)
                or <strong>https://</strong> means you’ve safely connected to the .gov website. Share sensitive information only on official, secure websites.            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>



<div class="usa-overlay"></div>

  <div class="grid-container">
      <div class="region region-header-top">
    <div id="block-nih2-uswds-verintforesee" class="block block-block-content block-block-content8bfdb0cc-3b85-4c96-95c1-f93aa5201854">
  
    
      
  </div>

  </div>

  </div>

<header class="usa-header usa-header--extended" role="banner">
      <div class="usa-navbar">
        <div class="region region-header">
    <div class="usa-logo" id="logo">

    <a class="logo-img" href="/" accesskey="1" title="Home" aria-label="Home">
    <img src="/themes/custom/nih2_uswds/assets/img/nih-logo.png" alt="Home" />
  </a>
    
</div>

  </div>

      <button class="usa-menu-btn">Menu</button>
    </div>

    <nav aria-label="Primary navigation" class="usa-nav" role="navigation">
      <button class="usa-nav__close">
        <img src="/themes/custom/nih2_uswds/assets/img/close.svg" alt="close"/>
      </button>
              <div class="usa-nav__inner">
        
                    <div class="region region-primary-menu">
      
  
          <ul class="usa-nav__primary usa-accordion">
    
    
              <li class="usa-nav__primary-item">
      
              <button class="usa-accordion__button usa-nav__link " aria-expanded="false" aria-controls="basic-nav-section-1">
          <span>Health Information</span>
        </button>
      
                
  
          <ul id="basic-nav-section-1" class="usa-nav__submenu" hidden="">

              <li class="usa-nav__submenu-item">
          <a href="/health-information" data-drupal-link-system-path="node/96">Health Information</a> 
        </li>
          
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://www.hhs.gov/programs/social-services/health-care-facilities/index.html">
                    <span>Health Care Providers &amp; Facilities</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/health-information/health-info-lines">
                    <span>Health Info Lines</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://medlineplus.gov/">
                    <span>MedlinePlus Health Info</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://science.education.nih.gov/">
                    <span>Science Education Resources</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/health-information/nih-clinical-research-trials-you">
                    <span>NIH Clinical Research Trials and You</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/health-information/your-healthiest-self-wellness-toolkits">
                    <span>Your Healthiest Self: Wellness Toolkits</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://newsinhealth.nih.gov/">
                    <span>NIH News in Health</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/institutes-nih/office-communications-public-liaison/clear-communication/talking-your-doctor-or-health-care-provider">
                    <span>Talking With Your Doctor or Health Care Provider</span>
                  </a>
                
              
              </li>
      
    
          </ul>
    
  

      
              </li>
      
    
              <li class="usa-nav__primary-item">
      
              <button class="usa-accordion__button usa-nav__link " aria-expanded="false" aria-controls="basic-nav-section-2">
          <span>Grants &amp; Funding</span>
        </button>
      
                
  
          <ul id="basic-nav-section-2" class="usa-nav__submenu" hidden="">

              <li class="usa-nav__submenu-item">
          <a href="/grants-funding" data-drupal-link-system-path="node/101">Grants &amp; Funding</a> 
        </li>
          
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://grants.nih.gov/">
                    <span>Grants Home Page</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://grants.nih.gov/funding/index.htm">
                    <span>Find Funding</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://grants.nih.gov/grants/how-to-apply-application-guide/due-dates-and-submission-policies/due-dates.htm">
                    <span>Due Dates</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://grants.nih.gov/grants/how-to-apply-application-guide.html">
                    <span>How to Apply</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://grants.nih.gov/new-to-nih">
                    <span>New to NIH</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://grants.nih.gov/grants/about_grants.htm">
                    <span>Policy &amp; Compliance</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://nexus.od.nih.gov/all/">
                    <span>Grants News/Blog</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://grants.nih.gov/funding/contracts.htm">
                    <span>Contracts</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://www.lrp.nih.gov/">
                    <span>Loan Repayment</span>
                  </a>
                
              
              </li>
      
    
          </ul>
    
  

      
              </li>
      
    
              <li class="usa-nav__primary-item">
      
              <button class="usa-accordion__button usa-nav__link " aria-expanded="false" aria-controls="basic-nav-section-3">
          <span>News &amp; Events</span>
        </button>
      
                
  
          <ul id="basic-nav-section-3" class="usa-nav__submenu" hidden="">

              <li class="usa-nav__submenu-item">
          <a href="/news-events" data-drupal-link-system-path="node/121">News &amp; Events</a> 
        </li>
          
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/news-events/news-releases">
                    <span>News Releases</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/news-events/nih-social-media">
                    <span>NIH on Social Media</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/news-events/media-inquiries">
                    <span>Media Contacts</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/news-events/images-b-roll">
                    <span>Images and B-roll</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://calendar.nih.gov/">
                    <span>Calendar of Events</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/news-events/nih-news-publications">
                    <span>NIH News Publications</span>
                  </a>
                
              
              </li>
      
    
          </ul>
    
  

      
              </li>
      
    
              <li class="usa-nav__primary-item">
      
              <button class="usa-accordion__button usa-nav__link usa-current" aria-expanded="false" aria-controls="basic-nav-section-4">
          <span>Research &amp; Training</span>
        </button>
      
                
  
          <ul id="basic-nav-section-4" class="usa-nav__submenu" hidden="">

              <li class="usa-nav__submenu-item">
          <a href="/research-training" data-drupal-link-system-path="node/106">Research &amp; Training</a> 
        </li>
          
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/medical-research-initiatives">
                    <span>Medical Research Initiatives</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/science-highlights">
                    <span>Science Highlights</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://science.education.nih.gov/">
                    <span>Science Education</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/research-nih-labs-clinics">
                    <span>Research in NIH Labs &amp; Clinics</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/training-opportunities">
                    <span>Training Opportunities</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/library-resources">
                    <span>Library Resources</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/research-resources">
                    <span>Research Resources</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/clinical-research-resources">
                    <span>Clinical Research Resources</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/safety-regulation-guidance">
                    <span>Safety, Regulation &amp; Guidance</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/research-training/lasker-clinical-research-scholars">
                    <span>Lasker Clinical Research Scholars</span>
                  </a>
                
              
              </li>
      
    
          </ul>
    
  

      
              </li>
      
    
              <li class="usa-nav__primary-item">
      
              <button class="usa-accordion__button usa-nav__link " aria-expanded="false" aria-controls="basic-nav-section-5">
          <span>Institutes at NIH</span>
        </button>
      
                
  
          <ul id="basic-nav-section-5" class="usa-nav__submenu" hidden="">

              <li class="usa-nav__submenu-item">
          <a href="/institutes-nih" data-drupal-link-system-path="node/111">Institutes at NIH</a> 
        </li>
          
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/institutes-nih/list-institutes-centers">
                    <span>List of Institutes and Centers</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/institutes-nih/nih-office-director">
                    <span>NIH Office of the Director</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/institutes-nih/directors-nih-institutes-centers">
                    <span>Directors of NIH Institutes and Centers</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/institutes-nih/nih-institute-center-contact-information">
                    <span>NIH Institute and Center Contact Information</span>
                  </a>
                
              
              </li>
      
    
          </ul>
    
  

      
              </li>
      
    
              <li class="usa-nav__primary-item">
      
              <button class="usa-accordion__button usa-nav__link " aria-expanded="false" aria-controls="basic-nav-section-6">
          <span>About NIH</span>
        </button>
      
                
  
          <ul id="basic-nav-section-6" class="usa-nav__submenu" hidden="">

              <li class="usa-nav__submenu-item">
          <a href="/about-nih" data-drupal-link-system-path="node/116">About NIH</a> 
        </li>
          
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/mission-goals">
                    <span>Mission and Goals</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/organization">
                    <span>Organization</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/nih-director">
                    <span>The NIH Director</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/nih-almanac">
                    <span>The NIH Almanac</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/nih-turning-discovery-into-health">
                    <span>NIH...Turning Discovery Into Health®</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/impact-nih-research">
                    <span>Impact of NIH Research</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/science-health-public-trust">
                    <span>Science, Health, and Public Trust</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/visitor-information">
                    <span>Visitor Information</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/get-involved-nih">
                    <span>Get Involved at NIH</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="https://hr.nih.gov/jobs">
                    <span>Jobs at NIH</span>
                  </a>
                
              
              </li>
      
    
              <li class="usa-nav__submenu-item">
      
      
                     
                          <a href="/about-nih/contact-us">
                    <span>Contact Us</span>
                  </a>
                
              
              </li>
      
    
          </ul>
    
  

      
              </li>
      
    
          </ul>
    
  




  </div>

        
                  <div class="usa-nav__secondary">
              <div class="region region-secondary-menu">
    

<ul class="usa-nav__secondary-links">
  <li class="usa-nav__secondary-item">
    
                  
    <a href="/virtual-tour">Virtual Tour</a>
  </li>
  <li class="usa-nav__secondary-item">
    
                  
    <a href="https://salud.nih.gov/">En Español</a>
  </li>
</ul>

<div class="search-block-form usa-form block block-search container-inline" data-drupal-selector="search-block-form" id="block-nih2-uswds-search" role="search">
  
    
  <form class="usa-search" role="search" id="search_form" action="https://search.usa.gov/search" accept-charset="UTF-8" method="get">
    <input name="utf8" type="hidden" value="&#x2713;" />
    <input type="hidden" name="affiliate" id="affiliate" value="nih" />
    <label class="usa-sr-only" for="extended-search-field-en-small">Search</label>
    <input class="usa-input usagov-search-autocomplete" id="extended-search-field-en-small" type="search" name="query" autocomplete="off" placeholder="Search NIH" />
    <button class="usa-button" type="submit" name="commit" data-disable-with="Search"><span class="usa-search__submit-text">Search</span><img src="/themes/custom/nih2_uswds/assets/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search the NIH Website" /></button>
  </form>
</div>

  </div>

          </div>
        
                </div>
          </nav>

    
</header>


  <div class="region region-breadcrumb">
    <div id="block-nih2-uswds-breadcrumbs" class="block block-system block-system-breadcrumb-block">
  
    
      <h2 id="system-breadcrumb" class="visually-hidden">Breadcrumb</h2>
<nav class="usa-breadcrumb usa-breadcrumb--wrap" aria-label="Breadcrumbs">
  <ol class="add-list-reset usa-breadcrumb__list">
      <li class="usa-breadcrumb__list-item">
      <a href="/" class="usa-breadcrumb__link">Home</a>
      </li>
      <li class="usa-breadcrumb__list-item">
      <a href="/research-training" class="usa-breadcrumb__link">Research &amp; Training</a>
      </li>
      <li class="usa-breadcrumb__list-item">
      <a href="/research-training/medical-research-initiatives" class="usa-breadcrumb__link">Medical Research Initiatives</a>
      </li>
  </ol>
</nav>

  </div>

  </div>




  <main id="main-content" class="main-content usa-section has-breadcrumbs">
    <div class="grid-container">
      <div class="grid-row grid-gap">
        <div class="grid-col-12">
            <div class="region region-highlighted">
    <div data-drupal-messages-fallback class="hidden"></div>

  </div>


          
        </div>
      </div>

      <div class="grid-row grid-gap">

                            <aside class="sidenav tablet-lg:grid-col-3">
              <div class="region region-sidebar-first">
    

  
  
          <ul class="usa-sidenav">
    
          <li class="usa-sidenav__item">
        <a href="/research-training/medical-research-initiatives" class="usa-current has-children">
          <span>Medical Research Initiatives</span>
        </a>

                    
  
          <ul class="usa-sidenav__sublist">
    
          <li class="usa-sidenav__item">
        <a href="/research-training/medical-research-initiatives/radx" class=" has-children">
          <span>Rapid Acceleration of Diagnostics (RADx®)</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/regenerative-medicine-innovation-project-rmip" class=" has-children">
          <span>Regenerative Medicine Innovation Project (RMIP)</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://recovercovid.org/" class="">
          <span>RECOVER</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://www.nichd.nih.gov/research/supported/IMPROVE" class="">
          <span>IMPROVE Initiative</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/21st-century-cures-act" class="">
          <span>The 21st Century Cures Act</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://www.cancer.gov/research/key-initiatives/moonshot-cancer-initiative" class="">
          <span>Cancer Moonshot℠</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="http://www.braininitiative.nih.gov/" class="">
          <span>The BRAIN Initiative</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research" class="usa-current has-children">
          <span>Advancing ME/CFS Research</span>
        </a>

                    
  
          <ul class="usa-sidenav__sublist">
    
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research/about-mecfs" class="">
          <span>About ME/CFS</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research/mecfs-research" class="">
          <span>Research</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research/publications" class="">
          <span>Publications</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research/announcements" class="">
          <span>Announcements</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research/events" class="usa-current">
          <span>Events</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research/mecfs-resources" class="">
          <span>Resources</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/advancing-mecfs-research/frequently-asked-questions-about-mecfs-research" class="">
          <span>FAQ</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="mailto:NIHME-CFSWorkingG@ninds.nih.gov" class="">
          <span>Contact</span>
        </a>

        
      </li>
        </ul>
  
        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://allofus.nih.gov/" class="">
          <span>All of Us℠ Research Program</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/echo" class=" has-children">
          <span>Environmental influences on Child Health Outcomes (ECHO) Program</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/amp" class=" has-children">
          <span>Accelerating Medicines Partnership (AMP)</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://grants.nih.gov/policy/reproducibility/index.htm" class="">
          <span>Rigor and Reproducibility</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://datascience.nih.gov/" class="">
          <span>Data Science at NIH</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://heal.nih.gov/" class="">
          <span>NIH HEAL Initiative</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/include-project" class=" has-children">
          <span>INCLUDE Project</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://womeninscience.nih.gov/" class="">
          <span>Women in Biomedical Careers</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="http://commonfund.nih.gov/" class="">
          <span>NIH Common Fund</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="http://neuroscienceblueprint.nih.gov/" class="">
          <span>NIH Blueprint for Neuroscience Research</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/sound-health" class=" has-children">
          <span>Sound Health</span>
        </a>

        
      </li>
        </ul>
  
        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/science-highlights" class=" has-children">
          <span>Science Highlights</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="https://science.education.nih.gov/" class="">
          <span>Science Education</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/research-nih-labs-clinics" class="">
          <span>Research in NIH Labs &amp; Clinics</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/training-opportunities" class="">
          <span>Training Opportunities</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/library-resources" class="">
          <span>Library Resources</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/research-resources" class="">
          <span>Research Resources</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/clinical-research-resources" class="">
          <span>Clinical Research Resources</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/safety-regulation-guidance" class="">
          <span>Safety, Regulation &amp; Guidance</span>
        </a>

        
      </li>
          <li class="usa-sidenav__item">
        <a href="/research-training/lasker-clinical-research-scholars" class=" has-children">
          <span>Lasker Clinical Research Scholars</span>
        </a>

        
      </li>
        </ul>
  



  </div>

          </aside>
          <div class="region-content tablet-lg:grid-col-9 has-sidebar">
          
                                  <div class="grid-row grid-gap">
            <div class="page-content">
                <div class="region region-content">
    <div id="block-nih2-uswds-page-title" class="block block-core block-page-title-block">
  
    
      
  <h1 class="hidden uswds-page-title page-title"><span class="field field--name-title field--type-string field--label-hidden">Metabolism – October 26, 2023</span>
</h1>


  </div>
<div id="block-nih2-uswds-content" class="block block-system block-system-main-block">
  
    
      <article data-history-node-id="2331">

  
    

  <div>

                            
    <div class="nih-body">
      
    </div>
          
      </div>

      <div class="layout layout--onecol">
    <div  class="layout__region layout__region--content">
      <div class="views-element-container block block-views block-views-blockh1-title-block-1">
  
    
      <div><div class="js-view-dom-id-8b16d163a2a9544290fafbb4b7551e4ba438d08e507de3fd4807deb79150310b">
  
  
  

  
  
  

      <div><h1 class="uswds-page-title">Metabolism – October 26, 2023</h1><p class="nih-subtitle">ME/CFS Research Roadmap Webinar Series</p></div>

    

  
  

  
  
</div>
</div>

  </div>
<div class="block block-layout-builder block-field-blocknodepagefield-component">
  
    
      
      <div class="field field--name-field-component field--type-entity-reference-revisions field--label-hidden field__items">
              <div class="field__item">  <div class="paragraph paragraph--type--media paragraph--view-mode--default margin-bottom-3">
    
            <div class=" "><article class="media media--type-remote-video media--view-mode-full-width">
  
      
            <div class="field field--name-field-media-oembed-video field--type-string field--label-hidden field__item"><iframe src="https://www.nih.gov/media/oembed?url=https%3A//www.youtube.com/watch%3Fv%3D1Y1Z5sU-2jo&amp;max_width=0&amp;max_height=0&amp;hash=kolkBXEoSMn8tmIv6YJQXlLOv33k4-JvSVuA9qTtXeI" width="200" height="113" class="media-oembed-content" loading="eager" title="ME/CFS Research Roadmap Webinar - Metabolism"></iframe>
</div>
      
  </article>
</div>

      </div>

    

</div>
              <div class="field__item">  <div class="paragraph paragraph--type--uswds-text paragraph--view-mode--default usa-section usa-content">
    
            <div class="grid-col-fill">
      
      
            <div class="clearfix text-formatted field field--name-field-uswds-body field--type-text-long field--label-hidden field__item"><h2>Transcript</h2>

<p><strong>Vicky Whittemore:&nbsp;</strong>I'm sure we'll have more people joining us later.&nbsp; So, welcome, everyone.&nbsp; This is the third webinar in a series of research webinars to help develop an ME/CFS Research Roadmap Webinar Series.&nbsp; So, I'm Vicky Whittemore.&nbsp; I'm a program director at NINDS at the National Institutes of Health, where I work&nbsp; with Dr. Koroshetz, who's the chair of the Trans-NIH Working Group for ME/CFS.&nbsp; And we're very closely with Joe Breen, my colleague at NIAID to help coordinate ME/CFS activities across NIH.</p>

<p>So, as many of you know, we're in the process of holding eight webinars.&nbsp; And the end result of these webinars will be to identify research priorities that will go together to form a report to the NINDS leadership and NIH for setting research priorities for chronic fatigue syndrome research.&nbsp; So, I just want to acknowledge everyone who's serving on this working group of council.&nbsp; And we've divided the -- all of these individuals up into webinar planning groups.&nbsp; So, the metabolism webinar planning group who organized the webinar today is chaired by Alain Moreau from the University of Montreal.&nbsp; And thanks to all of the individuals on the list here who have contributed to planning this webinar.</p>

<p>I'd also like to acknowledge my colleagues and friends from NIH who have been part of this effort, who worked behind the scenes with me to make these to help plan and organize and put this, the research roadmap process, into place and to make -- work very closely with our great colleagues at RLA, our contractor, who's helping us to coordinate all of these webinars.&nbsp; So, just to mention the upcoming webinars.&nbsp; So, November 1st, it&nbsp; one on genomics/genetic susceptibility.&nbsp; We have one on the 30th on chronic infection.&nbsp; December 8th, on physiology; January 5th, on lesser studied pathologies; and January 11th, on circulation.&nbsp; So, you can go to this link.&nbsp; And we can put that link in the Q&amp;A later for you, so that you can go and register to attend any or all of these upcoming webinars.</p>

<p>So, just some quick guidance for webinar participants.&nbsp; So, the goal of this webinar, as I said, is to identify research priorities.&nbsp; And in this case – oops, I didn't change my slide from last time, not immune system, but metabolism involvement in ME/CFS.&nbsp; So, we're asking, what do we know?&nbsp; What don't we know?&nbsp; And what do we need to know to accelerate research to move to clinical trials?&nbsp; So, what we're really after here is really understanding the state of the science.&nbsp; What is known?&nbsp; What are the big questions that are out there that can really help to push the research forward?</p>

<p>So, the questions for each speaker will be addressed after each presentation.&nbsp; So, put your questions in the Q&amp;A.&nbsp; We're not able to answer questions, any kind of medical health-related questions for your individual health situation.&nbsp; So, please, do not enter those kinds of questions into the Q&amp;A.&nbsp; So, the question should focus on clarifying points from the talk and the presentations and on research priorities and research on ME/CFS.&nbsp; So, as I said, all of the research priorities will be put together to form a report that will be presented in May to the NINDS Advisory Council.</p>

<p>So, ways to give additional feedback, you can send an email to this roadmap email address, mecfsresearchroadmap@ninds.nih.gov.&nbsp; The best way to receive announcements and updates from NIH are to sign up for the NIH ME/CFS listserv, which is listed here.&nbsp; It's www.nih.gov/mecfs.&nbsp; And so, watch for announcements coming out of ways to participate on a crowdsourcing platform called IdeaScale.&nbsp; So, once the research priorities from each webinar are sort of finalized by the webinar planning group, there'll be posted there for additional feedback from the community.&nbsp; And you'll get announcements as soon as those are posted, as well as when the videos, recordings, and transcripts from each of the webinar are posted.&nbsp;</p>

<p>So, with that, I'll pass it over to and introduce Alain Moreau, the chair of the metabolism webinar planning group, to introduce our first speaker.&nbsp; Thank you.</p>

<p><strong>Alain Moreau:</strong>&nbsp;Thank you very much, Vicky.&nbsp; First of all, I would like to thank to all the speaker today to the great contribution, though this is a wonderful agenda that we are proposing regarding the metabolism of ME/CFS.&nbsp; So, our first speaker is Mr. Chris Wikman, who has been ME/CFS patient for 11 years.&nbsp; While his background is in engineering and management, his extensive self education in health and medicine since his diagnosis, and his desire to help out, led him to becoming a part of this NIH working group of council.&nbsp; So, Chris, thank you.</p>

<p><strong>Chris Wikman:</strong>&nbsp;Thank you, Dr. Moreau.&nbsp; Okay.&nbsp; Next chart, please.&nbsp; So, I thought I'd start with what I call morning crashes.&nbsp; My fatigue has been a little bit different in recent years.&nbsp; So, I'm just going to kind of walk you through what I often go through.&nbsp; So, I usually wake up around 06:30 a.m.&nbsp; But I mean, I've never wake up refreshed and was always tired.&nbsp; It's not unusual.&nbsp; I lie around a little bit and get motivated to something I want to do.&nbsp; So, I get up.&nbsp; And I eat breakfast.&nbsp; I maybe watch some news or, you know, take some vitamins.&nbsp; And I have a smoothie/shake I do.</p>

<p>And my wife, she's a morning person, and she's running around getting things done.&nbsp; And I'm kind of sitting there.&nbsp; And she can see it as it comes on.&nbsp; It's usually around -- really curious -- about 90 minutes after I wake up and I've been out there to figure out what's going on.&nbsp; But I start to move less, my eyes lose focus, my head nods down, you know, and my energy is fading.&nbsp; So, I head to the couch.&nbsp; Don't know how deep this crash is going to be.&nbsp; Of course, brain fog sets in.&nbsp; So, I dropped on the couch.&nbsp; Usually, my eyes are closed.</p>

<p>And I developed a strange breathing pattern kind of like a short, quick breath and then just nothing for a bit, and then another short breath.&nbsp; And I assume that's energy saving behavior.&nbsp; And I don't know the name for it.&nbsp; But that's pretty typical.&nbsp; And I'm trying to sit down, but sometimes I have to lie down.&nbsp; And if it's bad enough, I can't even pick up my phone to check on what's going on.&nbsp; Strangely, even with the brain fog, I can hear what's going on.&nbsp; I can kind of have quick responses, but not much more than that.&nbsp; And that maybe after an hour or two, that will pass.</p>

<p>And eventually, my hands start moving a bit.&nbsp; My legs start moving a little bit, not necessarily consciously, but just kind of like a little stretch or something.&nbsp; And that's kind of a sign to me that I know I'm started to come out of it.&nbsp; My breathing pattern recovers.&nbsp; My eyes open.&nbsp; And eventually, I kind of back to functional.&nbsp; So, this is a little bit different from earlier on when there was very clear post-exertional malaise.&nbsp; We'll get back to that.&nbsp; Next chart, please.</p>

<p>So, as Dr. Moreau said, I've had this for about 11 years.&nbsp; I don't know exactly what triggered it.&nbsp; But it seems like it might have been a Lyme disease after a week-long backpacking trip in a local Shenandoah mountains.&nbsp; And that was eventually diagnosed somewhere around 2017.&nbsp; And I was treated for it, but long delay there.&nbsp; A little bit of background you can see up top.&nbsp; So, trying to get a diagnosis, which eventually came in 2015.&nbsp; I probably saw -- well, I know I saw up to 40 doctors, some of whom I still saw.&nbsp; A couple of, you know, side diagnoses along the way.</p>

<p>A single CPET that I talked my pulmonologist in was unremarkable.&nbsp; Went to Mayo Clinic, they didn't find anything.&nbsp; I've learned a lot, done a lot over the years.&nbsp; So, they've been better than before.&nbsp; But still -- well, summer and fall are pretty good.&nbsp; I like the warm weather like the sunshine.&nbsp; Winter and spring, not so good.&nbsp; But when I'm good, I'm really good.&nbsp; I can do some tactical mountain climbing even.&nbsp; So, I feel lucky compared to other patients.&nbsp; But it's been a long road.&nbsp; Next chart, please.</p>

<p>I do what's called self-tracking.&nbsp; I have a couple of 100 pages and worksheets that I can go back and look on things.&nbsp; I became very self-educated, read a lot of articles.&nbsp; The doctors I still see today, they're open to discussion.&nbsp; We learn new things from each other.&nbsp; They let me suggest things.&nbsp; But none of them are ME/CFS specialist.&nbsp; I'm in the D.C. area.&nbsp; But I think what I have is pretty good.&nbsp; A lot of tests on the way.&nbsp; I've been a "resolved," things like heavy metal toxin load, oxidative stress, nutrient deficiencies.&nbsp; I will admit my GI is still a work in progress.&nbsp; Next chart.</p>

<p>I do take around 100 pills and powders each day, typical mitochondrial support, a bunch of semi-stimulants.&nbsp; I've been on acyclovir for 10 years, mainly due to a previous ocular herpetic lesion.&nbsp; That may have something to do.&nbsp; I've been on and stopped LDN, ribose, maca root.&nbsp; But I do take a lot of other things.&nbsp; And I have very good documentation.&nbsp; I look back years and see what I took when.&nbsp; I do make changes quite often.&nbsp; But my number two goal besides resolving this eventually is to reduce the pill burden.&nbsp; But I think what I'm doing seems to work.</p>

<p>I do integrative, you know, therapies like acupuncture and lymphatic massage.&nbsp; I definitely pace myself.&nbsp; I self-tracking with an Oura ring and a WHOOP strap and a Fitbit.&nbsp; And I use those to help gauge, you know, what I'm up for.&nbsp; It is hard to say what single thing or group of things helped most, unfortunately.&nbsp; Next chart.&nbsp; So, kind of wrapping up with the research.&nbsp; I've lost a sibling, a parent, and a grandparent to cancer.&nbsp; Even with that, it still anguishes me over the disparity in funding for cancer research versus ME/CFS.&nbsp; And that's, you know, the society impact.&nbsp; I don't know.&nbsp; It seems pretty big even if you disregard the medical costs spent on cancer treatment.</p>

<p>You know, I definitely am in line with two main priorities, which seem to be out there.&nbsp; Number one, finding, you know, palliative treatment options that have been shown to be effective.&nbsp; And then getting to root cause, disease progression, and a cure someday.&nbsp; And I think that's it.&nbsp; Next chart.&nbsp; So, I thank you all for listening and your caring.&nbsp; And I'm happy to, you know, answer questions or people can contact me.&nbsp; Here's my email.&nbsp; So, thank you, everybody.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Thanks very much, Chris.&nbsp; So, I'll -- there are no questions that have come in yet.&nbsp; But like I said, anyone who wants to ask a question, you can enter it in the Q&amp;A.&nbsp; But you talk about how different things are now from when you first became ill.&nbsp; And do you think it's -- you said -- and you said, you don't know the combination of things.&nbsp; But do you think it's pacing has really helped you, especially with post-exertional malaise?</p>

<p><strong>Chris Wikman:</strong>&nbsp;I think pacing definitely.&nbsp; And using my data readiness, recovery, et cetera, I found that, you know, days where my numbers the next morning were low, where days were I was either too active, or occasionally had some alcohol or stayed up too late or a combination of those.&nbsp; But -- so, I really dialed back.&nbsp; But then I found out that if I really did nothing, it also got bad.&nbsp; So, I found I needed a minimum level of activity as well.&nbsp; And that was a surprise to me.&nbsp; But yes, I definitely paced myself.&nbsp; I try not to schedule any major, you know, trips in the winter and early spring because it's not -- I mean, especially at least the northern hemisphere because I know that'll get me down.&nbsp; But early on, I would do a workout and I couldn't finish it.&nbsp; I would go to bed.&nbsp; The next day, I was in bed.&nbsp; I've seen to have gotten a little past that.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>So, there are now some questions that have come in.&nbsp; Regarding lived experience of ME/CFS patients, many do keep detailed notes.&nbsp; Are there any clearing houses for collating the notes of ME/CFS patients across the country?&nbsp; This would be very valuable form of research.&nbsp; So, I think that's a really important question.&nbsp; And this has come up in other webinars.&nbsp; I know that there is the -- and I -- forgive me, I'm not remembering.&nbsp; The You + ME platform that Solve ME/CFS has -- where individuals can track their symptoms and track the disease over time.</p>

<p>I think they've changed the name and have a different platform for that.&nbsp; But that's one thing you can look at it.&nbsp; So, I think -- you know, I absolutely appreciate how important these experiences are, and everyone's different experiences.&nbsp; So, I think that's something we'll think about how we can maybe better capture those kinds of experiences going forward.&nbsp; So, the -- another question is, There's a large subgroup of patients who feel better in summer, fall and worse winter, spring.&nbsp; Are we -- any researchers interested in studying this phenomenon?&nbsp; I think that is very interesting.</p>

<p><strong>Chris Wikman:</strong>&nbsp;So, there was a paper given at the July conference in New York, IACFS/ME who touched on that a little bit.&nbsp; I don't recall the research scientists name.&nbsp; But that was the first that I've seen.&nbsp; I found that very interesting.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Yeah.&nbsp; That is interesting.&nbsp; I mean, it's interesting.&nbsp; Well, two things, kind of unrelated maybe, but perhaps related.&nbsp; The recent study out showing that individuals with long COVID have decreased levels of circulating serotonin.&nbsp; And, you know, what we know about winter depression and issues, people feeling more tired and depressed in the winter when there's less sunlight.&nbsp; I mean, so those are all things that I think are very important.&nbsp; But as far as I'm aware, have only begun to be looked at in ME/CFS.&nbsp; So, maybe one last question for you, Chris.&nbsp; Can you comment on hypoxia and body temperature correlation with carbon monoxide?</p>

<p><strong>Chris Wikman:</strong>&nbsp;So, this is an interesting area that I'm currently look into a fellow, Mr. Albert Donnay , who's a toxicologist.&nbsp; He was at the conference in New York and introduced some of us to a lot of overlap between carbon monoxide poisoning endogenously.&nbsp; And so, we're currently thinking about working on a paper to look into.&nbsp; I've been doing a lot of self-study, making measurements on -- you can measure your exhaled carbon monoxide, which goes up and down with things that you do.&nbsp; You know, hyperbaric oxygen therapy, for example.</p>

<p>And looking at the close correlation with body temperature, my temperature runs low as do some other ME/CFS patients.&nbsp; So, we're looking into, you know, how closely these overlap.&nbsp; And if they're not the same thing, which I currently am not convinced yet -- but there's definitely a lot of overlap.&nbsp; And so, it's an area of study.&nbsp; And we hope to put forth some paper on this shortly.&nbsp;</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Awesome.&nbsp; Thanks so much, Chris.&nbsp; All right.&nbsp; Over to you, Alain, with the next speaker.&nbsp; Thanks, Chris.</p>

<p><strong>Alain Moreau:</strong>&nbsp;Thank you very much.&nbsp; And thank you, Chris, for your excellent presentation.<br>
<strong>Alain Moreau:&nbsp;</strong>The next speaker is Dr. Shuzhao Li, who is an associate professor investigator at the Jackson Laboratory.&nbsp; Dr. Li is a leading expert in the field of metabolomics for precision medicine, immunometabolomics and multi-omics modeling of the immune system.&nbsp; Dr. Li?</p>

<p><strong>Shuzhao Li:</strong>&nbsp;Thank you, Alain.&nbsp; Also, thanks, Chris, for sharing your story and insights.&nbsp; So, let me start by walking you through some of the parameters here.&nbsp; So, both immunology and the metabolism have been around for over 100 years.&nbsp; But human immunology actually had a more recent report in the past 10 to 15 years.&nbsp; That's because, in the past, there were limited ways to study humans directly.&nbsp; So, with that, it was after genomics and other high super technologies.&nbsp; For the first time, we are able to collect a lot of data from humans directly.</p>

<p>So, the figure on the left side is what we presented in a review article 10 years ago, just to explain the common strategy in this field.&nbsp; For example, from a draw of blood, we can look at what cell population are there, proteins, cytokines, antibody, and T cell profiles.&nbsp; So, we can measure genes by heating RNA in solvents, metabolizing a large number.&nbsp; So, with all these data, then we try to build a predictive models of human immune response.&nbsp; Then the data will also generate a new hypothesis for us to get more mechanics in size.&nbsp; So, on the right side, this is what we call a genome-scale model.&nbsp; Basically, that's the compound knowledge over the past 100 years, or the metabolic pathways.&nbsp; So, we are looking at about 3000 metabolites, 4000 reactions, 2000 genes here, but still far from a complete action.</p>

<p>So, I highlighted the center, metabolomics.&nbsp; So, when I'm starting to feel that the metabolomics were already way behind in genomics, so, basically, in the past 10 years, my lab had a major step in improving metabolomic technology.&nbsp; So, our data are really complex data.&nbsp; So, to get into ME/CFS, I started reading some of the papers.&nbsp; And you probably saw this from a recent review article.&nbsp; Basically, the table on the left is the summary of previous data on immune cells.&nbsp; The next table is on cytokines.&nbsp; You would see the same cells were reported increased in some paper, decreased in other paper.</p>

<p>I know these may be very discouraging to a lot of people.&nbsp; But we actually saw this same story in the early days in immunology.&nbsp; But we're lucky to be part of Human Immune Project Consortium since very beginning.&nbsp; So, I spent a lot of my time to think it through&nbsp; how to interpret the data, especially the big data of genomics, transcriptomics, and metabolomics.&nbsp; How to combine them with the immune parameters?&nbsp; So, many of the problems we see here.&nbsp; From our experience in the Human Immune Project Consortium, we're&nbsp; number one, standardization of assays.&nbsp; Number two -- so, we are looking at the dynamic system.&nbsp; And quite often people are looking at data on a different time.</p>

<p>So, regarding to -- or the -omic technology, let me get back to the last slide.&nbsp; So, some may appear to be expensive in the beginning.&nbsp; But if you think through it, when you have unbiased data that are captured, you know, many things at the same time, they actually helped removed a lot of inconsistency in these sort of studies.&nbsp; So, I'm going to come back to this later.&nbsp; But let me just give you an example of how we conduct our human immunology projects in that context.&nbsp; So, here, we used vaccination as a tool because that allows us to control when the immune response starts.&nbsp; So, you know, once you give people a vaccine shot, you have the baseline.</p>

<p>So, in this study, we collect data, one day, three days, seven days, and maybe it's beyond actually.&nbsp; But the figure on the left, it is showing the result, how we constructed a model -- network model to capture cells, genes, metabolite, cytokines.&nbsp; And put together the major events as a network.&nbsp; So, it's still quite complex.&nbsp; So, on the right side, we are just putting out one piece of data from that complex network too.&nbsp; So, how this works?&nbsp; So, panel C and D from a day three, basically, we see very strong statistical association between a gene network and a metabolite network.&nbsp; So, the bar plot next to D is the list of pathways enriched in data metabolite.&nbsp; So, what's more interesting in this study is that this day three, metabolite network also has a significant association to gene network on a day seven.&nbsp; That's on the right side of panel E.</p>

<p>So, we see, you know, TCA cycles connect to electron transport to mitochondria.&nbsp; And the metabolic pathway in the stood out &nbsp;from cells.&nbsp; So, we see phosphatidylinositol signaling.&nbsp; We see purine metabolism.&nbsp; Actually, they are here in the metabolite about four days ahead of gene expression.&nbsp; So, these are actually one of the first piece of evidence that suggests the immune cells actually received systemic signals from metabolites.&nbsp; So, from there, let me just give a very opinionated summary of where the field stands right now from my experience.</p>

<p>So, the first items are released to explain, you know, when you think about metabolites.&nbsp; So, they are usually thought as components to provide energy or building blocks for biology.&nbsp; Then many metabolites also served as secondary messengers.&nbsp; So, are -- these are really part of the general biology.&nbsp; They are not specific immunology.&nbsp; Then in the past 10, 20 years, we have started to see a lot of evidence where specific events occur in immune cells -- immunological events.&nbsp; For example, you know, our own study.&nbsp; We saw that arginine was depleted in&nbsp; infection, also antigen-presenting cells.&nbsp; So, histamine is the major story in mast cells in allergy.</p>

<p>In the past few years, people also started proposing this idea.&nbsp; Nutritional licensing is actually another layer on top of this traditional two-step activation of T cells.&nbsp; Of course, so many metabolites in our body are from the microbiome.&nbsp; So, a lot of data from recent years, also showed data due to microbial metabolites as educated immune cells.&nbsp; For example, they are instrumental in the development of T cells in the muscosal environment.&nbsp; Although more recent data found quite a few new metabolites to be actually G protein-coupled receptor ligands.&nbsp; So, did I raise good examples of how metabolites contributed to intercellular communications?</p>

<p>The last three items, in my opinion, are rarely understudied.&nbsp; So, the interaction between xenobiotics, dietary ingredients, and a drug with immune cells right now are only emerging.&nbsp; So, we still have a lot to learn.&nbsp; Immunometabolism is really hot right now in the scientific field because, you know, it has huge implications to vaccine development to cut immunotherapies.&nbsp; But mostly, they actually are at a cellular level, so very little data is available at a systemic level.&nbsp; So, I emphasize systemic data because, you know, in human studies, quite often we measure peripheral blood.&nbsp; All those things are systemic.&nbsp; They are not local.&nbsp; So, we also require a lot of development in experimental techniques, in computational method to actually put up a story back together.&nbsp; So, this is my kind of summary of where the field is today.</p>

<p>So -- as I said, so, my lab works are really focusing on metabolomics in the past two years.&nbsp; We moved from Emory University to Jackson Lab Connecticut in 2020.&nbsp; Then the person next door from my office happened to be someone named Derya Unutmaz.&nbsp; So, Derya and I started talking.&nbsp; Then one day he told me, "So, we are starting this disease with a lot of immune profiling and the metabolomic data.&nbsp; Do you want to have a look at the data?"&nbsp; It was not really the focus of my life.&nbsp; So, Minghao asked me how -- a post-doc in my group, who is sitting next to me right now.&nbsp; It turned out that Minghao actually took a personal interest in this.&nbsp; And he dedicated a better of the past two years to analyze this dataset.&nbsp; So, what I'm going to show you in our data analysis were really mostly Minghao's work.</p>

<p>So, we started reading all the metabolomic papers, of course.&nbsp; So, from my understanding, there were a lot of discussion how the different studies agreed or disagreed in the past.&nbsp; So, I did not include any NMR studies here.&nbsp; So, in this day of age, NMR is not relevant here anymore.&nbsp; So, from my personal experience, so, I looked through metabolomic papers.&nbsp; Probably, a few projects actually, were analyzed using some of our computational tools.&nbsp; I was also sitting on the student committee of NIH Metabolomic Consortium made in the past five years.&nbsp; So, they look very familiar to me.&nbsp; So, if I sit down with people to go through all the studies, I bet all the problems can be explained by effect size, sample size, the coverage.&nbsp; The bias of technologies.&nbsp; And how the scientific methods were through then applied.</p>

<p>So, I'm going to leave it here.&nbsp; So, actually, I'm going to have to come back to compare some of our results against earlier studies.&nbsp; We are working with from Bateman Horne Center in Utah.&nbsp; So, there is live, you know, are the immune profiling.&nbsp; Then metabolomics data were actually generated by metabolome commercial vendor.&nbsp; So, in the clinic, they did a bunch of questionnaires.&nbsp; Also, the patients were subjected to a bunch of clinical laboratory tests from a commercial vendor.&nbsp; So, Dr. Vernon actually helped us a lot to organize the clinical data and make them useful.&nbsp; So, we really appreciate it.</p>

<p>So, the important aspect in our study is really that we included 78 patients that had ME/CFS for over 10 years.&nbsp; So, that is quite different from many earlier cohorts.&nbsp; Then for people under 4 years of the disease, we have 75 people.&nbsp; Then we had 96 healthy controls.&nbsp; So, we checked the participant over&nbsp; with this, so that -- in three years.&nbsp; So, these are what do we call -- plots, actually I think how we merged the slides it didn’t show very well.&nbsp; So, the idea is that these are data from flow cytometry to profile individual immune cell populations.&nbsp; So, I'm assuming Venn diagram to the upper right side.&nbsp; Basically, from each sample from each participant, we use flow cytometry to analyze how many cells are present carrying certain molecular markers.</p>

<p>Then for each marker, we do statistical analysis like those on the right side.&nbsp; So, we are comparing the three groups.&nbsp; You start on the right side in one participant.&nbsp; So, then we summarize each box plot here into one dot.&nbsp; We kind of pause to the left.&nbsp; We can already see on the slides.&nbsp; But the take home message here is really that, firstly, quite a few cell populations are significant between groups, for example, the two to the right side.&nbsp; So, some are decreased in the patients, some are increased.&nbsp; Then in the patient with longer-term disease, we actually have a lot of more significant cell populations that are different from controls.</p>

<p>In the metabolomic data, so we summarized the first wave in this heat map.&nbsp; Here, each column is one participant, then each row is a metabolite.&nbsp; So, the intensity here basically indicates the level of metabolites in each individual.&nbsp; So, you can see right away that a lot of lipids are much higher in patients of over 10 years disease duration and have significant features here.&nbsp; So, surprisingly, there are large number of xenobiotics that are really different between healthy control and patients.&nbsp; So, overall, I will see the signals are much stronger in the long-term patients. &nbsp;So, there is difference between short-term patient versus healthy controls, but not as strong as long-term patients in this particular dataset.</p>

<p>So, to show you some examples of whether metabolites are actually different among the groups.&nbsp; So, I'm picking three classic examples.&nbsp; So, to the upper left -- so we are looking at cholesterols here.&nbsp; Here are a couple of examples for carnitines.&nbsp; Then the bottom three are sphingolipids.&nbsp; So -- again, so the cholesterol data are quite consistent with what do we see in the clinical labs.&nbsp; Some metabolites are increased, some are decreased, which I noticed on this slide.&nbsp; But I'm going to actually explain to you some of the details on the sphingolipids on the next slide to account for some of the controversies we had in this field in the early days.</p>

<p>So, again -- so, these are the same three sphingolipids here.&nbsp; So, these age groups within one day.&nbsp; So, after one year -- so, we asked the same people.&nbsp; So, you guys should say we are missing the individual dots here.&nbsp; But the overall trend is actually very consistent with Visit 1.&nbsp; So, some don’t really make the cut off, or the threshold. So, that happens when you have the effect somewhat dynamic, or on the margin.&nbsp; So, but again -- so, the trend is consistent.</p>

<p>So, in the top panels, we actually regularized the data by taking out the age and sex effect.&nbsp; So, if we don't do that, each part of the low intensities on the bottom.&nbsp; Again, it's the same trend.&nbsp; So, once in a different in the single modeling, the third one here in the short-term patients.&nbsp; So, after removing age and sex from the data, so there is slight decrease.&nbsp; But in the raw signal data, there is a slight increase.&nbsp;</p>

<p>So, again, choosing different cycle method may actually include a bit on how you interpret data.&nbsp; But as long as the method is consistent -- so the readout is actually robust enough to show the same trend even one year after the first wave.&nbsp; So, of course -- so, we want to be able to put the data together to understand the whole picture.&nbsp; So, our analysis, actually employed three different strategies.&nbsp; The first one is multi-omics factorial analysis.&nbsp; So, that is also called somewhat latent factorial analysis by other people.&nbsp; The second method is to look for individual association between metabolites and immune cells.&nbsp; The third strategy is to use some multiscale modeling across data.&nbsp; We applied for quite a few of our previous studies.</p>

<p>So, let's jump into the multi-omics factor analysis.&nbsp; So, the idea here is that the data from immune profiles are high dimensional in the sense we have 311 subpopulations.&nbsp; So, we have over 200 samples from Visit 1 and 160 samples in Visit 2.&nbsp; Metabolomic data, we have over 1000 features.&nbsp; So, the same subjects.&nbsp; So, these are complex data that are often carried many different influence in how the variants is incorporating the data.&nbsp; So, by treating them as master data, we want to dissect that into different factors.&nbsp; So, in this particular approach, we want to gather.&nbsp; The factors are consistent among different data types.&nbsp; Here, we are the treated the two visits as different data types, because logistically speaking, sample were collected differently and analyzed differently.&nbsp; So, we wanted to see data would have tell us a consistent story.</p>

<p>So, actually, in the result -- so, in the top panel data are the top 20 factors.&nbsp; We identified in two visits in the immune profiling data.&nbsp; Two visits of the metabolomic data, we can't within the same data type.&nbsp; So, they're actually quite consistent.&nbsp; So, here we align from the bottom are the demographic data, health scores that's from questionnaires, then clinical lab tests.&nbsp; So, the first two factors are dominated by the immune data.&nbsp; So, there are a clear involvement in age and sex in the first two factors.&nbsp; So, I'm highlighting factor five and a six with which are the top factors in metabolic data here.&nbsp; So, again, so there is a effect from age and sex, that's why we try to remove that effect.&nbsp; And in some of the static analysis, interestingly, you can start to see that there are inverse correlations with health scores.&nbsp; There's positive correlation with the weights, cholesterol, triglycerides, and the white blood cells would probably be considered some sort of inflammation.</p>

<p>So, if we look at this particular factor closely.&nbsp; So -- again, so, the factors have the ranking order of our about 800 metabolites.&nbsp; We filter out some metabolites that are missing a lot of good samples.&nbsp; So, the top ones are the lipase.&nbsp; Again, they were associated with the lipid data in the clinical test as well.&nbsp; So, I'm showing a few examples here.&nbsp; So, the -- quite a few of them are significant different between healthy controls.&nbsp; And the patients here -- the patient uploaded in red.&nbsp; So, these 4-hydroxyglutamate in metabolite data had increased, which was also reported increased in the 2020 paper from Maureen Hanson’s lab.&nbsp; So, p-cresol glucuronide is actually uremic toxin.&nbsp; So, we also see a few other toxins in the data.&nbsp; So, that might affect some sort of function related to kidneys in patients.</p>

<p>Factor six, so, if we have another look at the factor five to six, they appear to go in two different directions.&nbsp; They are capturing different aspects of metabolomic data.&nbsp; So, not surprising that the lipase actually are clustered at the left end of this factor.&nbsp; But on the top side, a lot of things actually are in the [unintelligible] category.&nbsp; Again, so many are significantly different between patients and controls.&nbsp; So, if we match this factor more closely to the dietary questionnaire so -- again, so factor six is only one that it's very high in matching the questionnaire results.&nbsp; So, then to the right side of data are the same metabolites I showed on the last slide.&nbsp; But these are the top.&nbsp; These are the bottom.&nbsp; The top white arm was more associated with intake of fruit or vegetables.&nbsp; And the bottom part is more affiliated with intake of meat and sugar.&nbsp; So -- again, so metabolomics is really a complex type of data.&nbsp; So, it captures a lot of information.&nbsp; So, we are looking at them right now.</p>

<p>So, I also highlighted the Factor 11 on this particular figure.&nbsp; So, that's because Factor 11 are immune and is only visit one.&nbsp; So, there is no correlation to any of the clinical lab tests.&nbsp; But there is clear negative correlation with the health scores.&nbsp; So, actually, a large number of CD4+, CD45RO+ cells are concentrated to the left side of the factor.&nbsp; So, again, the data flow cytometry data.&nbsp; So, we actually on a few examples.&nbsp; Just that type of box parameters to show you in individuals against a green are health controls, red, our patients.&nbsp; So, these cell populations turned out to be decreased in the patients.&nbsp;</p>

<p>So, from there, we started looking at how individual metabolites compared to immune cells?&nbsp; I'm going to show you one example.&nbsp; Also, this is a sphingomyelin, which we saw earlier on a slide.&nbsp; So, then on x-axis, so we are looking at a particular cell population from flow cytometry.&nbsp; So, there is a clear correlation that across the board, regardless of patients or healthy controls.&nbsp; So, these are the one pair of correlation analyses.&nbsp; Remember, we had 300 cell populations, 800 to 1000 metabolites.&nbsp; We basically computed the correlation pairwise for all of them.</p>

<p>Here is a summary of the result.&nbsp; So, the block to the center is data from patients.&nbsp; The block on the right side is data from the healthy control group.&nbsp; As work large heat map each pixel on this heat map, would it be the correlated readout from something like this.&nbsp; So, basically, we summarize are the correlations of our pairwise features.&nbsp; So, right away, you can see that the patients have many more significant correlations compared to healthy controls.&nbsp; And that they are in different patterns.&nbsp; So, the highlight box here -- so, there are red ones are from lysophoshpholipids.&nbsp; So, they are associated with particular group of immune cells, mostly CD4 T cells.&nbsp; Then if we look at the same metabolite in the healthy controls, so you've seen very sparse signals there.&nbsp; So, this is kind of striking all over you, you know.&nbsp; How this connection between immune cells and metabolites are presumably dysregulated in patients compared with controls.</p>

<p>So, if we do a filtering of this figure, so we get the top significant correlations between metabolites to the left here.&nbsp; Immune cell to the right here.&nbsp; So, red ones are positive correlations, blue ones are negative correlations.&nbsp; With a same cut off, you can see that we have actually a few significant correlations in the healthy control.&nbsp; But we get a lot more in the patient here.&nbsp; But we've had look in individual correlation more closely.&nbsp; So, we are going to describe in the upcoming publication here.&nbsp; So, let me just go through a few examples.&nbsp; So, again, so in this particular CD4 population, so positive correlation in the patients, almost nothing in the healthy controls.&nbsp; In this one, we have no correlation in the patient plot, correlation in the healthy controls.&nbsp; So, these will give us some truth to start using our specific mechanisms underlying the systemic data.</p>

<p>So, this is the third approach I mentioned.&nbsp; So, in the past -- so, we employed this type of multiscale models to test high -level connection between different data types, because, you know, you may have noticed some cell populations actually have similar profiles.&nbsp; Some metabolized have similar profiles.&nbsp; What we're looking here is at the center, the six major communities we detect in the cell populations.&nbsp; To show you my example to the left here.&nbsp; Basically, community number nine, have almost exclusively the red ones are indicating with our data from gamma-positive cells and that they are off from negative cells.&nbsp; So, do you have similar profile in the data.&nbsp; That's why we choose them as group, similar for other cell population here.&nbsp; Then in the metabolomic data, we can detect a lot of metabolites that have similar profiles.&nbsp; So, we can group them into different red boxes here.&nbsp; So -- and these are community level.&nbsp; Can start testing which ones actually have significantly correlation among themselves using a multimodal method.</p>

<p>So, the red ones are establishing significant associations between, again, red metabolite communities, and the circles of self-communities.&nbsp; So, I don't have time to get into the details here.&nbsp; So, this allows us actually to reconstruct the network model to accommodate some of the nonlinear relationships.&nbsp; For example, if you look at this sphingomyelin community here, it actually further connects to some of the neurocognitive scores.&nbsp; There is a positive correlation.&nbsp; So, from here, we can start thinking whether this cell community have something to do with it?&nbsp; So, I'm going to stop here for the data analysis.&nbsp; So, overall, in the JAX cohort study, the signals from the long-term patients are pretty clear.&nbsp; So, in both immune profiles and in metabolomics, we have a significant number of robust features.</p>

<p>So, our metabolomic data have some agreement with the recent two papers from Hanson Lab and from the Columbia group.&nbsp; So, see there are mostly on sphingomyelins, ceramides, cholesterols, and also hydroxyglutamate there.&nbsp; We have significant differences in this cohort in Th17 cells, and a lot of interferon gamma producing cells&nbsp; are increased in patients.&nbsp; We have full robust correlations between immune cells and metabolites and the patients and controls.&nbsp; So, very different patterns in their correlations.&nbsp; So, the multi-omics factor analysis really allows us to link,&nbsp; these are high-dimensional data to the clinical symptoms and laboratory tests.&nbsp; So, the multiscale model I showed on the last slide is really kind of a step we want to build upon in the future.&nbsp; Because if we think about it, these are really complex human data.&nbsp; So, a lot of things are happening in different compartments.&nbsp; Basically, we have to be able to model them in a more intelligent way.</p>

<p>So, the last thing I want to highlight here is that it was really good.&nbsp; At least the two of&nbsp; -- we just started -- actually, three.&nbsp; So, two from Hanson lab use the same platform from a commercial vendor.&nbsp; But even with about 1000 metabolites -- so, the analysis of metabolome, or what I called biochemistry, had a huge gap.&nbsp; I showed you data of the metabolic chart in the beginning.&nbsp; So, if we take all the metabolome in the chart and the plot on this chemical property map -- so these are the black ones.&nbsp; They overlaid on top.&nbsp; The red ones are from the metabolome data.&nbsp; You can see that the overlap here is still quite limited.&nbsp; So, only a quarter of this map is covered by the metabolome data.</p>

<p>Actually, if we compare this map further against human metabolome database, the map only covers a small part of the metabolome database.&nbsp; So, there is actually a large chunk of data still missing in what do we measure.&nbsp; So, quick summary.&nbsp; So, what do we know?&nbsp; What do we don't know?&nbsp; So, the long-term patient starts -- so, were robust pattern in the data.&nbsp; So, the immune profile and metabolites do correlate with the clinical observations.&nbsp; So, the dysregulation of immune profiles and the metabolomics involved metabolites from microbiome, which I did not get into by the two recent papers from Jackson Lab and from Columbia were published.&nbsp; So, I think that will impact what we talk about a lot&nbsp; today.</p>

<p>A lot xenobiotics showed up in the cohort.&nbsp; We say you need to form many cohorts that I have looked at in the past.&nbsp; So, what we don't know?&nbsp; We are missing a large chunk of the biochemical data in metabolomic basically.&nbsp; I also highlight exposome because exposome&nbsp; is actually the right term for the xenobiotics.&nbsp; We don't yet know if there is a distinct subtype of disease.&nbsp; So, the metabolomic heat map I showed earlier.&nbsp; So, there is some pattern.&nbsp; But we don't know for sure yet.&nbsp; So, we don't have good pattern of causality versus etiology of the disease.&nbsp; I'm not going to read through all those things.&nbsp; I'm sure we will discuss some of the items later with the NIH people.</p>

<p>So, for priorities, I think, I don't know if you're agree with me on it or not.&nbsp; So, I feel that this field is really missing state-of-art systems immunology.&nbsp; So, transcriptomic data from the blood, whether it's a bulk data or single-cell data, can inform a lot more things than whatever you have seen in this field.&nbsp; So, of course, this will require a lot of computation development to understand the complex data.&nbsp; So, yeah, I'm going to just skip the items.&nbsp; So, I think I can stop here.&nbsp; But I have to thank the teams.&nbsp; So, Minghao here did all the hard work analyzing data.&nbsp; Derya's lab did all the immune profiling.&nbsp; So, Dr. Vernon and Dr. Bateman have been instrumental both for this cohort and for the field.&nbsp; With that, we appreciate the funding from NIH.&nbsp; I think I'm going to stop here.&nbsp; I'm going to take any questions people may have.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Thank you very much, Dr. Li.&nbsp; Impressive amount of data that I know there's a huge team behind both at Bateman Horne and Jackson Labs that's contributed to all of that, as well as the patients.&nbsp; So, one of the questions that came in to me was about whether location makes you think makes a difference in an individual's metabolomic profile and immune profile?&nbsp; So, you know, if -- I know some of the individuals that were recruited to this study in Salt Lake City are not from Salt Lake City.&nbsp; But do you think location and altitude and that sort of thing -- does impact individual profiles?</p>

<p><strong>Shuzhao Li:</strong>&nbsp;So, metabolomics is actually more than metabolism.&nbsp; So, the data capture a lot of biochemistry of us as human beings.&nbsp; So, when I showed the factor analysis, each factor actually only explains 5, 10 percent of what's going on in the data.&nbsp; So, other parts of the &nbsp;data would actually reflect where people are from.&nbsp; So, we had a very large study with the DoD a few years ago.&nbsp; With that study, we were almost able to pin down the tape code from the data.</p>

<p>That because when you leave here, you know, you get exposed to the environment and different things.&nbsp; So, the information there, but that does not prevent us from understanding data allowing you do data analysis critically.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Great.&nbsp; Thank you.&nbsp; I will ask more questions.&nbsp; But if anyone on the panel wants to raise their hand and ask a question, please feel free to.&nbsp; So, another question.&nbsp; "With regards to blood flow, platelet activation, lymphocyte, and macrophage abnormalities, abnormal levels of hormones in patients with ME/CFS, has anyone ever consulted with specialist researchers in the field of renin-angiotensin-aldosterone system, being this system can affect all of the above?&nbsp; And how does that relate to impact on metabolomics?"</p>

<p><strong>Shuzhao Li:</strong>&nbsp;I'm not familiar with this particular field.&nbsp; But that's the very reason I mentioned that we needed to have a state-of-art system immunology here.&nbsp; Because with good sequencing data from blood cells, we can very much tell what are the events from the data.&nbsp; So --</p>

<p>Which, you know, flow cytometry is great about it.&nbsp; So, today, I think we can do much better.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;So, question is are increased cholesterol numbers, or changes in all of the lipids that you are showing due to impaired lipolysis, or what do you speculate is causing those changes in individuals with ME/CFS?</p>

<p><strong>Shuzhao Li:</strong>&nbsp;So, there is very clear statistical association.&nbsp; but I always hesitate to draw any conclusion on causality.&nbsp; So, we need a design, particular experiments to test those things.&nbsp; So, far, it gives us very good clue where to look at it.&nbsp; But, you know, it's not a final conclusion.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Okay.&nbsp; Thank you.&nbsp; And so, I was noticing this as well.&nbsp; And there's a question here about the scatter in the data.&nbsp; So, if you look, there was one to me, though I don't remember what it was now.&nbsp; That there was a cluster of individuals with ME/CFS that were at the very bottom of that cluster of that chart.&nbsp; So, is the data suggesting that we need to be moving toward precision, or individualized analysis and thinking about individualized treatments based on each individual's profile?</p>

<p><strong>Shuzhao Li:</strong>&nbsp;Well, I’m not going to give a long answer here.&nbsp; So, when I showed the table of metabolomics studies, especially from the early years, so, you know, some cohort were like 20 people, 30 people.&nbsp; So, metabolites actually have very diverse events among general population.&nbsp; So, in general, if we take a healthy group of people -- 20, 30 people actually do not represent the average metabolic profile in this country.&nbsp; So, you need more&nbsp; just kind of good read of what's happening in the baseline population.&nbsp; So, you know, if you take a smaller cohort, then you may be biased, you know, by the particular phenotypes, somewhere here, somewhere there.&nbsp; Then coming back to, you know, if we can look at a small cohort or individuals, actually, we can.&nbsp; So, either we're talking about, you know, how to get a detailed monitoring phenotype, either from metabolomics or from fixed immunology sequencing data.</p>

<p>So, it's really the balance.&nbsp; How much information we know about the individuals?&nbsp; So, if we know enough information, we can actually decide the subtypes.&nbsp; You know, we can focus on the subgroup of patients in the future.&nbsp; Individual patient, if possible, you know.&nbsp; They're having a lot of inspiring studies to look at individual person.&nbsp; But over time to collect large data points, so that we can actually establish the basic level to actually get the right signals, if possible.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Thanks.&nbsp; Derya, you have a question or comment?</p>

<p><strong>Derya Unutmaz:&nbsp;</strong>I just wanted to comment, because this is a very important point.&nbsp; I think, you know, metabolomics is particularly one of the best ways to personalize treatments, in my opinion.&nbsp; I think we -- once we measure -- and as Li mentioned, measures, we link that to the pathology, whether deficiency or increases.&nbsp; You know, just like you can measure your vitamin D levels.&nbsp; And then if it's below a certain level, you give vitamin D to that person.&nbsp; You don't give it to everybody.&nbsp; So, I think it's very exciting, especially once we make these connections between the immunity and the microbiome.&nbsp; We identify which metabolites and how we can replace them.&nbsp; That's another point, whether by changing diet or microbiome or some other life changes or, you know, therapeutic intervention.&nbsp; So, just wanted to add that because it's very important.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;All right.&nbsp; Thank you very much.&nbsp; I think we're at time.&nbsp; So, thank you very much, Dr. Li and Derya for all of that work.&nbsp; And Cindy and Suzanne at Bateman Horne and your team as well.&nbsp; All right.&nbsp; Over to you, Alain.</p>

<p><strong>Alain Moreau:</strong>&nbsp;Thank you very much, Vicky.&nbsp; Thank you, Dr. Li, for your excellent presentation.<strong>Alain Moreau:</strong>&nbsp;So, our next speaker, speaker number three, is the next generation of ME/CFS, if I may say.&nbsp; Okay.&nbsp; And this is very important that we introduce to this audience Dr. Jessica Maya, who is a post-doctoral fellow in Dr. Maureen Hanson’s Lab at Cornell University.&nbsp; And she just completed her PhD and held at Alfred P. Sloan Fellowship.&nbsp; And she investigated immune cell exhaustion, dysfunction, and metabolism in ME/CFS.&nbsp; She will continue her stellar progression in the field of ME/CFS as post-doctoral fellow at the CDC.&nbsp; So, thank you very much, Jessica, for your contribution and the podium is yours.</p>

<p><strong>Jessica Maya:</strong>&nbsp;Thank you, Dr. Moreau.&nbsp; And thank you for that intro.&nbsp; I really want to thank you for inviting me to talk on immunometabolism in ME/CFS.&nbsp; And so, I'll get started.&nbsp; And so, I'll begin with this overview of the immune system starting with your blood, which contains white blood cells composed of granulocytes and PBMCs, or peripheral blood mononuclear cells.&nbsp; Now, these single round nucleus-containing cells include monocytes, dendritic cells, as well as various lymphocytes, such as T cells, natural killer cells or NK cells, and B cells that work to signal and kill infected cells.&nbsp; So, each cell type serves this unique role and can adapt during immune challenges.&nbsp; For example, CD8 T cells transitioning from naive to effector states and then differentiating to memory cells after infections pass.</p>

<p>So, I'd also like to emphasize the significance of examining individual cell types to identify abnormalities in ME/CFS since many ME/CFS studies have investigated PBMCs as a whole.&nbsp; But as you can see, this is a very heterogeneous population that can exhibit distinct metabolic profiles in different stages of their differentiation.&nbsp; So, the regulation of metabolism is really key for cellular function.&nbsp; An immune cell metabolism needs to adjust itself during immune challenges in order to work properly.&nbsp; So, at rest, oxidative phosphorylation is the primary way to make ATP in the cell.&nbsp; Once the body encounters some immune challenge, like a virus, the cells will begin to proliferate.&nbsp; And your immune cells will reprogram their metabolism in order to activate and differentiate into effector cells.</p>

<p>And so, to deal with this metabolically restrictive environment, glycolysis will get upregulated for ATP production, and upregulate oxidative phosphorylation as well.&nbsp; And this is because the intermediates from glycolysis, as well as the citric acid cycle, will be used to help synthetic pathways build biomass and allow for further proliferation of these important cells that are fighting against the immune challenge.&nbsp; And so, after the infection has been cleared, like I mentioned, memory cells that are specific to the infection stick around, and metabolically fatty acid oxidation is upregulated.&nbsp; And this is to support the longevity and the potential reactivation of these cells.&nbsp; And so, all of this reprogramming depends on things like cytokines, nutrients, oxygen, and other factors.&nbsp; So, then changes in the availability of these factors could also really be telling of a dysfunctional immune cell state because, as I said, metabolism ultimately dictates the function of these cells.</p>

<p>And so, other considerations are the discrepancies between cell types and the metabolic pathways that are used.&nbsp; So, for example, in T cells, one-carbon metabolism, as well as glutaminolysis, are used in order to activate the cells.&nbsp; Now, glutaminolysis is seen again in B cells to help B cells differentiate.&nbsp; But these two pathways haven't really been seen as necessary in natural killer cell activation.&nbsp; Another example is the effector cells within CD4 T cells.&nbsp; So, while CD4 T cells can differentiate into a helper T cell, which can produce cytokines, they can also differentiate into regulatory T cells that work to suppress and balance out the immune response.&nbsp; And their development and function will rely on either glycolysis or fatty acid oxidation.&nbsp; But overall, a basic rule has kind of become apparent, where quiescent or anti-inflammatory cells will rely more on fatty acid oxidation, whereas the activated or more inflammatory cell types primarily rely on glycolysis.&nbsp; And this gets upregulated by the mTOR signaling -- stress signaling pathway.</p>

<p>Now, as Dr. Li mentioned in the last talk, there have been a lot of studies that potentially have some inconsistencies between each other.&nbsp; And so, this is just showing ME/CFS studies that have examined total PBMCs.&nbsp; And they do reveal variations in energy pathways.&nbsp; Some report lower levels in ME/CFS seen in the first column, while others show higher levels seen in the green, such as enzymes that are related to the mitochondrial complex or fatty acid oxidation.&nbsp; And lastly, some studies find no differences between ME/CFS and healthy controls.&nbsp; However, like I said, you can see there are inconsistencies in some cases.&nbsp; For instance, one study indicates lower glycolysis rates in ME/CFS, while another demonstrates higher non-mitochondrial ATP production.&nbsp; This is typically linked to glycolysis.&nbsp; And this was compared to healthy controls.</p>

<p>So, similarly, variations exist in assessing mitochondrial mass, which some studies report lower levels and others detected no change in ME/CFS PBMCs.&nbsp; And these inconsistencies could arise from many things.&nbsp; For starters, the heterogeneity of the cell populations, like I mentioned, as well as just limited statistical power due to small sample sizes.&nbsp; And so, our lab is focused on studying the metabolism of isolated immune cells, particularly CD8+ T cells, CD4+ T cells, along with natural killer cells.&nbsp; Now, in T cells, we generally observe this hypometabolic state, whereas NK cells, we see less differences compared to healthy controls.&nbsp; But another group did find that natural killer cells exhibited a reduced glycolytic reserve.&nbsp; And so, this suggests potential limitations in responding to energy demanding situations in ME/CFS compared to healthy counterparts.&nbsp; Additionally, in NK cells, there is this increase in fatty acid oxidation, but a decrease in fatty acid uptake in NK cells.</p>

<p>And so, these findings allow us to better understand cell function.&nbsp; And it's really essential to consider both cell metabolic profiles, as well as the extracellular environment.&nbsp; And so, what I mean by this, for example, is, in our 2020 study, we found this correlation between higher IL-10 levels in ME/CFS plasma correlating with lower glycolytic rates.&nbsp; And this further supported this concept of a dysregulated metabolic dynamic in ME/CFS.&nbsp; And so, all of this that's highlighted in yellow has also been seen in exhausted CD8 T cell states.&nbsp; And so, exhaustion is where you have T cells that have decreased proliferation, cytotoxicity, and survival because of a chronic antigen stimulation.&nbsp; And this suggests that T cells may be exhausted in ME/CFS.</p>

<p>And so, regarding other cell types, there's a group in Australia that's conducted studies using immortalized cultured lymphoblastoid cell lines.&nbsp; They achieved this by infecting PBMCs with Epstein Barr Virus, which primarily is targeting B cells and transforming them into EBV-bearing B cell lines.&nbsp; Now, this model is really cool.&nbsp; It allows for extensive testing in high cell numbers.&nbsp; And their findings suggest inefficient ATP synthesis in the mitochondria of individuals with ME/CFS.&nbsp; And they also found increased capacity for fatty acid oxidation, glutaminolysis, and the pentose phosphate pathway.&nbsp; And so, although these results offer potential mechanisms to investigate, it's essential to remember that these cell lines may not precisely represent B cells derived directly from ME/CFS patients at the time of sample collection.</p>

<p>Dendritic cells have really seen limited metabolic analyses in ME/CFS.&nbsp; And the same holds true for granulocytes.&nbsp; And these are comprised of neutrophils, basophils, and eosinophils.&nbsp; Nevertheless, just as metabolism can inform function, examining cell function can also provide insights into their metabolic status.&nbsp; For instance, studies on neutrophils have shown increased apoptosis in ME/CFS.&nbsp; And this indicates a potential hypometabolic state in these cells as well.&nbsp; And so, then monocytes have also been recently a subject of study in ME/CFS, notably by the Grimson Lab in Cornell.&nbsp; And they employed single cell transcriptomics in a 2022 preprints.&nbsp; And although the paper primarily explores differentially expressed genes related to cell dysregulation, it offers some insights into ME/CFS metabolism.</p>

<p>Predictions suggesting that ME/CFS monocytes contribute to additional overall information exchange between cells, potentially due to this prolonged exposure to an inflamed environment in patients, which influences cellular metabolism and function since extracellular environments can play major roles in shaping cell function.&nbsp; And so, for example, studies that have delved into plasma metabolomics in ME/CFS, as we saw in the last talk, consistently reveal altered lipid profiles.&nbsp; These changes include increased carnitine compounds, which is an important component of fatty acid oxidation.&nbsp; And this aligns kind of with our findings in isolated ME/CFS immune cells.&nbsp; Another consistent observation in ME/CFS is this reduced natural killer cell cytotoxicity.&nbsp; And this phenomenon could be attributed to this altered extracellular environment since it's been seen in other studies that excessive lipid accumulation outside of the cells has been found to impede the cytotoxic machinery that's critical for NK cell function.</p>

<p>Moreover, many -omics papers actually provide the total data of the extensive metabolite measurements in their supplemental information sections.&nbsp; These studies often encompass hundreds or thousands of parameters, as you saw.&nbsp; But they typically present only, you know, the significant findings limiting the display of many metabolites, even though they're also investigated.&nbsp; But the data files and supplemental information allow for further analysis by researchers like us if we're interested.&nbsp; And so, for instance -- I think, actually, Vicky mentioned this just in the past hour -- that a recent study reported reduced serotonin levels in long COVID.&nbsp; But meanwhile, in a recent plasma metabolomic study of ME/CFS from our lab, we also obtained serotonin data, which was not discussed in the main text of the manuscript because of its lack of significance.</p>

<p>And so, we demonstrated no notable differences in serotonin levels between ME/CFS patients and healthy controls.&nbsp; So, healthy controls in red and patients in blue.&nbsp; And we even account for sex differences, which is an important consideration.&nbsp; That's already been discussed a lot already in other ME/CFS webinars and talks.&nbsp; Now, in investigating immune cell metabolism in ME/CFS, common methods include, first, cell isolation techniques, like magnetic bead sorting, for capturing those specific immune cell types like natural killer cells and T cells.&nbsp; And so, this process involves labeling cell surface markers with magnetic beads, and then using a magnet to separate positively or negatively tagged cells.</p>

<p>Another approach is flow cytometric sorting, which relies on applying a charge to individual cells based on the presence of specific fluorophores marking particular cell populations.&nbsp; However, it's essential to realize that flow cytometry sorting may have the potential to impact cell states, potentially activating pathways like p38 MAPK, or altering the redox states and affecting the cellular metabolome. &nbsp;And so, then a valuable method for assessing life cell metabolic rates is by using the machine, the Agilent Seahorse Technology.&nbsp; And so, this machine measures proton and oxygen production in target cells and provides insights into the glycolytic function, mitochondrial respiration.&nbsp; And you can even look at things like fatty acid oxidation in this as well.&nbsp; And so, one notable feature of this kind of technique is its ability to inject metabolic pathway stimulators and inhibitors sequentially during the assay, which offers a really detailed view of some of their metabolism.&nbsp; And so, for example, after utilizing a glycolytic rate assay, our lab was able to detect the reduced basal glycolytic levels in CDA8+ T cells through this observed proton efflux rate.</p>

<p>Now, to assess mitochondrial mass and membrane potential, markers like MitoTrackers Green and Red can be used.&nbsp; And so, these are markers that allow visualization of mass and membrane potential through microscopy, as well as quantification of these parameters using flow cytometry, which uses fluidics to look at each cell, represented by a dot in this graph, for the presence and the abundance of these markers.&nbsp; And so, for example, in the case of ME/CFS CD8 T cells, our measurements indicated a reduction in mitochondrial membrane potential using these tools.&nbsp; And so, analyzing stress signaling pathways like mTOR, which is linked to energy consumption, and AMPK which is associated with energy conservation and growth control, can also provide insights into ME/CFS immune cell metabolism.&nbsp; For example, in the lymphoblastoid cell line study, the Australian team employed a specific kit to gauge mTOR activity through analyzing levels of a phosphorylated amino acid in an -- intermediate of this pathway.&nbsp; And this shed lights on its dynamics within ME/CFS immune cells.</p>

<p>They also utilized whole cell proteomics to investigate specifically metabolic proteins for up or down regulation in ME/CFS.&nbsp; And then they also use Western blotting to validate these proteomic findings with the focus on proteins linked to metabolism, like this example from a separate paper.&nbsp; Now, the single cell transcriptomics performed in the preprint from the Grimson Lab at Cornell focused mainly on monocytes.&nbsp; But it does offer really valuable insights into PBMCs in ME/CFS.&nbsp; Because while monocytes exhibited the highest number of these differentially expressed genes in ME/CFS, you also see that here we see T and K and B cell populations that can also have differentially expressed genes.&nbsp; But they've just yet to be analyzed in depth to see their differentially expressed genes compared to healthy controls.</p>

<p>Now, this approach can give us metabolic data such as this, which is an example from this intratumoral immune cell study.&nbsp; And it's revealing variations in metabolic gene expression among different cell populations.&nbsp; And furthermore, bulk techniques like traditional RNA sequencing can also uncover metabolic dysregulations in isolated ME/CFS immune cells with a bit more sensitivity.&nbsp; So, for example, some unpublished data from a collaborator, David Iu, at Cornell noted decreases in hexokinase, as well as increases in glucose transporter gene expression in isolated CD8 T cells.&nbsp; And this is implying that there is impaired glycolytic potential, potentially due to a reduced use of glucose for glycolysis within these cells.&nbsp; It's important here to notice though, that these findings pertain to gene expression and can be better understood with additional protein level analyses.</p>

<p>Now, part of our research gaps in ME/CFS includes exploring immunometabolism in ME/CFS with other various metabolic methods that are pretty well established in other disease models.&nbsp; And so, the figure that I'm showing here presents a workflow of the evolving landscape of single-cell techniques, including the potential for spatial metabolomics, transcriptomics, and cytometry.&nbsp; That promises really interesting future datasets to analyze.&nbsp; And so, those are these right here on the right.&nbsp; But presently, technologies such as high-dimensional time-of-flight mass cytometry, otherwise known as CyTOF, offers a lot of sensitivity for quantifying over 40 parameters at the single-cell level that could be tailored to metabolic cell surface receptors and proteins.&nbsp; And this is really great since current flow cytometry can be difficult because it can be limited to only 10 to 20 components measured at a time, depending on how many lasers and filters that machine has.</p>

<p>Another interesting advancement is intracellular CITE-seq, which is an extension of the CITE-seq technique.&nbsp; And so, CITE-seq quantifies both protein as well as transcriptomic data in a single cell readout using antibodies that are conjugated to nucleic acid barcodes.&nbsp; So, applying this intracellularly could enable the creation of panels containing up to 200 metabolic specific components allowing with simultaneous investigation of these proteins and their transcripts in ME/CFS immune cells.&nbsp; One other unexplored approach in ME/CFS is actually a really interesting technique that's basically isotope tracing of molecules like glucose.&nbsp; And so, this method tags glucose to track its utilization within the cell and provides insights into the substrate flux, as well as metabolic preferences and strategies in ME/CFS.&nbsp; And so, this could give us information about pyruvate, serine biosynthesis, nucleotide biosynthesis, and oxidative metabolism.</p>

<p>So, it really just would reveal what happens to glucose when it enters ME/CFS cells at baseline or during activation.&nbsp; And lastly, I want to mention that all these methods I just talked about should be applied to all of these various isolated immune cell types, which have been pretty under explored.&nbsp; And furthermore, less study metabolic pathways like glutaminolysis and fatty acid synthesis deserve further attention.&nbsp; But what's more, I believe that it's crucial to investigate immune cells, not only in circulation, but also in resident tissue like muscle, stomach, spleen, and lymph nodes because, for example, over 95 percent of T cells primarily reside and function in tissues.&nbsp; But many studies focus only on peripheral circulating cells.&nbsp; And so, the significance of immunometabolism in ME/CFS really can't be overstated, because understanding the metabolic states informs the functional conditions of these immune cells, potentially getting treatment approaches.&nbsp; So, for instance, in cases of T cell exhaustion, monoclonal antibodies have been employed in cancer therapies to reverse this state.&nbsp; And while this isn't directly translatable to ME/CFS, it is a consideration for future insights into functionality, metabolism, and treatment targets in ME/CFS.</p>

<p>Additionally, something to note is that abnormal metabolic regulation isn't a consequence of the functional phenomena like exhaustion but, rather, metabolism really drives these immune cell states.&nbsp; And that offers opportunities for treatments involving drugs that target metabolic pathways, nutrient uptake, and energy production.&nbsp; And so, due to the diverse nature of immune cells and disease progression, as Dr. Li was hinting to in his talk, precision medicine does become important, in my opinion, when considering adopting any specific therapies that target metabolic components, because tailoring therapies to individuals would be important at specific metabolic stages when considering clinical trial options.</p>

<p>And so, with that, I'll end on the key immunometabolism research priorities that I've listed for ME/CFS, including comprehending the altered metabolic states and reprogramming in all immune cell subsets, both before and after activation, to really get an idea of how these cells are reprogramming their metabolism during immune challenges.&nbsp; Also, assessing the consequences of metabolic abnormalities in specific immune cell populations, such as T cell exhaustion, and what the potential trigger of the state could be.&nbsp; Another interesting thing to do would have -- would be to have in vitro testing of drugs that are known of how they modify immune cell metabolic pathways, as well as investigating the impact of the extracellular environment in ME/CFS on immune cells.&nbsp; And then finally, examining immune cell metabolism, not just in the periphery, but in tissues as well, including muscle biopsies and lymphoid tissues.</p>

<p>And so, I'll end there with just this acknowledgment slide of my fellow lab members, as well as my funding sources.&nbsp; I want to thank you again for allowing me to speak on this topic.&nbsp; And I'll be happy to take any questions and comments during now or the discussion section at the end.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Thanks so much, Jessica, for an awesome presentation.&nbsp; And thank you for presenting your serotonin data.&nbsp; I haven't picked up on that data in the paper.&nbsp; So, my question is regarding that, what was the average length of illness in the cohort yet in your study?&nbsp; Because if you think about the individuals with long COVID, none of them are going to be more than three years post infection.&nbsp; So, do you think there is the potential that there's a difference between individuals who are very early in their post-infectious state versus individuals who may be ill -- have been ill for a longer period of time?</p>

<p><strong>Jessica Maya:</strong>&nbsp;So, I'm not entirely sure what the exact number was for our average of illness duration.&nbsp; But it's definitely, I mean, an average longer than three years from what I know about our patient cohort for that study.&nbsp; And so, it would be worthwhile.&nbsp; And I think we have done some correlation analyses just with the survey data that we acquire during these experiments to see if there are any correlations with the metabolite data we have and the length of duration that are reported by the patients.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Thanks.&nbsp; So, I don't see any questions that have come other than a comment.&nbsp; "Wow, thank you, Jessica Maya for her presentation."&nbsp; Does -- do any of the panelists have a question for Jessica?&nbsp; Raise your hand or question or comment.</p>

<p><strong>Jessica Maya:</strong>&nbsp;I'll also be around at the end to help with the discussion at the end so --</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Excellent.</p>

<p><strong>Jessica Maya:</strong>&nbsp;Thank you.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Great.&nbsp; All right.&nbsp; Thank you.&nbsp; Back to you, Alain.</p>

<p><strong>Alain Moreau:&nbsp;</strong>Thank you very much, Dr. Maya.&nbsp; Also, congratulations for your excellent presentation.</p>

<p><strong>Alain Moreau:&nbsp;</strong>So, our next speaker is Dr. Jiabao Xu, who obtained her PhD in Engineering Science at the University of Oxford and remained in Oxford as a postdoctoral fellow during the last three years. &nbsp;She is now a lecturer at the James Watt School of Engineering at the University of Glasgow. &nbsp;She has been at the forefront of new innovation in the development of data mining and artificial intelligence models in single-cell Raman studies.</p>

<p>So, since 2016, Dr. Xu has been collaborating with Professor Karl Morten at University of Oxford, on investigating cell metabolism in myalgic encephalomyelitis/chronic fatigue syndrome. And this collaboration has yielded promising results, including the discovery of novel biomarkers in the blood of ME/CFS patients, as well as a pioneering single-cell Raman platforms that enhanced by the ensemble of learning as a diagnostic test has been recently published in Advanced Science. So, the implications of this innovation extend beyond ME/CFS, and holding great potential for diagnosing a wide range of chronic disease.&nbsp; Please, Jiabao Xu.</p>

<p><strong>Jiabao Xu:&nbsp;</strong>Thank you very much, Dr. Moreau, for the nice introduction, and also the invitation to join this webinar. &nbsp;So, hello, everyone.&nbsp; Thank you very much for being here.&nbsp; So, today, I'll be introducing single-cell Raman technologies for diagnosis and investigation of ME/CFS immune cells.&nbsp; So, my talk will first introduce a different aspect on the technologies.&nbsp; And then how we can leverage those technologies for the diagnosis, as well as to investigate the metabolism of the immune cells in ME/CFS.</p>

<p>So, first of all, why single cells?&nbsp; So, as we know, that for any form of life, single cells are the essential building blocks.&nbsp; However, for decades, biologists have been using ensemble measurements that is to take an average from a group of cells or a sample for biological investigations.&nbsp; But as we can see, on the right-hand side, for a single measured value there are actually many possibilities of the single-cell distributions.&nbsp; And these single-cell distributions can be masked from an average measurement.&nbsp; And these heterogeneous expression, as in the Jessica's talk, can be very important, especially in complex diseases like ME/CFS.</p>

<p>But luckily, thanks for the technical advancement, now we're equipped to study those single cells from the genomic DNA to transcription RNA, translation of protein, and then to metabolism, which is the final manifestation of the single cells.&nbsp; And then among the range of the technologies that can be used to study single cell metabolism, today, we'll be focusing on Raman spectroscopy.&nbsp; So, Raman spectroscopy is a vibrational spectroscopy that looks at the molecular vibration of a sample.&nbsp; So, as we all know, a molecule is constantly vibrating.&nbsp; And when an incident light shines on this vibrating molecule, a small portion of the light will be scattered.&nbsp; And the scattered light will have a changing frequency or wavelengths compared to the incident light.&nbsp; And this changing frequency is specific to the molecular vibration of this molecule, and therefore the chemical composition of the whole sample.</p>

<p>So, a modern Raman microscope consists of a Raman spectrometer, as well as a confocal microscope, which allows us to look at really small things like single cells or even single bacterial cells.&nbsp; And the cells from a spectrum contain a collection of all the molecular vibration, therefore, all of the biomolecules inside the cell and can be regarded as the cell's fingerprint, the cell's phenotypic identity.&nbsp; So, why do we want to use single-cell Raman compared to all the other single-cell technologies.&nbsp; So, first of all, it is label-free.&nbsp; Compared to fluorescence probing, it is very sensitive and specific, but it introduces fluorescence probes that are much larger in sizes compared to the molecules of interest.&nbsp; So, this may risk disturbing the intrinsic metabolism of cells.&nbsp; And also, Raman can tolerate water, not like, for example, in the infrared spectroscopy, which is also a vibrational spectroscopy, kind of the complementary to Raman.&nbsp; So, IR is water sensitive, so it cannot measure biological sample in situ within your aqueous environment.</p>

<p>And as I've talked about the confocal Raman microscope can have submicron resolution.&nbsp; So, we can look at subcellular scale investigations.&nbsp; Raman is also non-destructive, so we can recover all samples after use, and also involves really simple sample preparation and measurement steps.&nbsp; And complementary to, for example, single-cell sequencing techniques, it provides phenotypic information that can be understood together with the genomic information.&nbsp; So, in short, Raman spectroscopy provides a spatiotemporal phenotypic probing of a cell at a subcellular scale.&nbsp; So, nothing is perfect.&nbsp; The weaknesses of Raman is -- the Raman scattering process is naturally rare.&nbsp; So, that signal is intrinsically weak.&nbsp; And because of that, some autofluorescence, which is much higher in signal compared to Raman, and many biological samples can interfere with the Raman signal.</p>

<p>There are some turnabouts like using resonance Raman spectroscopy, coherent Raman scattering microscopy, surface-enhanced Raman spectroscopy to enhance the signal, as well as using machine learning techniques to further boost the signal-to-noise ratio.&nbsp; So, we've talked about fingerprints.&nbsp; So, imagine if we have a healthy cell and the disease cell, the Raman fingerprints will look different.&nbsp; But one Raman spectrum actually contains hundreds of Raman peaks.&nbsp; So, we'll need the power of machine learning to help us deconstruct and understand those differences.&nbsp; And by combining Raman fingerprints with machine learning, we can do useful things like building classification models for disease diagnosis.&nbsp; So, next, I will introduce a few examples using Raman fingerprints and machine learning in that disease diagnosis.&nbsp; These are not necessarily single cells, which I will cover in the second part of this talk.</p>

<p>So, firstly, the first example, researchers have to use normal cell exosome fingerprints compared to cancer cell exosomes excreted out of the cells.&nbsp; And then by building a machine learning model to classify those two categories, and comparing one's exosomes to these two categories, they were able to make early-stage lung cancer diagnosis with a high accuracy rate.&nbsp; And the second example, the serum of patients with Huntington's diseases was tested.&nbsp; And differences were found compared to the healthy controls and potential biomarkers were found in the serum of patients.&nbsp; In another study, the Raman fingerprints of normal tissues and cancerous tissues adjacent to each other were tested.&nbsp; And as I've mentioned, Raman is label-free and contact-free.&nbsp; So, this technique can actually be used to detect the tumor margins in vivo and help the surgeons do operations in real time.</p>

<p>And the final list example related to ME/CFS, so a partial least squares-discriminant analysis model was built based on the Raman fingerprints of blood EVs, extracellular vesicles, and EV miRNAs and the PBMC miRNAs.&nbsp; So, combining those features together, a machine learning model will achieve the perfect classification of the patients and controls.&nbsp; But this is quite a small cohorts of only 12 patients and 12 controls.&nbsp; So, hopefully, I've shown you a versatile range of the application's Raman spectroscopy can do in disease diagnosis.&nbsp; And we've talked about a range of sample test types as well, like serum, tissues, exosomes, EVs.&nbsp; But of course, there are single cells.&nbsp; So, in our study, and in the next part of my talk, I'll be talking about how we are developing a blood single-cell-based diagnostic test for ME/CFS using PBMCs via a single-cell Raman platform and machine learning.</p>

<p>So, I'm glad that my talk followed two fantastic talks, especially Jessica has introduced the background of immunometabolism, as well as the roles of PBMCs in ME/CFS.&nbsp; So, among all the triggers identified for ME/CFS, they're up to 75 percent of the cases that actually reported infection episodes before the onset of their condition.&nbsp; And in addition, the ME/CFS patients commonly report immune-related or inflammatory symptoms, those flu-like symptoms, such as headaches, muscular pain, tender lymph nodes, and fever.&nbsp; So, researchers have been looking into the PBMCs, the immune cells of the ME/CFS patients.&nbsp; And the metabolic assays of PBMCs in the patients actually have demonstrated a lot of metabolic alterations.&nbsp; For example, in the energetic pathways and mitochondrial functions, including decreased glycolysis, decreased mitochondrial respiration, ATP production, and dysregulation of the fatty acid oxidation.&nbsp; So, we ask ourselves -- especially when one of the most challenges seen in ME/CFS is the lack of a single sensitive and specific diagnostic test, so we ask yourself, can PBMCs be a good model for the diagnosis?</p>

<p>So, our group is actually the first group using single-cell Raman to look at the PBMCs in ME/CFS.&nbsp; So, what we did is we collected PBMCs from the healthy controls, ME/CFS patients, as well as multiple sclerosis, MS, patients.&nbsp; So, multiple sclerosis is kind of a disease-positive control because it shares many of the symptoms with ME/CFS, like central fatigue.&nbsp; But then it has a clear pathological changes in the brain.&nbsp; So, it can be diagnosed with a brain scan.&nbsp; So, here, we're using this as a positive control to compare to ME/CFS.&nbsp; So, we took those PBMCs to single-cell Raman measurements.&nbsp; And then by analyzing those Raman fingerprints, we found correlations in the fingerprints with the ME/CFS disease severity of the severe ME/CFS, moderate, and mild.</p>

<p>And then we went on to build a machine learning model using ensemble learning.&nbsp; First to look at if we can classify so as either healthy or disease.&nbsp; And then further to see if we can separate different ME severity groups.&nbsp; So, this is an overview of the study we've done, which has just been published last month in Advanced Science.&nbsp; So, more details into our study, the samples we had were from the U.K. ME/CFS Biobank.&nbsp; And we had 98 subjects.&nbsp; And before going into the Raman fingerprints, we actually looked at a series of different characterizations like the fatigue severity scale, clinical questionnaires, as well as mitochondrial respiration.&nbsp; So, the fatigue severity scale actually was different.&nbsp; So, it increased from the mild to the moderate and to severe ME.&nbsp; But then it wasn't that different compared -- if we compare it ME cohorts with the MS cohorts.</p>

<p>And the clinical questionnaires, as here down in the heat map showing -- so, the healthy controls really had very mild symptoms.&nbsp; But then the differences were not significant between the MS and different ME cohorts.&nbsp; So, the mitochondrial respiration assays, we'll look at it well because a lot of study has shown dysfunction mitochondria energetic functions.&nbsp; But then these studies were quite difficult to replicate.&nbsp; And we didn't find any significance in our data among different cohorts.&nbsp; So, that is where we went on to the Raman fingerprints.&nbsp; So, here, on the first column, we have the average Raman fingerprints for each group.&nbsp; There are Raman -- average Raman spectrum.&nbsp; So, for the first row, we have three groups, green as healthy control, red as ME/CFS, and blue as MS.</p>

<p>And for the second row, as the ME further split into different disease variety of mild, moderate, and severe.&nbsp; And for the second column and third column, they are just the projection of the high-dimensional Raman fingerprints onto a 2D space, so we can better visualize them.&nbsp; And the third -- and the second column, each dot is one single cell.&nbsp; And the third column, each dot is one individual subject.&nbsp; So, as we can see here, the single cell for the three groups really separates well from each other.&nbsp; And when we further add different ME subgroups into it, the mild group separates interestingly well from the moderate and severe.&nbsp; And then when we added the MS group into the picture on the last row, it's very interesting that the MS group actually overlaps more with the mild ME and the moderate ME, while the severe ME stands out.&nbsp; And in all cases, the healthy controls were really different from other disease cohorts.</p>

<p>And I should highlight that, as you can see, the differences were more obvious at the single-cell level.&nbsp; But when we averaged them into individuals, the differences were less obvious, so that highlights the importance of we're doing this at the individual, single-cell level.&nbsp; So, this looks all very promising.&nbsp; So, we next then looking -- discovered the diagnostic potential of our platform using this PBMCs and their single-cell Raman fingerprints.&nbsp; So, the approach we're using is called ensemble learning.&nbsp; So, basically, we combine six individual algorithms like linear discriminant analysis, neural network, and random forests.&nbsp; And then we basically combined those six individual algorithms and make use of best of them individually, and then make a more powerful algorithms that were able to give us a better diagnosis.</p>

<p>And during the training of the model, we used 80 percent of our data for training the model and then the 20 percent of data for testing the model performance.&nbsp; And then this process is repeated for five times so that all data has been entered into the training or testing dataset for at least once.&nbsp; And then the testing performance were averaged to give us the final predictions to the six.&nbsp; So, this is our final results.&nbsp; So, for the task of classifying three groups of individual is either MS, ME, or healthy control, we achieved an accuracy of 90.5 percent accuracy rate.&nbsp; And this is achieved by not compromising either the sensitivity or specificity.&nbsp; As you can see here, the sensitivity and specificity are quite balanced in all three groups.&nbsp; And when we further challenged the model in classifying the ME groups into mild, moderate to severe, we also achieved the good accuracy rate at 84 percent.</p>

<p>So, the conclusion of the diagnostic part is, this is actually the first research using single-cell Raman spectroscopy and advanced machine learning for developing a diagnostic test for ME/CFS using PBMCs.&nbsp; And hopefully, we -- I've shown you that these results are with high accuracy, sensitivity, and specificity.&nbsp; So, after particular interest is the 91 percent sensitivity and 93 percent specificity achieved for ME/CFS group.&nbsp; So, for disease-like ME/CFS, most diagnoses are based on self-reports and the questionnaires, so developing a sensitive and specific diagnosis test is very important.&nbsp; And also, our test has some merits like using minimally invasive blood test, and also the requirements for the single-cell techniques.&nbsp; The cell number can be really small, so that eventually we hopefully from just one drop of blood, the patients can be diagnosed.</p>

<p>And then in our study, we also used frozen and fixed PBMC samples from the biobanks, has additional advantage like once prepared, the samples can be stored for a long time.&nbsp; And also, we've shown in our study that the freezing duration did not show an impact on our Raman dataset.&nbsp; And also, I want to highlight that the advances in the healthcare technologies and also a greater focus on the prevention, early diagnosis, and well-being is kind of a trend over the next decade.&nbsp; And this is really important, in a chronic unexplained conditions such as ME/CFS, that allows us to use complex data and advanced machine learning techniques will be very important, not only just in diagnosis, but also in defining the early stages of the conditions where the patient shows no or few your symptoms.&nbsp; And this will help in establishing new treatment approaches linked to subtle changes in the patient's biology, preventing the development of a full-blown disease, which is harder to treat once established.</p>

<p>So, this leads to the final part of my presentation, which is to investigate the metabolic dysfunctions in immune cells of the ME/CFS.&nbsp; As I mentioned, Raman is a phenotypic technologies.&nbsp; So, we can actually quantify the amount of biomolecules in the PBMCs in different cohorts based on their Raman features.&nbsp; So, here, I've listed a panel of the most significant and most interesting molecules that are different in our cohorts.&nbsp; So, firstly, I want to highlight the altered utilization of amino acids in ME/CFS patients, including those of aromatic amino acids of tryptophan, tyrosine, and phenylalanine.&nbsp; So, tryptophan is a really interesting one.&nbsp; There are many studies, actually, have already reported altered serum level tryptophan in the ME/CFS patients.&nbsp; So, 90 percent -- 95 percent of the tryptophan goes into the kynurenine pathway to generate NAD.</p>

<p>And then there are studies shown dysregulation or overactivation of this pathway can lead to immune system activation and accumulation of neurotoxic compounds.&nbsp; Another 5 percent of the tryptophan catabolism actually goes into synthesis of serotonin, which we've heard about in previous talks.&nbsp; So, the serotonin, the neurotransmitter, is strongly associated with fatigue.&nbsp; And also, the brain tryptophan level is strongly associated with that.&nbsp; Our brain cells actually synthesize their own serotonin from tryptophan.&nbsp; And the build-up of tryptophan in the immune system might lead to reduced availability and decreased serotonin synthesis in the brain, therefore, contributing to central fatigue.</p>

<p>And because we were talking about amino acids, I want to direct you to a really nice study that looked at 20 amino acids in the serum in 200 ME/CFS patients, and 102 healthy individuals.&nbsp; They've shown that the amino acids that fuel oxidative metabolism via the TCA cycle is reduced in the patients.&nbsp; And this reduction is actually specific.&nbsp; So, if we look at the TCA cycle here, there was specific reduction in amino acids at the Acetyl-CoA level, but not reduction in the amino acids at the pyruvate level.&nbsp; So, there must be something wrong from the pyruvate to Acetyl-CoA.&nbsp; So, they then look at the mRNA levels related to the PDH, the pyruvate dehydrogenase.&nbsp; And they found the mRNA levels increased in the PDH-related genes, including PDK1, PDK2, PDK4, and SRIT4 in the patient's PBMCs.</p>

<p>So, the hypothesis is because there was immune response triggering metabolic dysfunctions in the TCA cycle, and in the PDH-related pathways, the ATP generation was not enough.&nbsp; So, then there were metabolic adaptations to compensate this ATP productions by increasing the consumption of amino acids independent of the PDH pathways in order to maintain the ATP production.&nbsp; And the further they've grown some muscle cells in the serum from severe ME/CFS patients, and actually showed metabolic adaptations, including increased mitochondrial respiration and excessive lactate secretion.&nbsp; And these findings are in agreement with the clinical disease presentation of the ME/CFS patients, with not enough ATP generation and excessive lactate generation upon assertion.</p>

<p>So, these findings related to energy sources are actually in agreement of our -- some of our findings in our Raman data.&nbsp; So, we found a decreased cholesterol in all of the patient cohorts, including the mild, moderate, severe ME, as well as the MS cohorts.&nbsp; And the increasing or fluctuating cholesterol and unsaturated fatty acids components.&nbsp; So, this could be that the lipid energy sources are broken down due to lipolysis into glycerol and free fatty acids to generate more energy.&nbsp; And then another energy storage, glycogen is found decreasing in other disease cohorts as well, potentially as another kind of energy sources.&nbsp; So, this really leads to the last few slides of mine.&nbsp; So, this is our future plan, not only for our work, but I think generally in the single-cell field, what I think we should do.</p>

<p>Firstly, for the diagnosis part, we had some exciting early results showing the potential of this platform to be developed into a diagnostic test.&nbsp; So, our future plan is to involve more independent datasets in order to validate our approach.&nbsp; And also, we're looking to expanding our cohorts, not just in ME/CFS, but also to more diseases like long COVID and chronic Lyme syndrome.&nbsp; Because it's fascinating that how similar symptoms these diseases share, but also they are still different.&nbsp; So, it'll be very interesting to look at the similarities and differences among those diseases.&nbsp; And, as Jessica has, like, strengthened in her talk, the heterogeneity within the PBMC population, as well as in, I think, all the single-cell populations.&nbsp; So, it is very essential to look at the subpopulation of the cells.&nbsp; So, this is what we are looking to do in the next steps as well is to look the subpopulations of PBMCs.&nbsp; And by combining with cell sorting techniques, we can do transcriptomics, proteomics, and metabolomics linked to this subpopulations of cells, so we can understand more specifically what these cells are doing.</p>

<p>And then for the metabolic investigation, we can combine single-cell Raman with techniques like stable isotope probing, as well as single-cell Raman imaging to look more deeply into the metabolism of the immune cells of the pathogenesis underlying ME/CFS to find potential biomarkers, and to study the metabolic and energetic pathways of these immune cells.&nbsp; And hopefully, I've illustrated that the most important research questions in my talks.&nbsp; I've touched upon, hopefully, lightly on each of them.</p>

<p>So, firstly, how can a single, objective, accurate, and reliable diagnostic test be developed for ME/CFS?&nbsp; And secondly, is ME/CFS an autoimmune condition?&nbsp; And can investigation of the immune cells help with diagnosis and treatment of the disease?&nbsp; And thirdly, how do we improve disease stratification, and the heterogeneity in complex disease like ME/CFS?&nbsp; And last but not least, why do some people develop ME/CFS following an infection?&nbsp; Is there a link between the ME/CFS with diseases like long COVID, and other post-acute infection and chronic syndromes?&nbsp; So, I would like to thank my collaborators, especially those at Oxford and also ME Association for funding and samples for this study.&nbsp; And thank you all very much for listening.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Thank you so much.&nbsp; A really excellent, wonderful, exciting talk.&nbsp; So, a couple of questions for you.&nbsp; So, I guess one question is, "Did you look further, and do you see differences in the Raman signatures in individuals with ME/CFS due to differences in their comorbidities, like if someone has IBS, hypothyroidism, different symptoms?&nbsp; Have you looked at your results in that way?"</p>

<p><strong>Jiabao Xu:&nbsp;</strong>Yes.&nbsp; We actually did a correlation test with many confounding factors, like, of course, the abnormalities at the age, the demographics, as well as the drug history, the treatment they've received.&nbsp; But surprisingly, we didn't find any correlation with the Raman fingerprints.&nbsp; And we also did like to find a combination of those features to see if they can help with the diagnosis.&nbsp; But then surprisingly, the Raman fingerprints were the most obvious significant to try to classify different cohorts.&nbsp; But I think to take consideration of these confounding factors is really important to make sure our baseline demographics is valid.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Yeah.&nbsp; And there was another question about, "What are your next steps here?"&nbsp; Which you addressed somewhat.&nbsp; But, you know, in thinking about what you said about looking at other post-infectious syndromes or diseases, clearly, for it to become an accepted diagnostic test, you're going to have to demonstrate sensitivity and specificity.&nbsp; So, do you want to comment a little more about your plans there to look at other disease entities and compare it to ME/CFS?&nbsp;</p>

<p><strong>Jiabao Xu:&nbsp;</strong>Yeah.&nbsp; This is an area I really -- I'm really excited about.&nbsp; I just attended actually ILADS, which is a conference for Lyme disease last week in Boston.&nbsp; So, it's amazing how often you heard the word of ME/CFS in the conference about Lyme.&nbsp; So --</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>I see.</p>

<p><strong>Jiabao Xu:</strong>&nbsp;-- and there's a case study.&nbsp; Actually, I was really surprised.&nbsp; So, a clinician treated the patient with Lyme disease.&nbsp; They recovered, but then COVID exploded.&nbsp; They got COVID.&nbsp; And then all the symptoms with Lyme came back.&nbsp; So, it's amazing how these comorbidities link with each other with -- in the sense, we -- I think we don't know yet.</p>

<p><strong>Jiabao Xu:</strong>&nbsp;So, by compare -- I think incorporating those cohorts, and very carefully differentiating these, will be really important to look at the similarities, as well as the differences.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>All right.&nbsp; Thank you very much.&nbsp; Yeah.&nbsp; So, what -- so, I guess I was thinking also in terms of -- I know that they're -- and we'll hear about this in an upcoming webinar about the genetics studies that have been done utilizing the biospecimens from the U.K. Biobank.&nbsp; So, it would be really interesting once those data are available to correlate your Raman fingerprints with the genetic findings from those.&nbsp; I would think probably some of -- at least some of the same individuals that would be evaluated both in these studies, as well as your studies.</p>

<p><strong>Jiabao Xu:&nbsp;</strong>Absolutely.&nbsp; Yeah.&nbsp; I think data mining, like, to compare the data at different levels is really essential.&nbsp; We cannot draw conclusion from just one level of the data.&nbsp; So, in our -- we also, actually, were planning to look at the metabolomics for a lot of the serum samples in our cohorts.&nbsp; And then I think I'm really interested to transcriptomics, as well as the genomics.&nbsp; And then by comparing the data at different level, really, we can have a comprehensive view of what is happening in those cells of ME/CFS patients.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Exciting.&nbsp; Very exciting.</p>

<p><strong>Jiabao Xu:</strong>&nbsp;I think yes.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>So, I don't have any other questions.&nbsp; Do any of the panelists have questions for Dr. Xu?&nbsp; Yes, Alain.</p>

<p><strong>Alain Moreau:&nbsp;</strong>Dr. Xu, did you look about the impact of post-exertional malaise?&nbsp; So, if you were able to stress participants, anyhow, by different approaches, or to see if you can see a rapid injection of change in your Raman fingerprint?</p>

<p><strong>Jiabao Xu:</strong>&nbsp;I missed the first word.&nbsp; Was it pathogen?</p>

<p><strong>Alain Moreau:&nbsp;</strong>No.&nbsp; I mean, amongst ME/CFS, if most of your samples you test them at baseline.&nbsp; Okay.&nbsp; You do not introduce any stress test, any challenges to look either at the cell level or the plasma level.&nbsp; So, my question is, we know that for most of ME/CFS patients, that concern is related to the crashes.&nbsp; Okay.</p>

<p><strong>Jiabao Xu:&nbsp;</strong>Okay.</p>

<p><strong>Alain Moreau:&nbsp;</strong>And if they can -- we can limit or prevent post-exertional malaise, we may have a big step forward for them.&nbsp; So, I'm wondering if your technology using Raman fingerprinting can capture the effect of post-exertional malaise?&nbsp; Okay. Do you understand?&nbsp; If there's a different fingerprints here based on the same patient at baseline?</p>

<p><strong>Jiabao Xu:&nbsp;</strong>This is a very good question.&nbsp; I think one limitation of our study is we only look at one time point.&nbsp; So, this is not a longitudinal study, which is I think is essential because patients, as you said, they're going through fluctuations of symptoms.&nbsp; So, yes, the important next step will be looking at different time points before and after the collapse, for example, and before and after treatment of the patients.&nbsp; Yes, we're looking forward to doing that.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>So, another question for you.&nbsp; "Have you seen abnormalities in ferritin or bilirubin levels when you compare CFS versus controls or MS?"&nbsp; Do you look at that?</p>

<p><strong>Jiabao Xu:&nbsp;</strong>Yes.&nbsp; So, these two are not the metabolites that were usually picked up by Raman.&nbsp; So, I think Raman is very -- so it -- the good thing is it look as an overall of the biomolecules.&nbsp; But the bad thing is they are not that specific.&nbsp; So, that's why I've mentioned aromatic amino acids, because they are resonance Raman spectrum, which is -- so, the signal of them is enhanced in the Raman spectrum, and also lipids and proteins in general.&nbsp; But if we're looking for a particularly one metabolites, it'd be good to look at metabolomics.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Excellent.&nbsp; Any other questions from anyone?<strong>Vicky Whittemore:&nbsp;</strong>Okay.&nbsp; I'd like to welcome everyone back to the webinar.&nbsp; And turn it over to you, Alain, to introduce the next speaker.</p>

<p><strong>Alain Moreau:&nbsp;</strong>Thank you very much, Vicky.&nbsp; So, it's my pleasure to introduce you, Dr. Wenzhong Xiao.&nbsp; So, Wenzhong is someone that I know very well.&nbsp; We are collaborating on different projects.&nbsp; So, Dr. Xiao is the assistant professor of Bioinformatics at Harvard Medical School and director of the Immuno-Metabolic Computational Center at Mass General in Boston, Harvard Medical School.&nbsp; He also leads a Computational Genomics Group at Stanford Genome Technology Center at Palo Alto.&nbsp; His research is at the interface of computation, genomics, and medicine.&nbsp; And as you may know, a major bottleneck of genome medicine today is around data analysis, interpretation, and integration.&nbsp; So, that's why his research focused on integrative analysis and interpretation of multi-dimensional molecular, cellular, and clinical data of many types of patients, including those suffering of ME/CFS.&nbsp; Wenzhong?</p>

<p><strong>Wenzhong Xiao:&nbsp;</strong>Thank you, Alain.&nbsp; I just want to say, I feel really glad to be at this conference, this seminar.&nbsp; I heard a lot of excellent talks already.&nbsp; And, you know, I think the field has really moved along.&nbsp; And by working together, we probably would be able to solve some of the problems that we have with research in ME/CFS very soon.&nbsp; So, I guess this is the task that I was assigned to.&nbsp; So, basically, from a computational biologist perspective, what do we know?&nbsp; What we don't know?&nbsp; And what do we need to know?&nbsp; And what are the research priorities in ME/CFS research, especially in studying metabolism in ME/CFS?</p>

<p>Many of us are very familiar with this early publication by Bob Naviaux' group, where they did metabolic profiling in the blood plasma of ME/CFS.&nbsp; And I'm sure everybody knows their findings.&nbsp; That many of the metabolites in the blood plasma seemed to have lower levels in the patients comparing to the controls.&nbsp; And therefore, this idea of hypometabolism in ME/CFS was proposed.&nbsp; And the field obviously moved fast since 2016.&nbsp; And this is a paper from Maureen Hanson's group from last year, where they showed the increased disparity between patients and controls in their blood plasma, when the patient went through exercise, or as Alain mentioned earlier, your stress test.&nbsp; So, I think this really highlights the importance of challenging the patients and be able to capture the difference at a molecular level when the patients underwent post-exertional malaise.</p>

<p>However, as the previous speakers already mentioned, one of the observations from the computational side when we do data analysis, we see is that if we look at the changes or metabolic features identified between different studies, often they're not really consistent to each other.&nbsp; So, I think, as a research field, we need to address, you know, what causes this differences?&nbsp; And how we might be able to improve the consistency between the studies?&nbsp; So, that's what I'm going to talk a little bit in the next few slides.&nbsp; So, in terms of what we don't know so far.&nbsp; I think many on this call probably would agree that we do not have a robust metabolic signature in blood plasma of ME/CFS, or urine for that matter.</p>

<p>So, the question to a computational researcher would be, you know, what contributes to the differences seen between the metabolic studies?&nbsp; And try to figure out the technical and biological variations.&nbsp; The second topic I think we need to address, again, a couple of previous speakers have already talked about this is, how much the metabolic profiles we see in circulation actually can reflect the underlying disease mechanisms?&nbsp; Because, as we know, the metabolites in circulation do not necessarily reflect levels of the same metabolite in the tissues that also relevant to this disease.&nbsp; We already heard from the speakers in the previous section in terms of the cellular metabolomics in circulating blood cells.&nbsp; But the question then is, you know, would avid tissues be also important for the disease mechanism?&nbsp; And if so, how can we actually use the metabolic profiles in the blood plasma to infer the mechanisms going on in those tissues?</p>

<p>So, these are just some of -- we did analysis of the published studies in terms of metabolic profile in ME/CFS.&nbsp; But these are just some of the studies that we did internally.&nbsp; On the right side, you see a study between NMR and mass spectrometry platforms in terms of measuring blood plasma metabolite levels.&nbsp; So, here, it just shows some of those amino acids and other molecules where you would see the discordance between the measured levels.&nbsp; So, the question is, what actually contributes to this?&nbsp; These are likely going to be technical issues specific to the platforms that we should be aware of, if we're actually doing metabolic profiling using multiple platforms.&nbsp; Obviously, we know that NMR-based metabolic profiling provides unique type of information.&nbsp; That currently, I think, mass spec-based platforms still are lacking.&nbsp; But the correlation between different metabolites is one of the factors that are important when we actually do this multi-platform metabolic profiling.</p>

<p>On the left side, this shows a study that we did with severely ill ME/CFS patients and controls, where we used both mass spec-based metabolic profiling, as well as some of the available clinical lab tests.&nbsp; So, as you can see here, while the overall correlation is there between the patients and controls between the lab tests and mass spectrometry, but you do see some of these outliers that have even negative correlation between each other.&nbsp; So, these are just some of the technical issues that I think anybody who's doing metabolic profiling need to be aware of.&nbsp; Another perhaps a more difficult issue with metabolic profiling, especially in the blood plasma, is that there are many other factors that came, as previous speaker have already talked about, would affect the metabolites in the blood.&nbsp; And these include, for example, the diet, activity of the patients, sleep status, and medication in addition to the heterogeneity of the disease.&nbsp; So, this is just one example on the top panel is -- in one of studies of the severely ill patients, where we measured the saliva level of cortisol levels.&nbsp; And here you see that the severely ill patients in red had significantly lower cortisol level in the morning.&nbsp; But you don't really see that, you know, along the day.</p>

<p>On the bottom left, this shows the -- one of the studies by the Yale group on long COVID patients, where you have sort of similar observation on the long COVID patients comparing to the controls.&nbsp; And in that publication, they actually showed that just by using -- the measurements were done, I think, between 10:00 a.m. to 12:00 p.m.&nbsp; So, basically, in the morning.&nbsp; You know, by just using the morning cortisol level, they showed an area under curve of 96 percent, distinguishing long COVID patients from the controls.&nbsp; And on the right side, this was Mark Davis lab work, that showed as well that the patients with PASC had the low cortisol level comparing to healthy controls.&nbsp; So, it seems like, you know, we see consistency signature in this particular case between severe ME/CFS and long COVID.&nbsp; However, you know, one needs to realize that, for a lot of ME/CFS patients, as well as long COVID patients, a characteristic symptom is sleep problem of the patients.</p>

<p>So, the question then becomes whether this is actually essentially a symptom of the patient that actually altered the metabolite level of the patient that we can identify here.&nbsp; So, then the observation may not be really indicating a causal component of the disease mechanism.&nbsp; So, this is another level of complexity, I think, we need to be aware of when we're studying metabolism in patients.&nbsp; So, these are just summary of some of the considerations from a computational perspective on the biomarkers study of ME/CFS, because as we know, you know, the important reason for studying metabolism in ME/CFS, especially in plasma, is to develop definitive diagnostic, prognostic, or predictive biomarkers.&nbsp; And as I explained earlier, you know, the quality of the assays, the accuracy and reproducibility; disease heterogeneity; the biological variations; and, you know, some of the issues on the computational analysis that we see in the past might contribute to this reproducibility issue that we see so far in the blood plasma studies in ME/CFS.</p>

<p>In the field, we know that for the past 20 years, also, in order to get -omics studies into clinical practice into FDA approval, there are several large-scale consortium.&nbsp; I only listed a few of them that, you know, I'm familiar with, that try to address this reproducibility issue.&nbsp; That I think, you know, what we -- these are some of the things that we can learn when we study metabolomics in ME/CFS, or metabolomics in general.&nbsp; So, this is just the listing of some of those recommendations as, you know, one can find from the results of some of these consortiums, which include, for example, standard operation protocols of sample collection, processing, and assays.&nbsp; The standardization of patient selection and collection of clinical information.&nbsp; And I should add, also, the results would be dependent on what type of controls we actually use.&nbsp; And the multi-center definitive studies certainly is needed when we're talking about biomarker studies of ME/CFS.&nbsp; And rigorous training, test, and validation would be needed in this field.</p>

<p>So, for the next few slides, let me just move on to the second topic, I think, we need to address in order to advance the studying of metabolism in ME/CFS.&nbsp; And that is how much plasma in metabolic profiles can reflect disease mechanism in ME/CFS.&nbsp; So, for the, you know, next few slides, I'm just basically going to go through a few of the major hypotheses that I think, you know, most likely everybody on this call are quite familiar with.&nbsp; One is certainly close to what I'm studying, which is, you know, the mechanisms for exertional intolerance.&nbsp; And this is in collaboration with David Systrom at the Brigham and Women's Hospital.&nbsp; So, what his lab showed is that, you know, they say, preload failure in ME/CFS patients, where you basically get the returning of blood through the venous vein to the heart decreased because of the issues in, you know, presumably, the peripheral nervous system.</p>

<p>And another mechanism that we see is this mitochondrial dysfunction that potentially reduced the oxygen extraction in the tissue that caused problems in getting oxygen into the -- you know, for example, skeletal muscle.&nbsp; That, therefore, caused potentially exertional intolerance that we see in patients.&nbsp; So, the question here would be that, you know, we're looking at largely the muscle problem and the nervous system problem of the patients.&nbsp; And how we can actually use the blood plasma data to identify this mechanism, and to be able to measure the severity of the issue just based on blood plasma data?&nbsp; The -- another hypothesis, as we all know, is this autoimmunity, where, you know, in this particular hypothesis, you get autoimmunity that would, you know, end up reducing muscle perfusion and the cerebral blood flow.&nbsp; And in this situation, again, in measuring blood plasma may or may not be able to reflect what's going on in the muscle and in the cerebral blood flow.</p>

<p>And I think we're quite familiar with the idea of neuroinflammation.&nbsp; And needless to say, the ways to be able to directly study the brain is extremely limited.&nbsp; So, I think many of us in the research field have tried to then look at the cerebral spinal fluid.&nbsp; And this is a study by Bob Naviaux' group published last year, where they looked at the blood plasma, as well as CSF in healthy controls, before and after 90-minute exercise.&nbsp; And the point here is that when they directly looked at the metabolite levels in the CSF and in blood plasma, what they found was only few metabolites actually show direct concordance in terms of their levels, you know, as example, serotonin with metabolites in the blood plasma.&nbsp; So, this again raised this issue, if we're only looking at, for example, blood plasma, can we actually study disease mechanisms if indeed neuroinflammation is important in ME/CFS?&nbsp; If I remember correctly, here, serotonin is not actually significantly correlated in the blood plasma versus in CSF.</p>

<p>On the other hand, the CSF -- in CSF, serotonin is -- serotonin level is most correlated with, I think, alanine level in the blood plasma.&nbsp; So, this potentially is consistent with some of the points that previous speakers mentioned with respect to serotonin levels seen in patients.&nbsp; Perhaps a more impressive finding that they came up with was, you know, for dopamine levels in the CSF, you know, there's, you know, basically no metabolites in the blood plasma seem to be significantly correlated with the dopamine level, before and after exercise, in the individual study.&nbsp; So, this gets to the point that, you know, whether we should actually ask the next step of research in metabolism in ME/CFS, you know, start to look at some of the tissues.&nbsp; And in this case, CSF, which, you know, understandably, are very hard to get to in a research setting.&nbsp; And lastly, as we know, certainly chronic viral infections is one of the, you know, issues that we all are considering and studying.&nbsp; And, you know, whether there is a chronic viral infection metabolic profile that we can establish, and that would certainly, you know, help us to understand the disease mechanism and what's going on in each individual patient, you know, if that can be established.</p>

<p>So, I think what we -- what do we need to know?&nbsp; One is we need to identify the metabolic dysfunctions in relevant tissues, for example, muscle or CSF, in order to understand the disease mechanisms and to identify treatments.&nbsp; And secondly, I think we need to develop robust metabolic biomarkers for disease diagnosis.&nbsp; And further, we should try to develop predictive biomarkers for targeted treatments, because at the moment, everybody knows there's not really an effective treatment for the disease.&nbsp; But then, you know, whether we can actually identify the molecular signatures using metabolomics for subset of patients that potentially can benefit for a particular treatment.&nbsp; I think that would be a success for the field.</p>

<p>Locally, in Boston, Dr. Systrom's lab is starting to recruit patients for this muscle biopsy study in a two-day CPET setting, as shown here.&nbsp; And I hope that, you know, this can help shed light to the issue of tissue metabolism and metabolic dysfunctions in ME/CFS patients.&nbsp; There have been at least a couple of studies already showing the potential metabolic dysfunction in muscle in ME/CFS.&nbsp; One of them is this WASF3 study conducted by the NIH group, as well, I guess, many people on this call are quite familiar with.&nbsp; I think the speaker -- the first speaker today already showed the human metabolism network.&nbsp; So, I wouldn't really get into a lot of the details here.&nbsp; But needless to say, the metabolic network, especially -- oops, especially the cellular metabolism is just like a complex pipeline of a complex system, where you have some of the pathways that has of very high capacity, and some of them, you know, have really low capacity.</p>

<p>And the low-capacity points are typically going to be something like a bottleneck of the system.&nbsp; So, we and other labs developed different ways to model cellular metabolism.&nbsp; And I'll just show you one example that I think might be relevant to ME/CFS.&nbsp; So, these are patients that we studied for the past couple of decades of those severely ill patients after they went through ICU and prolonged recovery stage.&nbsp; So, we did muscle biopsies of these patients following up for more than two years after their initial injury.&nbsp; And we try to identify the choke points in their cellular metabolism, in this case, the muscle, and try to identify potential treatments that might be able to help these patients.&nbsp; So, you know, as we know, in this field and in medicine genre, for those patients who suffer severe injury -- you know, this is up to two years after their injury -- you'll see a significant decrease of their ATP production capacity in their skeletal muscle.</p>

<p>And if we tried to model using in silico knock-in and knock-out to see what are the genes, if you in silico activate them, or to help the production of energy in these patients.&nbsp; You know, to our surprise, this was actually before oxaloacetate.&nbsp; It was tested in ME/CFS.&nbsp; You know, we were surprised that, you know, getting genes activated and increased oxaloacetate might be able to help the patients in -- you know, after severe injury to increase their ATP production and, hopefully, can, you know, improved their recovery.&nbsp; So, I hope that, you know, once we have the muscle biopsy study done in the ME/CFS patients at David Systrom's Group, we would be able to, you know, use similar approaches to see whether we can identify some of the candidates that can help to improve the metabolism in ME/CFS patients.</p>

<p>And with that, I just want to say the priorities that -- from a computational perspective that we can see so far.&nbsp; One is to try to understand the metabolic problems in relevant tissues in ME/CFS, which means that we probably should look into the muscle and, for example, CSF of the patients in order to really understand what's going on in the tissue of the patients.&nbsp; And secondly, we probably should compare the plasma profiles that are accumulating so far with the profiles of other diseases to infer the disease mechanism, whether it's a hypoxia type of problems in the patients, or perhaps that, you know, has to do with, you know, autoimmunity or chronic viral infection.&nbsp; The third is to, as I showed in previous few slides, integrate the plasma metabolomics data with the tissue data to be generated, I think, by everybody in this field, and try to understand the disease mechanism and identify potential treatments.&nbsp; The fourth is to, you know, really try to make the results more reliable and reproducible in terms of identifying metabolic biomarkers for the disease diagnosis and predictive markers for targeted treatments in ME/CFS.&nbsp; And that's the end of my presentation.&nbsp; Thank you.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Thank you very much for excellent presentation.&nbsp; There's a couple of questions that have come in.&nbsp; So, first, Peter, I'm acknowledging your question that I see in the Q&amp;A.&nbsp; And I think we'll save that for the end of discussion, because I think it really comes back to sort of trying to wrap all this up and make sense of everything we've heard today.&nbsp; So, one question is, "Is it possible that ME/CFS is an autoimmune response to a metabolic process underlying exertion?"</p>

<p><strong>Wenzhong Xiao:&nbsp;</strong>Yeah, absolutely.&nbsp; I think there's a lot of folks on this call probably were aware of this.&nbsp; You know, there have been, you know, at least several studies on autoimmunity in ME/CFS.&nbsp; And, you know, just use the example that, you know, we participated in the study with David Systrom, where he was trying to use Mestinon to increase the, you know, effectiveness of acetylcholine.&nbsp; And, you know, even there whether, you know, the acetylcholine receptor itself, you know, can be influenced by, for example, in autoimmunity.&nbsp; As we know in other -- certainly, in other diseases, you know, that has happened.&nbsp; The question is, you know, whether that is indeed happening in ME/CFS patients?&nbsp; And if so, how many ME/CFS patients or what percentage have that underlining disease mechanism?</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Yeah.&nbsp; Another question for you is, "Which, in your opinion, is more indicative of metabolic signature in the brain and muscle, urine, plasma?"&nbsp; I'll add CSF to that question.&nbsp; "And what do you believe are the essential inclusion, exclusion criteria for selecting healthy controls in studies involving ME/CFS?"&nbsp; So, if you take the first one, sort of what's the most indicative metabolic signature?</p>

<p><strong>Wenzhong Xiao:</strong>&nbsp;Yeah.&nbsp; Very good question.&nbsp; So, if we're to think that, you know, in fact, neuroinflammation is indeed important.&nbsp; Then we get into, you know, some of these well-known neurotransmitters like serotonin, dopamine.&nbsp; From Bob Naviaux' publication in 2022, you know, there, you know, they identified a few metabolites in the blood plasma that seem to be highly correlated in serotonin level.&nbsp; For example, they're not serotonin in the blood plasma, but there are other metabolites in the plasma.&nbsp; The question then, you know, gets to our sort of first topic, you know.&nbsp; How reliable these or reproducible these are going to be, you know?&nbsp; So, that I think remains to be seen.</p>

<p>That -- you know, that study needs to be, in my opinion, repeated and verified.&nbsp; But suppose that is reproducible, the short answer would be, you know, there is a group of metabolites in the blood plasma that can be used to predict the level of serotonin in the CSF of the patients.&nbsp; And I don't know if, you know, there are any studies in -- you know, correlating, for example, serotonin level in the CSF with the urine levels.&nbsp; I just haven't seen that type of study yet.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>So, the second part of that question was, "Essentially what healthy controls should we be using to compare ME/CFS?"&nbsp; I know Maureen Hanson's group has these sedentary controls.&nbsp; Other people have -- and we heard today comparisons to individuals with MS.&nbsp; So, what do you think about that?</p>

<p><strong>Wenzhong Xiao:</strong>&nbsp;Yeah.&nbsp; That's an excellent question.&nbsp; I also have discussions with Maureen about this as well.&nbsp; You know, in an ideal world, I think we actually need both.&nbsp; The reason -- I think Maureen can articulate this better than I can.&nbsp; The, you know, justification of using, "healthy controls" is potentially for developing diagnostic markers, you know, whether just showing, you know, objective lab test results.&nbsp; Showing that the patients are different than healthy controls would already help the patients in getting, you know, healthcare.</p>

<p>You know that argument, you know, I can certainly agree.&nbsp; But then, you know, in order for the, let's say, primary care physicians, or even any CSF specialists to be able to, you know, diagnose the patients quickly, because we're -- we know that the disease is so much underdiagnosed.&nbsp; Then they are -- you know, you would want to be able to distinguish patients from other related diseases.&nbsp; For example, as the previous speaker mentioned, MS, let's say.&nbsp; So, they're probably going to be at least four or five -- I mean, we can -- you know much more than I do, you know, the related diseases that we should actually look at in order to distinguish the signature of ME/CFS from those related diseases.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Yeah.&nbsp; So, this is like, I guess, question, comment from -- so, the other hat I wear at NINDS is to oversee grants on the genetic epilepsies.&nbsp; And there are now, I think, 110, 120 known genes.&nbsp; That when there are mutations in those genes can lead to seizures and epilepsy syndromes.&nbsp; So, looking at the complexity of the metabolic pathway, it's not at all surprising to me to think that there could be problems in -- or any different parts of that pathway that could lead to potentially similar symptoms that we diagnosis ME/CFS.&nbsp; So, it's like thinking about your pipeline picture.&nbsp; There could be leaks at many different joints in that --</p>

<p><strong>Wenzhong Xiao:&nbsp;</strong>Exactly.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;It's still leads to a leaky pipe.&nbsp; So, just your thoughts about that.&nbsp; And is that what we're seeing?&nbsp; In part one, we see this heterogeneity, potentially, between individuals with ME/CFS.</p>

<p><strong>Wenzhong Xiao:&nbsp;</strong>Yes.&nbsp; I definitely say agree.&nbsp; And, you know, I think we see this for some of the other so-called complexity disease&nbsp; as well, where, you know, you could potentially have multiple places, you know, almost like we were driving a car there, you know.&nbsp; If the car is broken, there could be, you know, multiple places that can, you know, make the car to stop.&nbsp; So, that seems to be, you know, something that we should, you know, definitely talk about.&nbsp; I know, you know, there are at least six or eight different seminars that you're organizing.&nbsp; You know, I think this is not only an issue with metabolism, but also with some of these other, you know, aspects of the -- yeah, of the research.</p>

<p>It's relevant to those topics as well.&nbsp; But the bottom line is, definitely, it seems like we're seeing more than one place that could be relevant to the mechanism of ME/CFS.&nbsp; And we also see some of the, you know, whole genome sequencing data of ME patients.&nbsp; And, indeed, as you said, you know, there seems to be, you know, neurological components that, you know, might cause similar symptoms in at least some of the patients.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Thank you, Wenzhong.&nbsp; Any other additional questions or comments for Wenzhong?&nbsp; If not, thank you very much for really excellent presentation.&nbsp; Back to you, Alain.</p>

<p><strong>Alain Moreau:&nbsp;</strong>Thank you very much, Vicky.&nbsp; By the way, Wenzhong, you should ask David Systrom to send you a more recent picture.&nbsp; I think the one that you use was during his residency.</p>

<p><strong>Wenzhong Xiao:&nbsp;</strong>That's true.<strong>Alain Moreau:</strong>&nbsp;Our next speaker is Dr. Brent L. Williams, who is an assistant professor of epidemiology and pathology and cell biology at Columbia University Medical Center, and in the Center for Infection and Immunity.&nbsp; Dr. Williams has conducted the microbiology and epidemiology research for the past 25 years.&nbsp; His research broadly investigates the relationship between the microbiome and human diseases with a focus on the role of the microbiome.&nbsp; And the gut-brain and gut-immune axis, and its influence on neurodevelopment behavior, gastrointestinal dysfunction, infection, and cancer.&nbsp; He has been involved in microbiome research efforts for two NIH support multi-institutional studies of ME/CFS patient in U.S.&nbsp; The chronic fatigue and the pathogenic discovery and the pathogenesis program at the Center for Solutions for ME/CFS.&nbsp; Brent, thank you.</p>

<p><strong>Brent Williams:&nbsp;</strong>Thank you, Alain, for that introduction.&nbsp; And thanks for inviting me to talk about microbiome metabolism in ME/CFS pathogenesis.&nbsp; So, you know, first I wanted to talk about what -- a little bit about what we know about microbiome, metabolism in general, and its role in human health.&nbsp; Microbial cells outnumber human cells in the body at a ratio of 1.3 to 1.&nbsp; And collectively, the microbiome encodes 150 times more genes than the human genome.&nbsp; And this actually may be an underestimate when you consider genes that may be expressed only in certain strains of bacteria found in the gut microbiome.&nbsp; And these microbes contribute to a range of physiological functions that include fermentation of dietary components, vitamin synthesis.&nbsp; They confer resistance to pathogens.&nbsp; And they also play a role in maintaining gut barrier function.</p>

<p>The gut microbiome has actually been likened to the human liver due to its metabolic capacity and the diverse reservoir of metabolites that it produces.&nbsp; And it -- because it is -- you know, the first exposure to dietary components, drugs, and also plays a role in the breakdown of endogenous compounds.&nbsp; The sheer number of metabolites that it can produce is quite extensive and is actually not entirely known.&nbsp; Next slide.&nbsp; And the microbial metabolites have been shown to influence host physiological processes, not only within the gut, but also in peripheral systems.&nbsp; And this has led to a wide number of studies now that have proposed that there are microbiome metabolite-driven axes between the gut and nearly every organ system in the body, not shown here in this picture.&nbsp; There's also a microbiome or gut-lung axis and also the gut-immune axis.</p>

<p>And when microbes are in balance in the gut, they promote human health.&nbsp; The balance is sort of described as eubiosis.&nbsp; And when they are out of balance, or dysbiosis, their metabolite profile changes.&nbsp; And this may be contributing to a range of diseases.&nbsp; The microbial messengers that are produced have been shown to regulate host metabolism in various ways.&nbsp; And there are -- this has sort of been a simplification of some of those major metabolites that have been most thoroughly investigated.&nbsp; And these include tryptophan metabolites.&nbsp; Microbes can produce these metabolites themselves, but they can also regulate how the host sort of metabolizes tryptophan through the serotonin, kynurenine pathway.&nbsp; But unique to microbes, there's actually a third pathway where they convert tryptophan into indole products.&nbsp; And these indoles have been shown to play an important role in regulating host immune responses, because they bind to the aryl hydrocarbon receptor on immune cells.</p>

<p>The microbiome also, as a result of dietary fiber fermentation, produces short-chain fatty acids.&nbsp; Similar to these short-chain fatty acids have been shown, and these include butyrate have been shown to bind to GPCR receptors, and to regulate the barrier function of the intestinal tract to prevent, you know, unwanted things from getting through the intestinal tract into the circulation.&nbsp; They also directly bind to receptors on immune cells and can influence and generally have anti-inflammatory effects on immune cells.&nbsp; The microbiome also converts primary bile acids into secondary bile acids.&nbsp; And these bile acids have been shown to result in altered release of hormonal peptides that regulate satiety in the brain and energy harvest, fat storage, and adipose inflammation and a wide range of other functions.</p>

<p>Of course, bacteria can also produce lipopolysaccharide, which is proinflammatory.&nbsp; If it gets through the intestinal wall, they produce trimethylamine, which from L-carnitine, which when it reaches the liver gets converted to trimethylamine oxide, which then promotes inflammation and fatty acid metabolism.&nbsp; It also -- the gut microbiome also produces branched-chain amino acids that affect physiology. &nbsp;N-acyl amides and a range of neurotransmitters that can have direct influence within the intestine, but can also bind to receptors on the vagus nerve and directly influence the vagal output to the brain.&nbsp; And this is really just a simplification because there are many other metabolites that microbes produce that have either not been fully characterized, or it's -- or just too extensive to cover.</p>

<p>So, I wanted to just touch a little bit on prior studies here that have looked at metabolites -- or metabolites in ME/CFS.&nbsp; And there are a range of studies that have looked at serum, plasma, and urinary metabolomics, and using different methods, different numbers of participants in each study.&nbsp; And I sort of summarized these findings to look at -- and looked at whether any of them -- any of the metabolites that were identified might have some relevance to the microbiome.&nbsp; And I sort of show on the right where there may be some links with microbiome.&nbsp; The study by Naviaux found 4-hydroxyphenyllactic acid reduced in males, and phenyllactic acid reduced in females.&nbsp; This is produced by bacteria.&nbsp; And it can affect the -- and it is actually a broad-spectrum antimicrobial.&nbsp; So, it inhibits the growth of bacteria and fungi.&nbsp; And it's produced primarily by lactic acid bacteria.&nbsp; They also found bile acid differences with chenodeoxycholic acid reduced in females.</p>

<p>The study by Germain found -- in addition to primary bile acids -- that there were some secondary bile acids that were changed.&nbsp; Some tryptophan metabolites along with kynurenine pathway were decreased.&nbsp; The study by Simonato also found tryptophan related products.&nbsp; Hoel found bile acid and branched-chain amino acids and tryptophan metabolites as well.&nbsp; So, there's some things that do show some agreement.&nbsp; I would also mention that the 2020 paper by Hanson's group found indole-3-lactate was reduced in ME/CFS, which is one of those specific metabolites that is produced by bacteria in the intestinal tract, and has been shown to bind to aryl hydrocarbon receptors to regulate immune responses.&nbsp; And so, those are some studies that were just primarily metabolomics, where there's, you know, a little bit of evidence that there might be an involvement of microbiome.</p>

<p>And then some of the more recent studies have begun to look a bit more closely at sort of that direct relationship between the microbiome and metabolomics.&nbsp; So, there were two studies that have used fecal metabolomics to look at metabolites present in feces.&nbsp; And those studies also looked in plasma and urine.&nbsp; They had somewhat limited range, primarily focusing on amino acids.&nbsp; And one of those studies also looked at bacteria, but used culture-based methods.&nbsp; But they did find that fecal bacteria and fecal metabolites correlated with many of the plasma and urinary metabolites.&nbsp; So, that sort of was the first suggestion that there may be a relationship between what's happening in the gut and what's happening in the periphery.</p>

<p>The study by McGregor also found low levels -- sorry, found that the percentage of fecal uracil correlated with the seven-day post-exertional malaise scores.&nbsp; And there's very little that has been studied about uracil in feces but -- in humans.&nbsp; But in Drosophila models, the good bacteria, the symbiont, don't produce uracil.&nbsp; And uracil is released from opportunistic pathobionts, which provokes chronic inflammation in that model.&nbsp; Then the study by Nagy-Szakal, which I was involved in, looked at plasma metabolomics and also conducted shotgun metagenomic sequencing on 50 ME/CFS subjects and 50 controls, and found various alterations in plasma metabolites for choline, carnitine, complex lipids, and ceramides.&nbsp; And they found that combining metabolomics and microbiome improved classification models of disease.&nbsp; And that several of the gut bacteria correlated with plasma metabolites.&nbsp; One of those was in -- that was included was indole-3-lactate, which I mentioned, the Germain group also found.</p>

<p>These more recent metabolomics study that we've done on our current cohort for the -- that was published by Che hasn't yet been integrated with the microbiome research that I'm going to talk about more today.&nbsp; But that study also found decreased indole-3-lactic acid in plasma.&nbsp; And then there have been other studies that have used 16S sequencing with different methods for metabolomics analysis.&nbsp; Some of them are limited by the number of samples, or the number of metabolites that they analyzed to really sort of understand how the plasma metabolite might relate to bacteria.&nbsp;</p>

<p>But one of the studies, even though they did 16S sequencing, only did metabolomics on five of the patients from each of the groups, which makes it difficult to really sort of synthesize those data.&nbsp; And then the two most recent studies, one by the Jackson group, Xiong, et al, and our own study, Guo, et al, used both plasma metabolomics and fecal shotgun metagenomic sequencing on all of the participants studied.&nbsp; The Jackson Laboratory had 149 ME/CFS patients with both short- and long-term disease and 79 control -- healthy control patients.</p>

<p>And we had 106 ME/CFS and 91 controls in our study.&nbsp; And both studies found indications for changes in plasma metabolites.&nbsp; And the study by the Jackson Lab looked specifically at both short- and long-term disease, and found that long-term patients had more severe metabolic abnormalities.&nbsp; And that for the bacterial differences, short-term patients showed more extensive microbiome differences.&nbsp; But it's important to note that even within the short-term patients, there were differences in the microbiome.&nbsp; Just not as extensive as -- sorry, even in the long-term, differences were found in the microbiome.&nbsp; And the microbes that were found between the two studies showed a high degree of overlap.&nbsp; And I'll talk a little bit more about those studies now.</p>

<p>So, this study, I've sort of put my acknowledgments upfront.&nbsp; We published this earlier this year in&nbsp;<em>Cell Host &amp; Microbe</em>.&nbsp; And this study was part of the Center for Solutions for ME/CFS.&nbsp; And the number of people, including the director of the study, Ian Lipkin, and a number of other people within the Center for Infection and Immunity, contributed to the study.&nbsp; And of course, the other clinicians who recruited patients and collected samples were a big part of this study as well.&nbsp; And the goal of the Center for Solutions study was to apply multi-omics analyses to gain new insights into immunological and pathophysiological mechanisms of disease and for biomarker discovery.&nbsp; This study actually had longitudinal sampling, collecting samples quarterly over a year.&nbsp; So, there are four timepoints from this study.</p>

<p>They -- the samples were also collected at five geographic sites, which I think is a big part -- a big support to the study because it can control then for geographic variation, at least within the U.S.&nbsp; And had detailed participant metadata collected.&nbsp; Samples were collected for stool, saliva, PBMCs, and plasma.&nbsp; Shotgun metagenomic sequencing, metabolomics, proteomics, and RNA sequencing were conducted.&nbsp; And the 106 cases and 91 controls were matched for the site of collection, the age, sex, BMI, and demographic index.&nbsp; And we assessed the microbiome with the shotgun metagenomic sequencing and looked for differences between ME/CFS in cases.&nbsp; But we also know that IBS is a common comorbidity within this group.&nbsp; That isn't really found in the healthy controls at the same prevalence.&nbsp; In fact, 35 percent of the ME/CFS patients that were recruit have self-reported diagnosis of IBS.</p>

<p>And when we looked at alpha diversity, we found that, initially, there was a difference between ME/CFS.&nbsp; But when we use regression -- generalized linear regression models and adjust for range of covariates, including IBS diagnosis, the site of sampling, sex, BMI, race, ethnicity, age, antibiotic use 6 to 12 weeks prior to sample collection, probiotic, and prebiotic supplement use, as well as the most commonly used medications that the patients were taking.&nbsp; That the finding of differences in alpha diversity was largely attributable to those individuals who have comorbid IBS diagnosis.&nbsp; We also then applied regression analyses.</p>

<p>And through all of our study, we controlled for the -- all of those covariates, for all of the factors that we looked at, because we wanted to be sure that what we were identifying was not confounded by any of those factors.&nbsp; That the true difference was between ME/CFS and controls.&nbsp; And so, using those sort of, you know, strict requirements, we identified 12 bacterial species that differed in relative abundance in ME/CFS.&nbsp; And many additional species were identified that were specific to the IBS group -- subgroup.&nbsp; And among those 12 bacterial species, we found that there was a deficiency in Faecalibacterium prausnitzii and Eubacterium rectale, which are the two primary butyrate-producers in the gut microbiome.&nbsp; So, that sort of drove a lot of the analyses that we did subsequent to that finding.&nbsp; And most of the other species were actually increase in ME/CFS.&nbsp; And some of these species overlap with findings from other groups, and especially with the Jackson Laboratory.&nbsp; And that led to us co-publishing both of our findings together, because it sort of showed that there is consistency between two independent studies in terms of bacteria and pathways that were identified.</p>

<p>The next step was that we wanted to see if we could understand or find evidence that the gut microbiome was deficient in its ability to produce butyrate.&nbsp; Using functional metagenomic analyses, we can essentially look at all of the genes that are in these pathways.&nbsp; And there are actually four pathways that bacteria can use to produce butyrate.&nbsp; And we found that, specifically, the Acetyl-CoA pathway that the genes in that pathway were all deficient in ME/CFS.&nbsp; And that was independent of IBS status, or any of the other covariates that we adjusted for.&nbsp; And as an additional measure, we also used qPCR to quantitate the terminal gene, the last gene, which is considered to be rate limiting in the production of butyrate along that pathway,&nbsp;<em>but&nbsp;</em>gene and found again that was deficient.&nbsp; That gene is deficient in fecal samples from individuals with ME/CFS.</p>

<p>So, functionally, we're seeing that there's this reduced capacity to produce butyrate.&nbsp; And so, then we use GC-MS quantitation to look at fecal butyrate.&nbsp; And also, I should mention that the Jackson Laboratory found also a reduced metagenomic capacity for butyrate production in their study.&nbsp; And then we directly measured the three short-chain fatty acids, acetate, butyrate, and propionate, and found that impact in fecal samples from ME/CFS subjects that was a reduction in butyrate, and also reduction in acetate, though not as extensive as what was found for butyrate.&nbsp; And we again applied regression models, adjusting for all of the covariates to confirm those findings in our study.&nbsp; And the Jackson Laboratory also found isobutyrate, metabolite of butyrate, was decreased in the plasma patients.&nbsp; But while our study focused heavily on the short-chain fatty acids because of their known role in gastrointestinal and host physiological processes, we also looked at sort of the global view of what pathways were enriched and deficient based on the gene content within the microbiome, and found that there were a range of additional pathways in addition to butyrate being deficient in ME/CFS.</p>

<p>The process of sulfate reduction was reduced in the bacteria overall.&nbsp; And pathways for mucus degradation and ethanol production, acetogenesis.&nbsp; And the use of sugars was increased in ME/CFS.&nbsp; But we can further sort of increase the granularity of our analysis by looking at metabolic modules within the microbiome.&nbsp; And there we found even more pathways, including differences in S-adenosylmethionine synthesis, which was decreased in ME/CFS.&nbsp; Glycolysis was decreased.&nbsp; Sorbitol degradation and urea degradation were decreased.&nbsp; And there were increased functions for lactate production, arabinose degradation, and as well as GABA synthesis.&nbsp; So, one of those neurotransmitters that bacteria can produce in the intestine, and also increased vitamin synthesis for vitamin K2, or menaquinone.</p>

<p>And then additionally, we looked at whether differences in bacteria may relate to symptoms.&nbsp; And we found that the bacteria that were associated with fatigue severity, based on the multidimensional fatigue inventory scores.&nbsp; We're inversely associated with Faecalibacterium prausnitzii and Lactobacillus rogosae.&nbsp; So, general fatigue, physical fatigue, and total scores for the MFI were significantly associated.&nbsp; And these are based on regression models that are adjusted for all of the covariates I mentioned.&nbsp; And then there were additional bacteria that showed specific associations with different dimensions of fatigue.&nbsp; And while this doesn't show causation, it provides an intriguing association for further exploration of treatments and development of models that support a causal pathway.</p>

<p>And of course, butyrate plays a role within the intestine in barrier function and in regulating intestinal immunity.&nbsp; But it also makes its way into the blood and through the portal vein and can reach the liver.&nbsp; And it has effects outside of the intestine on the liver, the enteric nervous system, muscles that can affect glycogenesis and fatty acid oxidation.&nbsp; In the brain, it can influence neurogenesis.&nbsp; It's also been shown to fortify the blood-brain barrier and can also have peripheral effects through binding to GPR receptors on immune cells.</p>

<p>So, the question of what we know, and what we need to know about microbiome metabolism in ME/CFS is the short answer, a lot.&nbsp; And we still don't know the primary factors that underlie discordant findings across studies for both metabolomics and microbiome.&nbsp; What confounding factors should be considered in study design and analyses?&nbsp; We don't really know the scope of microbiome metabolomic disturbances in ME/CFS.&nbsp; Because while we use targeted analyses of short-chain fatty acids, there has not been any global analysis of fecal samples for which we also have other -omics knowledge.&nbsp; The influence of microbiome metabolic disturbances on peripheral metabolic and immunological disturbances or vice versa.&nbsp; Because many of those, the gut axes to other organs and systems are bidirectional, meaning, that the gut can influence the brain.&nbsp; But the brain can also influence the gut.&nbsp; What's happening in the gut?&nbsp; And so, that complicates establishing causal link between things that are changed in the gut and things that are changed in other systems.</p>

<p>We -- there's also lack of multi-omics integration studies to look at the relationship between the microbiome and metabolites, what -- and to determine what gut axes might be most likely linked within ME/CFS?&nbsp; We also don't know how stable these disturbances are across time, or whether instability and these differences in the microbiome or the metabolites that produces might track with disease progression with symptom flares, stressors, such as the exercise tolerance test, clinically or biochemically defined subgroups.&nbsp; And whether microbiome and metabolic disturbances are cause or consequence of ME/CFS and its symptoms?&nbsp; So, we don't really have studies that have taken us from association to causation.&nbsp; And, you know, there are potential uses of model systems and experimental approaches that could provide better mechanistic evidence of causation?&nbsp; And ultimately, whether treatments targeting the microbiome could effectively alleviate symptoms?</p>

<p>And, you know, just some sort of general recommendations on studies aimed at microbiome metabolism.&nbsp; Have been proposed for other diseases, where the microbiome has been found to play an important role.&nbsp; And those general recommendations include longitudinal sampling, so that you can, you know, sort of track changes in microbes and metabolites over time.&nbsp; And look at their relation in terms of the clinical data or clinical presentation of disease.&nbsp; And within cohorts to have multiple sampling, including fecal sampling of the -- to look at the microbiome, as well as metabolites within fecal samples.&nbsp; And then there's a need for methods that allow for metabolomics and microbiomes.&nbsp; Data to be integrated and effectively interrogated because they're on different scales.&nbsp; And, you know, microbiome data is really compositional data.&nbsp; So, you have to use specific methods to combine this data type with other data types.</p>

<p>And then computational approaches that should include potentially confounding factors.&nbsp; And ultimately, the hope would be that identifying some of these metabolites from the microbiome could then be interrogated further.&nbsp; And in vitro and in vivo models to identify and confirm biological mechanisms or explore causal pathways.&nbsp; And so, those recommendations kind of drive what I would see as research priorities for microbiome metabolism studies in ME/CFS.&nbsp; Fecal metabolomics studies are lacking in ME/CFS.&nbsp; These could be either rationally targeted based on metagenomic or metatranscriptomics analyses.&nbsp; And this area could be expanded to both prospective studies or, in extant, well-powered, data-rich studies with banked sample repositories, where other omics datasets are already available for data integration.</p>

<p>And metabolomics studies in ME/CFS, with longitudinal single- or multi-omics datasets are also lacking in ME/CFS.&nbsp; Studies employing or developing in vitro and in vivo model systems to investigate mechanisms or phenotypes associated with the microbiome and metabolic disturbances found in ME/CFS.&nbsp; And preclinical models or clinical trials to assess the efficacy of metabolites, probiotics, prebiotics, or synbiotics that normalize microbial structure and function, and as well as studies employing or developing state-of-the-art computational modeling approaches for microbiome and metabolome data mining, integration, meta-analyses, and generation of informed predictions about the role of microbial metabolism in ME/CFS.&nbsp; And that's all.&nbsp; Any questions?</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Thanks so much, Brent.&nbsp; Yeah.&nbsp; There's a question in the Q&amp;A.&nbsp; Do you think there's any metabolic link between SIBO, which I don't know what that stands for, maybe you do -- in ME/CFS, if IBS is a factor?</p>

<p><strong>Brent Williams:&nbsp;</strong>So, it's possible that there is a link.&nbsp; So, in our study, we also -- I didn't mention it here.&nbsp; But we used qPCR to quantitate the levels of Faecalibacterium prausnitzii, as well as Eubacterium and Roseburia species, and found that those were quantitatively decreased in ME/CFS.&nbsp; But in doing that, we also quantitated the total bacterial load.&nbsp; And we found that in ME/CFS, there was actually an increase in the total levels of bacteria per gram of fecal material.&nbsp; And so, that could suggest that there is some level of overgrowth of bacteria in the intestine.&nbsp; And, you know, small intestinal overgrowth that is seemed with SIBO could be linked.&nbsp; But there could be additional explanations.&nbsp; It could be occurring primarily in the colon because there is -- butyrate has properties of inhibiting the growth of other bacteria.&nbsp; And so, if you have a reduction in the level of butyrate-producing bacteria, it may allow other bacteria to expand.&nbsp; And that -- you know, that occurs almost primarily in the large intestine rather than the small intestine.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Thanks, Brent.&nbsp; The question I had that occurred to me as you were speaking, and not specifically directed to you, but I guess all of the speakers too, is that all of the studies that are being described are in adults.&nbsp; So, what do we know -- I guess, specific to the microbiome.&nbsp; Have studies been done in children and adolescents with ME/CFS?&nbsp; Do they have IBS?&nbsp; Do they have the same sort of microbiome changes, or do we not know?</p>

<p><strong>Brent Williams:&nbsp;</strong>So, we don't really know.&nbsp; I believe there's one study in children that was small and what -- microbiome only.&nbsp; But I don't think that there -- that has been looked at extensively.&nbsp; You know, one of the -- one of the issues with microbiome is that it changes across the lifespan.&nbsp; And so, you might not expect that what you would see in children, would be the same as what you see in adults -- because, you know, in -- you know, from infancy to sort of early childhood, the microbiome changes.</p>

<p>And then it establishes a more adult-type microbiome.&nbsp; And then again, in elderly populations, the microbiome changes again.&nbsp; And so, that's why it's important in these studies to consider potential confounders like age, which we did in our study to sort of identify those bacteria that are specific to disease.&nbsp; But also, you can look for interactions.&nbsp; So, interactions between age and the microbiome in ME/CFS, or interactions between obesity and ME/CFS.&nbsp; We did actually find that there were, you know, significant associations for butyrate producers in individuals who were overweight.&nbsp; But it was controlled for in our analyses.&nbsp; And so, the actual changes in overweight or obese individuals were well above what was found in health controls.&nbsp; And so, it's suggested that the changes are potentially compounded in individuals with ME/CFS, who also have obesity.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Interesting.&nbsp; So, this is an interesting question.&nbsp; "Given the potential impact of household animals on the microbiome, have you come across any ME/CFS research studies that consider this factor by categorizing patients in those with pets, and those without?"&nbsp; Interesting question.&nbsp; I know that there was a study done by a student in -- one of Simon Carding's students in England, where they we're looking at microbiome in individuals with ME/CFS in going to the home and collecting microbiome from the individual with ME/CFS and relatives.&nbsp; But --</p>

<p><strong>Brent Williams:&nbsp;</strong>Yeah.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>I don't remember those results offhand.&nbsp; But I don't know if you've come across any of those studies.</p>

<p><strong>Brent Williams:</strong>&nbsp;But typically, people who reside in the same household tend to have a more similar microbiome.&nbsp; And the studies that have suggested that having a pet and -- specifically a dog, I don't think it's -- the same was found with cats.&nbsp; That can have an influence on the microbiome of children.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Okay.</p>

<p><strong>Brent Williams:&nbsp;</strong>So, earlier in life, having a pet.&nbsp; And that might relate more to the actual -- the instability of the microbiome in children.&nbsp; It's really fluctuating at that time point. &nbsp;And so, once an adult microbiome is established, you know, that becomes more stable.&nbsp; So, I don't think that they found the same thing in adults. But in children, they didn't.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Interesting.&nbsp; Okay.&nbsp; Thank you so much, Brent.&nbsp; I think we'll move on to the next speaker.&nbsp; Back to you, Alain.&nbsp; Thank you, Brent.</p>

<p><strong>Alain Moreau:&nbsp;</strong>Thank you very much, Vicky.&nbsp; Thank you, Brent, for your excellent presentation.</p>

<p><strong>Alain Moreau:</strong>&nbsp;So, our last but not the least speaker of today is Dr. Christopher Armstrong.&nbsp; So, Chris is well known for his research using metabolomics to observe biochemical alteration in ME/CFS patients.&nbsp; He began his work during his doctoral training at the University of Melbourne, where he is now a research fellow and director of Open Medicine Foundation Collaborative Center.&nbsp; His research program is centered on the application of metabolomics to study ME/CFS.&nbsp; And he published his first metabolomics study on blood and urine in 2015.</p>

<p>Since then Chris has set up several collaborative efforts to apply metabolomics to immunological experiments and other types of experiments on ME/CFS, observing how metabolism may relate to immune cell function, for instance.&nbsp; He has also focused on longitudinal research on ME/CFS while looking to extend metabolomic capabilities across the field to help collate different patient groups and subgroups.&nbsp; Chris?</p>

<p><strong>Christopher Armstrong:&nbsp;</strong>Thank you very much, Alain, for that kind introduction.&nbsp; And I wanted to thank you and obviously the other organizers for thinking of me and inviting me to give a talk here for NIH.&nbsp; I appreciate that very much.&nbsp; And I'm sure you've heard a lot about metabolomics here within this webinar today.&nbsp; So, yeah, I think, Alain, said a little bit -- I mean, a lot about my things that -- or the work that we've been doing.&nbsp; So, I don't really need to go a little bit more in introducing myself.&nbsp; But I'll --</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>You're muted, Chris.</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Oops.&nbsp; Is that okay?</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Yes, we hear you know.</p>

<p><strong>Christopher Armstrong:&nbsp;</strong>Okay.&nbsp; So, for this webinar today, I'm going to be talking about metabolomics in biofluids.&nbsp; And this is an area where we've conducted a number of research studies.&nbsp; But there's also been quite a few research studies broadly across the world.&nbsp; And I'm going to be focusing on a lot of people's work and trying to summarize a large majority of the type of work that's been done in this space.&nbsp; I've broken this up into three sections, the pre-metabolomics era.&nbsp; So, this is from about 1990, to 2014.&nbsp; The case-control studies, which were the first large metabolomic studies, looking between 2015 and 2020, roughly.&nbsp; And more recently, there's been a few repeat measure studies in this space.&nbsp; I'll be highlighting some untapped potential research in this area, as well as future study design considerations, and research priorities towards the end.</p>

<p>So, I'm going to give you a broad slide here.&nbsp; And I'm probably sure you've seen quite a bit of this.&nbsp; So, just quickly, metabolites are any small organic molecule, or chemical.&nbsp; And metabolomics is really the study of these.&nbsp; This is really a simple definition.&nbsp; But metabolomics came on the back of genomics, transcriptomics, proteomics, that cascade that was kind of popularized.&nbsp; And the idea was originally to create standardized methods of metabolomics, or ways to quantitate metabolites.&nbsp; That could then be utilized by people all over the globe to characterize different states of being in people or animals or plants or any different types of situations to understand the different signatures that could exist within those populations, during different periods when they're exercising, when they're eating, when they have disease.</p>

<p>And so, on the right here, actually, have an image that I took from a paper regarding exercise.&nbsp; And looking at metabolism and exercise, and specifically in that space, it really highlights how sensitive metabolomics is, because they also have a lot of issues trying to control for all different types of extraneous factors.&nbsp; They're talking about healthy people, trying to understand how metabolites change before and after exercise.&nbsp; But they have to deal with all types of heterogeneity and issues within their cohorts.&nbsp; Simply because people are different, and metabolomics is very sensitive.&nbsp; This is very much some of the things, and the themes that you'll see trending throughout this presentation.&nbsp; But at the end of the day, we do get some interesting findings that come out of this type of work.&nbsp; And there's always potential to improve methods.</p>

<p>So, we're going to start with, why don't we look at biofluids?&nbsp; Biofluids are mostly invasive -- non-invasive, I should say.&nbsp; Not cerebrospinal fluid, in this case, but blood plasma, blood serum, spot urine, 24-hour urine, stool samples, saliva.&nbsp; A lot of these are easy to collect.&nbsp; And so, there's a pragmatic part about this.&nbsp; It's useful for diagnostics or predictive biomarkers.&nbsp; I guess, in some ways, it's easy to get a study going, and analyzing these types of materials because of how non-invasive they are.&nbsp; But then the outcome of it is also clinically pretty useful for early screening, diagnostics, for pathology, labs, but also useful for objective or identifying objective measures and outcomes for your trials.&nbsp; And so, this is the target for a lot of research studies, not just in ME/CFS, but worldwide.&nbsp; And typically, blood is the main target for this type of study.</p>

<p>They provide a complex picture of the host.&nbsp; This is both a gift and a curse, because obviously, the cells themselves are really conducting most of the metabolism work.&nbsp; And the biofluids are providing nutrients and also removing excretory compounds, or things that those cells don't want.&nbsp; And that's all kind of combining.&nbsp; And so, you have many different types of cells with different types of metabolism in the host or in a multisystem organism.&nbsp; And so, you end up having a complex picture within a biofluid-like plasma or urine or any of these ones that I have listed here.&nbsp; So, you get a picture of all of these.&nbsp; But also, it's complicated by all of these.&nbsp; And in this way, a lot of this type of research is typically hypothesis generating.</p>

<p>So, first I want to start with going over kind of what went down during the pre-metabolomics era from metabolism work in ME/CFS.&nbsp; So, initially, metabolomics was driven by the standardization of collection of methods.&nbsp; And I said, picking on the back of -- piggybacking on genomics and proteomics.&nbsp; But metabolite studies do predated.&nbsp; People we're measuring metabolites.&nbsp; You'll notice that, in clinical labs, you're picking up metabolites if you go to a doctor's clinic.&nbsp; And so, initial studies attempted to link fatigue and PEM to exercise physiology.&nbsp; And a lot of these studies existed. &nbsp;And in combination, they would often try and measure things like glucose or lactate or other compounds in the blood.</p>

<p>I guess when ME/CFS was first defined back in the late 1980s, or at least to a level that we had enough people and caught a lot of interest, globally, there was a push in this type of area.&nbsp; That's why you might recognize a lot of exercise physiology, or exercise physiologist, here trying to treat patients.&nbsp; I think a lot of the initial work was done in this area by exercise physiologists. &nbsp;And, effectively, when we started to do metabolomics work, we thought we'd first try and do a comprehensive review.&nbsp; And this was part of my PhD, actually, back in -- started my PhD back in 2014.&nbsp; We conducted a comprehensive review of all studies prior to 2014, that characterized metabolism in ME/CFS.&nbsp; And so, this is the reference for that.&nbsp; There's really a book chapter.&nbsp; It's a large review back.&nbsp; Worth giving a read if you want to know what happened in the past.&nbsp; But I'm going to give a brief rundown of some of the main findings.</p>

<p>Reduced amino acids was something that popped out within this period of time.&nbsp; A lot of studies were focused on urine at this point.&nbsp; Interestingly, because since then really no one's touched it.&nbsp; But reduced excretion of amino acids in the urine was a key finding.&nbsp; And a few amino acids were reduced in blood samples over this period of time.&nbsp; Another finding that was popular was, probably, common was this reduced oxygen consumption.&nbsp; Again, this was by the exercise physiologist looking CPET&nbsp; studies, which is cardiopulmonary exercise testing.&nbsp; And looking to identify reduced, or the way that patients used oxygen, their aerobic capacity.&nbsp; And they identified that there was reduced oxygen transport and/or utilization by ME/CFS patients.</p>

<p>This group down the bottom here, Snell and also Van Ness, they did a number of different studies.&nbsp; Looking at CPETS -- well-characterized CPETS in ME/CFS.&nbsp; And they account for some of the deconditioning issues.&nbsp; They actually identified the idea of realizing the two-day CPET, which was an issue initiated with significant drop off in oxygen consumption in patients in the second day.&nbsp; And this is quite a famous study.&nbsp; I think a lot of -- everyone will probably know about the two-day CPET.&nbsp; I believe it's utilized there in the U.S. to help individuals apply for disability, to show that they have exertional tolerance.&nbsp; And so, this was a very clever research design.&nbsp; And I think we're going to talk about a study later on in the repeat measure study, that has conducted metabolomics in this similar design.</p>

<p>Finally, oxidative stress markers.&nbsp; This was common back in the early 2000s.&nbsp; And I think for the last 20 years, we've had a bunch of oxidative stress hypotheses.&nbsp; And not moving, the oh, no hypothesis.&nbsp; The idea of oxidative stress and nitric oxide, combining peroxynitrite and creating significant stresses in ME/CFS patients.&nbsp; But also, generally, oxidative stress has been found in this patient's blood samples for a number of years.&nbsp; These markers were peroxidative lipids, elevated isoprostanes return.&nbsp; Absolute peroxidative lipids, I guess, and oxidized now, you know.</p>

<p>So, there were a number of markers on this area, and I think in combination, these were really the three main physiological or interesting findings that carried across multiple studies back before 2015.&nbsp; So, then we moved on to case-control metabolomics.&nbsp; And so, the idea here is ME/CFS versus healthy controls.&nbsp; And these types of studies were really set up to identify critical biomarkers or indications of pathogenesis.&nbsp; If you're taking an ME/CFS population, looking for its control, you really want to try and identify if there is a marker that is 100 percent diagnostic or very close to being clinically useful, as well as what separates those two groups to try and get an idea of what was happening.&nbsp; And obviously, we know that ME/CFS, not a lot of biological information was known about it.</p>

<p>And so, trying to capture anything we could was really the main idea behind a lot of the research that happened during this period, between 2015 and 2020.&nbsp; I mean, on -- it's still ongoing.&nbsp; People still do case-control metabolomics.&nbsp; But a lot of this initial work was done then.&nbsp; And it's all really been done in blood, I should mention.&nbsp; Well, the original idea was to have these two groups in a case-control study.&nbsp; And you want them consistent really in every single way, except for the fact that one -- another group has for the disease.&nbsp; It doesn't really work out that way.&nbsp; Because there are a lot of complications with the disease group.&nbsp; We were having a disease that makes people quite different from other ones.&nbsp; That metabolomics wouldn't actually pick up.&nbsp; Those have a lot of variety.</p>

<p>Effectively, we may imagine that looking at ME/CFS works healthy controls, may look something like this.&nbsp; But even still, it's actually a bit more complicated than the variety here that we see.&nbsp; People vary and because of this, sample sizes really do matter for metabolomic case-control studies.&nbsp; And it's possible that we haven't been utilizing large enough samples to do the type of research we're really looking for.&nbsp; So, I'm going to start to talk about our papers.&nbsp; And I think this will be it for talking about how papers, and the rest will be other people's research.&nbsp; But I wanted to mention this because we -- previously, we published the first comprehensive metabolomics papers in ME/CFS and healthy controls, or the first metabolomics application back in 2015.&nbsp; And then we had follow-up papers.&nbsp; And really, overall, there was a lot of study where we looked at blood, urine, and stool in quite a number of ME/CFS and healthy controls.</p>

<p>Now, the outcome of that was that we did find or hypothesize increased amino acid catabolism, which was fairly similar to what they previously found.&nbsp; And as you'll see in the future, a lot of people have found this following up.&nbsp; We hypothesized it was through this thing called GOT2 and GDH.&nbsp; Both of these utilize -- enzymes that utilized glutamate and provide glutamate to the TCA cycle for energy production through the mitochondria.&nbsp; We also have oxidative stress markers.&nbsp; These was identified as low hypoxanthine and elevated allantoin through an enzyme called xanthine oxidase.&nbsp; And typically, it happens when you have an increase of oxygen in the environment.&nbsp; You go through these.</p>

<p>You transform&nbsp; that to allantoin and produce reactive oxygen species.&nbsp; So, that was one way in which we could identify reactive options species be produced.&nbsp; We also saw an altered production of purines, which are pretty critical, obviously, for genes.&nbsp; But also, adenine is the basis for adenosine and ATP, AMP as a carrier of energy -- or as a carrier of potential energy, I should say.&nbsp; And then you have the host metabolism.&nbsp; Generally, overall, what we kind of both sides was the host metabolism was altering the gut metabolism and creating a cycle.&nbsp; So, what I have here on the image is just kind of a summary of what we thought was happening overall, within this population between the blood, urine, and stool samples that we caught.&nbsp; And so, we identified an increased usage of amino acids for energy production.&nbsp; On this little chart here, you can see that there's energy per molecule and energy per second.</p>

<p>You have reduced efficiency with amino acids per molecule and per second, compared to glucose through the mitochondria and definitely through glucose through glycolysis.&nbsp; Do you have a significant reducibility to make energy per second using amino acids and fatty acids?&nbsp; So, this is an important point.&nbsp; And we could explain some of the reduced oxygen component or take up that we have been seeing in a number of different studies.&nbsp; But also, the reducibility to make energy per second really translates to physiological findings of fatigue, we believe.&nbsp; But the outcome of that is also that you've reduced cell proteins and cell fats.&nbsp; And this is well-characterized in a lot of studies where they may starve people of amino acids, or general starvation as well.&nbsp; And you end up, one of the first things that goes is the production of enzymes and bile acids.&nbsp; And so, the outcome of that is you have less enzymes, and bile acids going into the small intestines to break down those lipids and proteins that you have there.</p>

<p>And so, what happens is those larger molecules get deeper into the gut and end up interacting more with the large intestines.&nbsp; And you can, I guess, provide a substrate for a new type of bacteria, particularly scavenging ones, that are able to break down lipids and proteins in this environment to short-chain fatty acids.&nbsp; And so, the outcome of that is that you get more short-chain fatty acids production.&nbsp; But you get less absorption, I guess, of fatty acids, amino acids, directly to the blood.&nbsp; And you get a different overgrowth or changing in gut bacteria.&nbsp; This was our hypothesis, generally, from what we identified.&nbsp; And it'd be cool to see some follow-up papers in this area.</p>

<p>So, also, I wanted to focus on the idea of amino acid catabolism, because this was found by a number of different studies.&nbsp; But I think one study in particular was extremely focused on this area.&nbsp; And this was the one by Fluge and Mella in Norway.&nbsp; And they looked at a number, or maybe 100 patients, 100 controls.&nbsp; And they found that amino acids were generally depleted in this population.&nbsp; And they hypothesized that amino acids were being utilized for energy production, very similarly to us.&nbsp; I don't know even if they saw our paper before they came to that conclusion.&nbsp; So, it was interesting that we came to a similar idea.</p>

<p>There were also a number of studies that highlighted GOT2.&nbsp; I think any studies looked at cells have found an upregulation of this particular enzyme, which enables glutamate to be utilized or upregulated.&nbsp; Its usage of glutamate into the TCA cycle by mitochondria.&nbsp; So, to me, this is one of the stronger findings within this field.&nbsp; That amino acids are being used more readily for energy production in ME/CFS patients.&nbsp; There were also a number of lipid anomalies.&nbsp; So, we have this paper here by Naviaux, et al, but also Arnaud Germain and Maureen Hanson's group at Cornell.&nbsp; And then also, we have the Columbia group, who had a number of different -- so, these guys had between them quite a few different metabolomic case-control studies.&nbsp; And typically, I think, they found lipid anomalies.</p>

<p>I think there was some disagreement regarding phospholipids and the sphingolipid changes.&nbsp; But overall, I think, generally, we find a reduction in phospholipids across multiple studies.&nbsp; I think they do align more with Acyl choline lipids being decreased, and an increase in potential triglycerides.&nbsp; And most of these studies -- they come to a similar conclusion.&nbsp; This idea that perhaps general catabolism of lipids and proteins was upregulated to produce ATP.&nbsp; And it would be a suitable explanation generally for what we identify.&nbsp; I say generally because at the end of the day, a lot of these studies add some clues about the disease process but didn't validate specific biomarkers or metabolites that were consistent across all the different studies.&nbsp; And this might have been jarring, I think.&nbsp; But just know that this sort of thing is actually quite difficult to find even studying, exercises.</p>

<p>As I mentioned, at the start, it's hard to find consistent markers. And typically, in metabolomics, I think we become very good at understanding that the pathway is upregulated and downregulated.&nbsp; And the areas that have been impacted seem to be providing more of a clue generally what's going on.&nbsp; And this -- really, disease control versus disease group looks -- ends up looking a little bit more like this.&nbsp; His ME/CFS population is a little bit more heterogeneous than I thought.&nbsp; We may be first envisaged back in 2015.&nbsp; And -- but it kind of makes sense when you think about this ME/CFS population.</p>

<p>The way that they can vary with triggers and onsets, and the way that they described their triggers and onset.&nbsp; But symptoms also vary in symptom type, vary in severity, fluctuate over time, or can be persistent within individuals.&nbsp; Then you have variations in functionality and, of course, comorbidities.&nbsp; And this isn't even really mentioning medications or treatments that they may have been on.&nbsp; But comorbidities contribute quite significantly to a lot of the heterogeneity as well within this population.&nbsp; And so, when you're looking at metabolomics, really, you're looking at the combination of all these factors in one person.&nbsp; And they might be quite different from someone else within that same cohort.</p>

<p>And a lot of the findings may be kind of overlaying each other, which makes it very complicated.&nbsp; And I think this moved towards clustering or sub-typing.&nbsp; I think a couple of papers have kind of looked at doing this.&nbsp; And recognizing ME/CFS as potentially an umbrella of multiple pathologies, typically allow us to better-defined clusters to be uncovered and helps to try and work out how to characterize individual groups in different ways in which ME/CFS patients may be presenting within metabolomics.&nbsp; It opens up, I think, a path more towards precision medicine as well.&nbsp; So, typically, generally, I think, from this research so far, I would say the untapped potential within this area is looking -- considering that we have this heterogeneity, it is still possible that a signature of metabolites might be uncovered that can aid differential diagnosis.&nbsp; But I think the way to do that is significantly larger cohorts.</p>

<p>We ourselves are actually soon to publish a study on a large cohort with several disease controls.&nbsp; Lead author is Kathy Huang, and this is a U.K. Biobank study looking at metabolite markers within blood within ME/CFS population, and a number of comorbidity population.&nbsp; There are also several biofluids that are yet to be well metabolically characterized in ME/CFS.&nbsp; Most of this work has been done on blood.&nbsp; But things haven't really been worked much on urine, very little on stool, really nothing on saliva, cerebrospinal fluid, exhaled breath condensate.&nbsp; There are a lot of different biofluids potentially -- that have potentially useful information for ME/CFS.&nbsp; And so, current studies in these spaces, I think, should be encouraged.&nbsp; As -- and the last thing is, I think combining metabolomics with physiological measures in ME/CFS would be something that should be more common.</p>

<p>So, the idea is that you're trying to characterize the disease, in this case, ME/CFS.&nbsp; That is defined a lot by subjective symptoms and -- or symptom expression in combination with working with the GP to diagnose and define that condition.&nbsp; And a lot of the time, it is very difficult, even in any different types of study, to relate symptoms to biological data.&nbsp; Regardless of the disease, it's a complicated thing to do.&nbsp; And typically, characterizing a physiological measure.&nbsp; Something, like say, define reduced oxygen transport or reduced oxygen utilization or preload, failure or some type of other physiological finding, you try and characterize that with metabolomics within that ME/CFS population.&nbsp; I think, there are a lot more interesting information from something like that.</p>

<p>So, repeat measure metabolism, or metabolites with papers have become more and more popular recently.&nbsp; Just because metabolites vary rapidly moment to moment.&nbsp; They aren't just really a reflection of how the person's disease but is going -- but a reflection of what they just ate, how much they just walked, the environment.&nbsp; Capturing the changes of these metabolites in individuals over time is an ideal study design for understanding ME/CFS.&nbsp; Because ME/CFS patients do have these fluctuations, there's variations.&nbsp; And they have their reaction to certain stresses, is can be somewhat abnormal compared to healthy.&nbsp; So, the fact that metabolomics is so rapidly changing and sensitive to this, makes it actually quite an ideal area to look at ME/CFS patients.&nbsp; But it just requires thought on study design, and study design of repeat measures can happen over minutes.</p>

<p>You can do taking samples minute to minute, or you can take samples over days.&nbsp; You could take samples, one sample one day.&nbsp; And then sample a month later or a few months later or a year later to look at long-term change.&nbsp; So, different areas, and I think there's a lot of potential in the area of repeat measure metabolism.&nbsp; I mentioned I talked about this before.&nbsp; But this was some great work coming out of Maureen Hanson's Lab at Cornell with Arnaud and Katherine, producing a couple of papers in this area.&nbsp; And the idea here was kind of tracking similar to what a two-day CPET was doing.&nbsp; But track capturing metabolic changes that happened before and after exercise on two separate days.&nbsp; I think this is a really great study design.</p>

<p>And they were able to get the delta change, or the difference of what was occurring in exercise on those two days, and capture it here in this diagram on the bottom left.&nbsp; You have controls and ME/CFS patients.&nbsp; And then you see that coinciding with this reduced oxygen transport or reduced oxygen utilization that other previous studies have identified, they found significant changes in what looks like amino acid metabolism.&nbsp; Largely -- I think this paper largely focused on glutamate.&nbsp; But they have a lot of pathways, and here around general metabolism of amino acids for energy production, which was really fascinating.&nbsp; That follow-up study on urine also identified abnormal recovery.&nbsp; They see changes in the urine that would typically be in controls following exercise.&nbsp; But those changes, weren't as explicit in ME/CFS, which suggests that perhaps maybe there is some type of altered risk recovery response or a delayed recovery response occurring in this population.</p>

<p>So, there's certainly a lot of untapped potential in repeat measures areas.&nbsp; There's only been a few little studies in this space.&nbsp; But we are definitely working on a number.&nbsp; And I know a number of groups around the world are working in this space.&nbsp; This is probably the future of metabolomics research on biofluids.&nbsp; Further studies before and after exercise to characterize recovery in ME/CFS.&nbsp; I think that's necessary.&nbsp; But also characterizing metabolic changes in reaction to other stressors, non-exertion, some type -- in response to food type or diet or eating generally or cognitive tests, different types of stresses that may be impactful for these patients.&nbsp; Identifying metabolite profiles in individuals during good and bad days.&nbsp; It could be something useful to develop objective metabolite markers of disease improvement.&nbsp; This would be really critical for clinical trials.&nbsp; Something that we desperately need these objective markers with those clinical trials, to measure improvement.</p>

<p>And patient tracking for clustering and personalized medicine, I think is something what a lot of people think about when they think of repeated measures study.&nbsp; The idea of following patients and looking at how they're reacting to treatments over time, and developing predictive markers for other patients in the future that look similar.&nbsp; I think some future design considerations we need to consider, is deconditioning a factor that we aren't accounting for?&nbsp; And how do we account for this?&nbsp; I mentioned this because some of these things, like reactive oxygen species, elevation, reduced efficiency of ATP production from mitochondria.&nbsp; These things are to push -- can be -- these are produced in deconditioning study.&nbsp; Simply getting people to stop exercising for six weeks drastically increases reactive oxygen species and depletes energy efficiency from mitochondria.</p>

<p>And so, we need to start working on controls that we're utilizing for our studies to make sure that we're capturing this well.&nbsp; And understanding what we're really looking at, is whether it's part of the disease process, or whether it's an outcome of the disease process by reduced functional capacity.&nbsp; I think we need to also think about how do ME metabolic profiles compare to other diseases?&nbsp; So, again, thinking about positive disease controls here, and what differentiates ME metabolic profiles from other diseases that are similar, also quite different.&nbsp; I think also understanding whether the data that we're collecting is comparable across all the different studies.&nbsp; And this might require some getting together and agreeing on some central methods that we utilized.&nbsp; Because the way that you acquire metabolomic data can be variable based on from the tubes that you use to collect the blood, to the way you spin it down, to the way you prepare for analytical technique.&nbsp; There are a lot of things they could vary.&nbsp; But there are ways you can kind of correct for variations if you want to go there.&nbsp; Typically, utilizing things like internal standards really do help.</p>

<p>Are we accounting for heterogeneity between people?&nbsp; Do we have the sample size of this that are really large enough to account for variations that we keep on seeing within this population?&nbsp; I think these are questions and things that we should be thinking about for future research designs.&nbsp; And finally, I just wanted to highlight and summarize some of the research priorities in this space.&nbsp; So, I think large-scale diagnostic biomarker projects, talking on the realm of hundreds to thousands of patients and controls.&nbsp; Validating those markers, I think that's obviously a significant research priority in an area that hasn't been really touched on.&nbsp; Objective metabolite markers of disease improvement.&nbsp; Again, this is really important for clinical trials, and something that I think could be potentially pulled out of metabolism, or metabolomics on biofluids.&nbsp; Characterizing metabolic changes that happen in reaction to different types of stresses.&nbsp; I think this is a research priority to understand what's going on here.&nbsp; To understand or better characterize patient reaction to exercise or diet or whatever.</p>

<p>Connecting metabolome to physiological measures and function, as I already highlighted here.&nbsp; The idea of trying to characterize physiological changes that are occurring within patients could actually put some benefits.&nbsp; And working more in that phase of patient tracking for personalized medicine, following patients, understanding individuals, and looking if you can produce treatments directly for individuals based on your deeper understanding of that individual at that time, as well as clustering for different treatments.&nbsp; I think these avenues are the major research priorities.</p>

<p>Thank you very much, everybody, for listening to the talk.&nbsp; And I want to acknowledge, obviously, the large list of collaborators that we work with.&nbsp; University of Melbourne, where I do most of -- do all our work; the Australian government; and of course, Open Medicine Foundation for supporting a lot of the research that we do, but also supporting the collaborative network of researchers that we collaborate with.&nbsp; Thank you very much.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Thank you very much, Chris, for an excellent presentation and for joining us from Australia.&nbsp; Thank you so much.</p>

<p><strong>Christopher Armstrong:&nbsp;</strong>Pleasure to be here.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>So, I see there's a couple of questions here for you.&nbsp; So, "Are adding amino acids or acidic fluids like vinegar, small-chain fatty acids, to the diet likely to help people with ME?"&nbsp; So, supplementing those kinds of things.</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Yeah.&nbsp; So, that's interesting.&nbsp; I think people have trial this.&nbsp; I think some clinicians have trial this.&nbsp; And some, I think, do this.&nbsp; I'm not sure if all of them do it, if they all find benefit.&nbsp; It's hard for me to comment on something like this being that's -- I'm not a clinician.&nbsp; And we haven't done a trial on the benefits of utilizing that.&nbsp; But I believe, yeah, if you ask questions or if you look around, I think within asking other patients, I think you would probably find that a few utilize this to some improvement.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;There's a question here too.&nbsp; And I was going to follow up on this point as well.&nbsp; So, the question is, "Can you clarify your point about deconditioning?"&nbsp; Is a concern about deconditioning in individuals with ME, or having matching controls and/or both?</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Right.&nbsp; So, yeah.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;So, having the question.&nbsp; But I think that's what they're getting at?</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Yeah, definitely.&nbsp; That's a good question.&nbsp; So, yeah, it's the matching of controls to try and make sure that's not an extraneous factor.&nbsp; Obviously, we know when you have a chronic illness, especially this one, as debilitating as it is, that reduced functional capacity.&nbsp; You're not going to get out.&nbsp; There is much as maybe a general healthy control that's being compared as a population.&nbsp; So, that consideration of the healthy control population.&nbsp; Looking for people with a sedentary lifestyle, I think would be a great start for this.&nbsp; Because simply looking at healthy controls and get studies that look at healthy people and get them to stop exercising for six weeks, make dramatic differences for increasing reactive oxygen species and reducing mitochondrial efficiency to get 50 percent reductions, I think, in complexes 1to 4 and citrate synthase and stuff like that.</p>

<p>You also have when -- on the flip side, when they're exercising for six weeks, you get dramatic improvements in these areas and efficiencies. &nbsp;So, obviously, not patching for exercise as a therapy.&nbsp; I'm really just talking about the fact that when we're doing these studies, we need to consider the controls that we're utilizing.&nbsp; So, we can get a deeper understanding of exactly what's going on and what's not an outcome of the pathophysiology.&nbsp; But something that is the pathophysiology, we probably need sedentary controls.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Yeah.&nbsp; No.&nbsp; I've thought about that myself in terms of studying the very severely ill, bed-bound individuals that, you know, really are getting very little, if any, exercise at all.&nbsp; And what are the appropriate controls for that group of individuals with ME?</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Yeah.&nbsp; That's right.&nbsp; That's a tough one.&nbsp; Because even if we -- even if we went for hundreds of samples.&nbsp; Say, usually, you might go for hundreds of patients and hundreds of control to try and separate out variation.&nbsp; You are going to get a bias in the ME/CFS population, and especially at the severe population of that particular element, which is lifestyle factor of where you actually inhabit your life and what you're doing physically.&nbsp; So, those things do matter when it comes to the metabolism, metabolites projects.&nbsp; So, I think simply having disease controls is a good way to work around this, and having other people with different chronic illnesses that may be well-defined in other capacities as a control for these types of study should really be strongly encouraged.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;There's a question here now, "Is there any advantage to extracting and testing lymph fluid in individuals with ME/CFS?"&nbsp; Lymphatic fluid?</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Yeah.&nbsp; I mean, I think, obviously, any type of tissue or biofluid that you could extract would be very beneficial to study to characterize what's going on.&nbsp; I mean, they're all going to have unique variations.&nbsp; And a lot of the studies, as I mentioned initially, the metabolomics was set up to identify biomarkers.&nbsp; If you're going to do a ME/CFS with the heterogeneity of it versus the healthy control population, what you're looking for is a really robust biomarker that will pop out.&nbsp; And that will pop up if there is something specifically marking the disease.&nbsp; That will pop out in something like that.&nbsp; But it may not be that marker is in the biofluids we've been looking at.&nbsp; So, it's always -- if there's a biofluid that hasn't been looked at, if you can get your hands on it, and then run as many different types of tests on it as possible, I would definitely -- I would definitely encouraged that.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Another question for you.&nbsp; "Do you see value in monitoring body temperature in CFS, as a biomarker of metabolic change at rest versus exercising, or in response to treatment?"</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Yeah, definitely.&nbsp; I think you'd have to do it in conjunction with some of these other tests to know that what you're looking at is a function of basal metabolic rate, and not another alteration because of a thermoregulatory issue, that we know that ME/CFS patients can have, perhaps produced by hormones or autonomic issues.&nbsp; So, I think if you did it in combination with one of those, that'd be actually an excellent thing to occur.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Another question.&nbsp; "From your extensive experience in metabolomic studies, which common repeating metabolite -- or metabolites do you believe has the potential to shape the future of ME/CFS, or lead to the development of novel medications, treatments?"</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Yeah.&nbsp; I really think -- I think probably the oxidative stress markers and amino acid metabolism, generally, and nitrogen metabolism are the areas of most interest for me, I think.&nbsp; Yeah.&nbsp; For future.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Great.&nbsp; So, those were all the questions that came in.&nbsp; Any other questions from the panel or from anyone for Chris?&nbsp; Okay.&nbsp; If not, at this point, I'd like to ask all the panelists to turn their cameras on.&nbsp; I'm going to ask Chris and everyone, actually, this question that came in from Peter.&nbsp; So, metabolism not being my area of expertise.&nbsp; There's so much information.&nbsp; It's kind of a little bit hard for me to wrap my head around all of it.&nbsp; But that's my editorial, not the question.&nbsp;</p>

<p>So, "How do you think about the primary drivers?"&nbsp; The question that came in from Peter is "How do you think about the primary drivers of ME/CFS symptoms?"&nbsp; We can see how metabolic changes can cause immune system responses and vice versa.&nbsp; Metabolic dysfunction can cause neural dysfunction, et cetera.&nbsp; In your opinion, do you think metabolic dysfunctions are primary causes, or secondary effects that then generate ME/CFS symptomology?</p>

<p><strong>Christopher Armstrong:&nbsp;</strong>I'm happy to open with --</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Sure.</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;-- what I think and then obviously other people.&nbsp; But I think trying to find -- I guess, it's like the chicken and the egg kind of a question.&nbsp; And I think the way we think about it is a cycle.&nbsp; So, these things are all happening in coordination with each other.&nbsp; And they do somewhat feed into each other as well.&nbsp; And I guess, creating an illness or a phenotype requires lots of different systems working in a similar way.&nbsp; And it's not something -- the body is pretty stable.&nbsp; Even though it's changing, it's pretty stable.&nbsp; If it was simply that you could change all these systems with a simple path or changing a few metabolites every time you ate something, or did something, you would have dramatic changes and effects in the way that you feel constantly.</p>

<p>So, it's actually pretty stable.&nbsp; And the interaction between metabolites and all these other factors, I think more of like a cycle.&nbsp; And it's -- the cycle is sort of moving, right?&nbsp; And that's the change, you know.&nbsp; It's not from one thing to another.&nbsp; It's the cycle of these kind of interacting and moving across like that.&nbsp; And if you think of it like that, that's -- it kind of makes more sense of how these things interact together, and not necessarily causative of each other.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Anyone else want to comment on that?&nbsp; Derya?</p>

<p><strong>Derya Unutmaz:</strong>&nbsp;So, maybe, I think -- I mean, that's a great question.&nbsp; But maybe we need to divide the metabolomics into sort of intrinsic and extrinsic metabolites.</p>

<p>As Shuzhao mentioned, you know, we find a lot of xenobiotics, or you can tell the zip code of a person based on their metabolomics.&nbsp; So, clearly, environmental microbiome produced metabolites would be -- if they are, would be a primary driver.&nbsp; But then as Chris was saying, that could drive a cascade of events, where if you have the susceptibility, now you're going to trigger certain metabolic pathways, which in turn produce other metabolites.&nbsp; Which then could make things worse, or sometimes better.&nbsp; So, I don't know.&nbsp; I'm not an expert in the field.&nbsp; But I wonder if we could somehow compartmentalize the metabolomics.&nbsp; And then maybe this question would make more sense.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Anyone wants to comment?</p>

<p><strong>Shuzhao Li:&nbsp;</strong>Yeah.&nbsp; I would like to disagree on that, the cycles.&nbsp; So, I feel Peter asked probably the most important question here.&nbsp; So, I went through our own data.&nbsp; So, the long-term patient look was different.&nbsp; Sort of within the short-term patient, we see a lot of connection between the immune and the metabolite about it quite often, your dynamic sense.&nbsp; So, I feel that the readout from our immune profiling, or in the field in general, had been word limited.&nbsp; So, there, I haven't seen enough evidence to rule out autoimmunity or other causes.&nbsp; If the immune system is active all the time, it is where energy consuming.&nbsp; So, immune system, and while most expensive part in our energy consumption.&nbsp; That can certainly cause a lot of subsequent events.&nbsp; I feel that unless we have good data to curve -- the early patients, we may not have a clear answer.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Thank you.&nbsp; Any other comments to that?</p>

<p><strong>Derya Unutmaz:&nbsp;</strong>Just maybe to elaborate a little bit more on that issue.&nbsp; Shuzhao, I think what you're trying to say is that the chronicity of the situation suggests that things are just building up over the years.&nbsp; So, that kind of rules out the cascading event, which you would expect to happen in a much quicker pace.&nbsp; Isn't it?&nbsp; Did I understand that correctly?</p>

<p><strong>Shuzhao Li:&nbsp;</strong>Yes.&nbsp; That part of it.&nbsp; So, I'm sure there is causality somewhere.&nbsp; It's just that we haven't found it.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Okay.&nbsp; There are a few questions that have come in.&nbsp; "So, no one has talked about the unknown metabolites that turn up in studies and are shown to be significant.&nbsp; How do we identify what those are?&nbsp; And if they are important in ME/CFS?"</p>

<p><strong>Shuzhao Li:&nbsp;</strong>Okay.&nbsp; I'm going to take a stab at it.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Sure.</p>

<p><strong>Shuzhao Li:&nbsp;</strong>So, I know metabolites.&nbsp; We report it.&nbsp; From metabolome data, actually, have a certain level of allocation.&nbsp; There's just not completely identified.&nbsp; That's why with the area, I really believe that science has to have open data.&nbsp; So, if we do not have raw data from metabolome, it makes things so difficult.&nbsp; We can look into the raw data.&nbsp; So, there are many ways we can pull up to do additional identification on the compounds.&nbsp; Of course, not an easy problem.&nbsp; So, we've spent many years actually just trying to enhance identification in the metabolomic field.&nbsp; But for any high priority targets, I think we have enough method to get there.&nbsp; As they're already -- we need to make the data shared across researchers and prioritize compounds.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Yeah.&nbsp; I was struck your slides showing how few of the metabolites the metabolome platform is actually measuring.&nbsp; That was quite striking to me.&nbsp; There is a data sharing platform, mapMECFS, that was put in place by the ME/CFS Data Management Coordinating Center.&nbsp; It's just mapMECFS.org, where currently the majority of data that's on there is published data, curated from publications.&nbsp; However, there's a lot of data being uploaded from the centers that some is public, some is still in a private setting until it gets published.&nbsp; But I think that will become a really valuable resource for sharing data going forward.&nbsp; Derya, you have a comment?</p>

<p><strong>Derya Unutmaz:</strong>&nbsp;Yes.&nbsp; There's comment, question actually along the same lines.&nbsp; Yeah, I mean, I totally agree.&nbsp; There's a whole space of metabolites that we don't know.&nbsp; But the question also is, how many of those are really important?&nbsp; How many are byproducts?&nbsp; How many do have biological effects?&nbsp; I don't think we understand that.&nbsp; And that might be really important if you want actionable targets.&nbsp; It's also a question.</p>

<p><strong>Shuzhao Li:&nbsp;</strong>Derya, I have some answer to it.&nbsp; So, in our experiment using either bacteria cells or human cells -- so, we have a large amount of data that do not match to the non-metabolite.&nbsp; But when we introduced stable isotope tracer there, we note 20 to 30 percent of them to carry the isotope label. That means they participated in the active metabolism.&nbsp; So, I would think that they are important.</p>

<p><strong>Derya Unutmaz:</strong>&nbsp;I mean, I wonder if they could integrate that information to the database that you mentioned, Vicky?</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Yeah.&nbsp; Absolutely.&nbsp; So, I think we have time maybe for one more question here for the discussion by the panel.&nbsp; "So, for clinical use as a diagnostic biomarker, is it realistic to think that a signature gathered from global measurements could be used, or would it have to be one or two factors measured using traditional measurements?&nbsp; So, is there any precedent for these types of global measurements being used for clinical diagnosis?”&nbsp; So, I think this really gets at the basic question of how do we get to a diagnostic for ME/CFS?&nbsp; And Jiabao, I don't know if you want to take a stab at that first.</p>

<p><strong>Jiabao Xu:</strong>&nbsp;Yeah.&nbsp; I think finding, like, universal biomarkers is the goal.&nbsp; But then I think it's really difficult to do.&nbsp; So, because with the Raman fingerprints I see, like, with hundreds of Raman peaks, we get one diagnosed result.&nbsp; That could be same for other approach that measuring one molecule, one metabolites, maybe cannot give us enough power. And by combination of a lot of measurements, then we can get a more accurate diagnosis.</p>

<p><strong>Vicky Whittemore:</strong>&nbsp;Anyone else want to comment on that?</p>

<p><strong>Christopher Armstrong:</strong>&nbsp;Yeah.&nbsp; I think the idea of being able to capture a marker, specifically, as a diagnostic, would be complicated.&nbsp; It would be difficult to do it.&nbsp; But by the methods we've been applying.&nbsp; And I think, by trying to understand some of those unknowns that we've just talked about, perhaps there is an avenue.&nbsp; There are new metabolites and methods being developed all the time.&nbsp; That's right.&nbsp; New methods to measure metabolites are being developed all the time.&nbsp; There are ways to capture different ones.&nbsp; So, potentially, there's still avenues for that.&nbsp; But I think the idea of a signature makes a lot of sense.&nbsp; And in particular, I think it could be worthwhile to recognize that the actual -- the difficult parts of the diagnosis for ME/CFS is about excluding other diseases and differentiating ME/CFS from those other diseases.</p>

<p>And so, that's the area that needs to be targeted, in my opinion.&nbsp; That's the part that slowed down within the GP's offices and creates a problem for diagnosis.&nbsp; And so, finding markers that differentiate ME/CFS.&nbsp; So, ME/CFS has a signature from other similar conditions that a GP is trying to separate from through a myriad of different tests.&nbsp; If you could get a marker for that?&nbsp; I think that's well within reach.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Okay.&nbsp; I'll say thank you to all the panelists and participants who spoke today, and to all the participants who listened in and asked some really excellent questions.&nbsp; I'll turn it over to Alain.&nbsp; If you have any closing comments?</p>

<p><strong>Alain Moreau:&nbsp;</strong>This was a very great session.&nbsp; So, thank you all for the hard work.&nbsp; It's not a traditional way to make a big summary of what has been done before, and your own work and where we are going from that.&nbsp; And we really enjoyed all of your presentations.&nbsp; And thank you.&nbsp; And we'll use all the questions and comments in your slide show, of course, to build in part of the report.&nbsp; So -- and in the closed session, we will have more discussion about that with you guys.&nbsp; So, that will be even more sharper question, I hope so.&nbsp; Thank you all.</p>

<p><strong>Vicky Whittemore:&nbsp;</strong>Thank you, everyone.</p></div>
      

            </div>
      
      </div>
</div>
          </div>
  
  </div>

    </div>
  </div>


</article>

  </div>
<div class="views-element-container block block-views block-views-blockpage-last-reviewed-block-1" id="block-views-block-page-last-reviewed-block-1">
  
    
      <div><div class="js-view-dom-id-b48e1c22198eb98d70e5aef2238e050756dfd3bb92563a26972b32928676eb91">
  
  
  

  
  
  

      <div><p class="x-small-text">This page last reviewed on <time datetime="2025-01-22T14:35:43-05:00" class="datetime">January 22, 2025</time>
</p></div>

    

  
  

  
  
</div>
</div>

  </div>

  </div>

            </div>
          </div>
        
      </div>
    </div>

  </main>


<footer
  class="usa-footer usa-footer--medium" role="contentinfo">
          <div class="usa-footer__primary-section">
              <div class="grid-row grid-gap">
        <div class="mobile-lg:grid-col-12">
            <div class="region region-footer">
    <nav role="navigation" aria-labelledby="block-footer-menu" id="block-footer" class="block block-menu navigation usa-footer__nav menu--footer">
            
  <h2 class="visually-hidden" id="block-footer-menu">Footer menu</h2>
  

        

    
                <ul class="grid-row grid-gap add-list-reset flex-justify">
        
    
                  <li class="usa-footer__primary-content">
        <a href="/" class="usa-footer__primary-link" data-drupal-link-system-path="&lt;front&gt;">NIH Home</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="/about-nih/contact-us" class="usa-footer__primary-link" data-drupal-link-system-path="node/1686">Contact Us</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="https://salud.nih.gov/" class="usa-footer__primary-link">En Español</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="/virtual-tour" class="usa-footer__primary-link">Virtual Tour</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="/about-nih/visitor-information" class="usa-footer__primary-link" data-drupal-link-system-path="node/911">Visitor Information</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="/about-nih/frequently-asked-questions" class="usa-footer__primary-link" data-drupal-link-system-path="node/1691">FAQs</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="/news-events/nih-social-media" class="usa-footer__primary-link" data-drupal-link-system-path="node/186">NIH on Social Media</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="/press-room" class="usa-footer__primary-link" data-drupal-link-system-path="node/19366">Press Room</a>
      </li>
      
    
                  <li class="usa-footer__primary-content">
        <a href="/nih-website-archives" class="usa-footer__primary-link" data-drupal-link-system-path="node/18211">NIH Website Archives</a>
      </li>
      
    
        
        </ul>
        
  


  </nav>

  </div>

        </div>
      </div>
      </div>
  <div class="usa-footer__secondary-section">
    <div class="grid-container">
      <div class="grid-row grid-gap">
                  <div class="usa-footer__logo grid-row mobile-lg:grid-col-6 mobile-lg:grid-gap-2">
                                      <div class="mobile-lg:grid-col-auto">
                <p class="usa-footer__logo-heading">National Institutes of Health</p>
              </div>
                      </div>
                      </div>
              <div class="grid-row grid-gap">
          <div class="mobile-lg:grid-col-12">
              <div class="region region-footer-secondary">
    
<div id="block-secondaryfooter" class="block block-block-content block-block-content47fd288b-3b8e-4a6d-a175-073dee9475cb">
  
    


  <div class="grid-row grid-gap">
    <div class="usa-width-one-half">
            <div class="callout__text clearfix">
        <p>9000 Rockville Pike<br>
Bethesda, Maryland 20892</p>

<p>NIH…Turning Discovery Into Health<sup>®</sup></p>
      </div>
          </div>
    <div class="usa-width-one-half">
      <div class="usa-footer__social-links grid-row grid-gap-1">
        <div class="grid-col-auto">
          <p class="usa-footer__logo-heading">Follow Us</p>
        </div>
      </div>
      <div class="usa-footer__social-links grid-row grid-gap-1">
                <div class="grid-col-auto">
          <a class="usa-social-link" href="https://www.facebook.com/nih.gov"
            ><img
              class="usa-social-link__icon"
              src="/themes/custom/nih2_uswds/assets/img/usa-icons/facebook.svg"
              alt="Facebook"
          /></a>
        </div>
                        <div class="grid-col-auto">
          <a class="usa-social-link" href="https://twitter.com/NIH"
            ><img
              class="usa-social-link__icon"
              src="/themes/custom/nih2_uswds/assets/img/usa-icons/x.svg"
              alt="X"
          /></a>
        </div>
                        <div class="grid-col-auto">
          <a class="usa-social-link" href="https://www.youtube.com/user/nihod"
            ><img
              class="usa-social-link__icon"
              src="/themes/custom/nih2_uswds/assets/img/usa-icons/youtube.svg"
              alt="YouTube"
          /></a>
        </div>
                        <div class="grid-col-auto">
          <a class="usa-social-link" href="https://www.instagram.com/nihgov/"
            ><img
              class="usa-social-link__icon"
              src="/themes/custom/nih2_uswds/assets/img/usa-icons/instagram.svg"
              alt="Instagram"
          /></a>
        </div>
                        <div class="grid-col-auto">
          <a class="usa-social-link" href="https://www.flickr.com/photos/nihgov/"
            ><img
              class="usa-social-link__icon"
              src="/themes/custom/nih2_uswds/assets/img/usa-icons/flickr.svg"
              alt="Flickr"
          /></a>
        </div>
                        <div class="grid-col-auto">
          <a class="usa-social-link" href="/news-releases/feed.xml"
            ><img
              class="usa-social-link__icon"
              src="/themes/custom/nih2_uswds/assets/img/usa-icons/rss_feed.svg"
              alt="RSS"
          /></a>
        </div>
              </div>
    </div>
  </div>

    

  
</div>

  </div>

          </div>
        </div>
          </div>
  </div>
  </footer>

    <div class="region region-identifier">
    
<div id="block-identifier" class="block block-block-content block-block-content73e4ae9b-0020-48de-befe-3dda35d7590d usa-identifier">
  
    

  <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency Identifier">
    <div class="usa-identifier__container">
      <div class="usa-identifier__logos">
        <a href="http://www.nih.gov/" class="usa-identifier__logo"><img class="usa-identifier__logo-img" src="/themes/custom/nih2_uswds/assets/img/nih-mark-reversed.png" alt="Logo of the National Institutes of Health" role="img"></a>
      </div>
      <div class="usa-identifier__identity" aria-label="Descripción de la agencia">
        <p class="usa-identifier__identity-domain"><a href="https://www.nih.gov">www.nih.gov</a></p>
        <p class="usa-identifier__identity-disclaimer" aria-label="Official website of the Department of Health and Human Services">An official website of the <a href="http://www.hhs.gov/">Department of Health and Human Services</a></p>
      </div>
    </div>
  </section>
  <nav class="usa-identifier__section usa-identifier__section--required-links" aria-label="Important links">
    <div class="usa-identifier__container">

          
      <div class="field field--name-field-component field--type-entity-reference-revisions field--label-hidden field__items">
              <div class="field__item">  <div class="paragraph paragraph--type--uswds-text paragraph--view-mode--default usa-section usa-content">
    
            <div class="grid-col-fill">
      
      
            <div class="clearfix text-formatted field field--name-field-uswds-body field--type-text-long field--label-hidden field__item"><ul>
	<li><a href="/privacy-policy">Privacy Policy</a></li>
	<li><a href="/disclaimers">Disclaimers</a></li>
	<li><a href="/accessibility">Accessibility</a></li>
	<li><a href="/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office">Freedom of Information Act</a></li>
	<li><a href="https://eeo.nih.gov/no-fear-act">No Fear Act</a></li>
	<li><a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html">HHS Vulnerability Disclosure</a></li>
	<li><a href="https://oig.hhs.gov/">Office of Inspector General</a></li>
	<li><a href="/nondiscrimination-notice">Nondiscrimination Notice</a></li>
</ul></div>
      

            </div>
      
      </div>
</div>
          </div>
  
    
    </div>
  </nav>
<section class="usa-identifier__section usa-identifier__section--usagov" aria-label="U.S. government information and services">

    <div class="usa-identifier__container">
<div class="usa-identifier__usagov-description">Looking for U.S. government information and services?</div>

      <a href="https://www.usa.gov/" class="usa-link">Visit USA.gov</a>
    </div>
  </section>

</div>

  </div>


<div class="return-to-top">
  <a href="#">
    <svg class="usa-icon usa-icon--size-5" aria-labelledby="return-to-top" focusable="false" role="img">
      <title id="return-to-top">Subir</title>
      <use xlink:href="/themes/custom/nih2_uswds/assets/img/sprite.svg#arrow_upward"></use>
    </svg>
  </a>
</div>


  </div>

    
    <script src="/sites/default/files/js/js_IkN42OOTk7DC0mohqXSgF8BQJgK5VfFJkR6y7o-mnQ8.js?scope=footer&amp;delta=0&amp;language=en&amp;theme=nih2_uswds&amp;include=eJxdjYEOwiAMRH8IRuIPLQ3UWkUgpczp14uKZjNprr13ydULPO5I5Pw4DK4aOV1ckFYgTsOaI-vCobqxf54Tq6GcKeKsQI66_PsJzrDu4dUkPh3mVm-vzsjFepCwhYIV1RYg3FNtkqzmPmUbVA6YYOmY-pd9omghoug3_PC3PgGlVmKC"></script>


        <script>
    //<![CDATA[
          var usasearch_config = { siteHandle:"nih-espanol" };

          var script = document.createElement("script");
          script.type = "text/javascript";
          script.src = "//search.usa.gov/javascripts/remote.loader.js";
          document.getElementsByTagName("head")[0].appendChild(script);

    //]]>
    </script>

        <script type="text/javascript" async defer src="//assets.pinterest.com/js/pinit.js"></script>

  </body>
</html>
